{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "SciSpacy in Manual Text.ipynb",
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "metadata": {
        "id": "mULkvgCWZyyO"
      },
      "source": [
        "!rm -r /content/sample_data"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "z8sa4Bn9bP46"
      },
      "source": [
        "%mkdir an_data"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-yRPmUzkbQ4r",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "4beb8f55-8b77-44f4-8e8c-d3b154ecc172"
      },
      "source": [
        "%cd /content/an_data"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "/content/an_data\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "LYy0E41KbSUr",
        "colab": {
          "resources": {
            "http://localhost:8080/nbextensions/google.colab/files.js": {
              "data": "Ly8gQ29weXJpZ2h0IDIwMTcgR29vZ2xlIExMQwovLwovLyBMaWNlbnNlZCB1bmRlciB0aGUgQXBhY2hlIExpY2Vuc2UsIFZlcnNpb24gMi4wICh0aGUgIkxpY2Vuc2UiKTsKLy8geW91IG1heSBub3QgdXNlIHRoaXMgZmlsZSBleGNlcHQgaW4gY29tcGxpYW5jZSB3aXRoIHRoZSBMaWNlbnNlLgovLyBZb3UgbWF5IG9idGFpbiBhIGNvcHkgb2YgdGhlIExpY2Vuc2UgYXQKLy8KLy8gICAgICBodHRwOi8vd3d3LmFwYWNoZS5vcmcvbGljZW5zZXMvTElDRU5TRS0yLjAKLy8KLy8gVW5sZXNzIHJlcXVpcmVkIGJ5IGFwcGxpY2FibGUgbGF3IG9yIGFncmVlZCB0byBpbiB3cml0aW5nLCBzb2Z0d2FyZQovLyBkaXN0cmlidXRlZCB1bmRlciB0aGUgTGljZW5zZSBpcyBkaXN0cmlidXRlZCBvbiBhbiAiQVMgSVMiIEJBU0lTLAovLyBXSVRIT1VUIFdBUlJBTlRJRVMgT1IgQ09ORElUSU9OUyBPRiBBTlkgS0lORCwgZWl0aGVyIGV4cHJlc3Mgb3IgaW1wbGllZC4KLy8gU2VlIHRoZSBMaWNlbnNlIGZvciB0aGUgc3BlY2lmaWMgbGFuZ3VhZ2UgZ292ZXJuaW5nIHBlcm1pc3Npb25zIGFuZAovLyBsaW1pdGF0aW9ucyB1bmRlciB0aGUgTGljZW5zZS4KCi8qKgogKiBAZmlsZW92ZXJ2aWV3IEhlbHBlcnMgZm9yIGdvb2dsZS5jb2xhYiBQeXRob24gbW9kdWxlLgogKi8KKGZ1bmN0aW9uKHNjb3BlKSB7CmZ1bmN0aW9uIHNwYW4odGV4dCwgc3R5bGVBdHRyaWJ1dGVzID0ge30pIHsKICBjb25zdCBlbGVtZW50ID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnc3BhbicpOwogIGVsZW1lbnQudGV4dENvbnRlbnQgPSB0ZXh0OwogIGZvciAoY29uc3Qga2V5IG9mIE9iamVjdC5rZXlzKHN0eWxlQXR0cmlidXRlcykpIHsKICAgIGVsZW1lbnQuc3R5bGVba2V5XSA9IHN0eWxlQXR0cmlidXRlc1trZXldOwogIH0KICByZXR1cm4gZWxlbWVudDsKfQoKLy8gTWF4IG51bWJlciBvZiBieXRlcyB3aGljaCB3aWxsIGJlIHVwbG9hZGVkIGF0IGEgdGltZS4KY29uc3QgTUFYX1BBWUxPQURfU0laRSA9IDEwMCAqIDEwMjQ7CgpmdW5jdGlvbiBfdXBsb2FkRmlsZXMoaW5wdXRJZCwgb3V0cHV0SWQpIHsKICBjb25zdCBzdGVwcyA9IHVwbG9hZEZpbGVzU3RlcChpbnB1dElkLCBvdXRwdXRJZCk7CiAgY29uc3Qgb3V0cHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKG91dHB1dElkKTsKICAvLyBDYWNoZSBzdGVwcyBvbiB0aGUgb3V0cHV0RWxlbWVudCB0byBtYWtlIGl0IGF2YWlsYWJsZSBmb3IgdGhlIG5leHQgY2FsbAogIC8vIHRvIHVwbG9hZEZpbGVzQ29udGludWUgZnJvbSBQeXRob24uCiAgb3V0cHV0RWxlbWVudC5zdGVwcyA9IHN0ZXBzOwoKICByZXR1cm4gX3VwbG9hZEZpbGVzQ29udGludWUob3V0cHV0SWQpOwp9CgovLyBUaGlzIGlzIHJvdWdobHkgYW4gYXN5bmMgZ2VuZXJhdG9yIChub3Qgc3VwcG9ydGVkIGluIHRoZSBicm93c2VyIHlldCksCi8vIHdoZXJlIHRoZXJlIGFyZSBtdWx0aXBsZSBhc3luY2hyb25vdXMgc3RlcHMgYW5kIHRoZSBQeXRob24gc2lkZSBpcyBnb2luZwovLyB0byBwb2xsIGZvciBjb21wbGV0aW9uIG9mIGVhY2ggc3RlcC4KLy8gVGhpcyB1c2VzIGEgUHJvbWlzZSB0byBibG9jayB0aGUgcHl0aG9uIHNpZGUgb24gY29tcGxldGlvbiBvZiBlYWNoIHN0ZXAsCi8vIHRoZW4gcGFzc2VzIHRoZSByZXN1bHQgb2YgdGhlIHByZXZpb3VzIHN0ZXAgYXMgdGhlIGlucHV0IHRvIHRoZSBuZXh0IHN0ZXAuCmZ1bmN0aW9uIF91cGxvYWRGaWxlc0NvbnRpbnVlKG91dHB1dElkKSB7CiAgY29uc3Qgb3V0cHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKG91dHB1dElkKTsKICBjb25zdCBzdGVwcyA9IG91dHB1dEVsZW1lbnQuc3RlcHM7CgogIGNvbnN0IG5leHQgPSBzdGVwcy5uZXh0KG91dHB1dEVsZW1lbnQubGFzdFByb21pc2VWYWx1ZSk7CiAgcmV0dXJuIFByb21pc2UucmVzb2x2ZShuZXh0LnZhbHVlLnByb21pc2UpLnRoZW4oKHZhbHVlKSA9PiB7CiAgICAvLyBDYWNoZSB0aGUgbGFzdCBwcm9taXNlIHZhbHVlIHRvIG1ha2UgaXQgYXZhaWxhYmxlIHRvIHRoZSBuZXh0CiAgICAvLyBzdGVwIG9mIHRoZSBnZW5lcmF0b3IuCiAgICBvdXRwdXRFbGVtZW50Lmxhc3RQcm9taXNlVmFsdWUgPSB2YWx1ZTsKICAgIHJldHVybiBuZXh0LnZhbHVlLnJlc3BvbnNlOwogIH0pOwp9CgovKioKICogR2VuZXJhdG9yIGZ1bmN0aW9uIHdoaWNoIGlzIGNhbGxlZCBiZXR3ZWVuIGVhY2ggYXN5bmMgc3RlcCBvZiB0aGUgdXBsb2FkCiAqIHByb2Nlc3MuCiAqIEBwYXJhbSB7c3RyaW5nfSBpbnB1dElkIEVsZW1lbnQgSUQgb2YgdGhlIGlucHV0IGZpbGUgcGlja2VyIGVsZW1lbnQuCiAqIEBwYXJhbSB7c3RyaW5nfSBvdXRwdXRJZCBFbGVtZW50IElEIG9mIHRoZSBvdXRwdXQgZGlzcGxheS4KICogQHJldHVybiB7IUl0ZXJhYmxlPCFPYmplY3Q+fSBJdGVyYWJsZSBvZiBuZXh0IHN0ZXBzLgogKi8KZnVuY3Rpb24qIHVwbG9hZEZpbGVzU3RlcChpbnB1dElkLCBvdXRwdXRJZCkgewogIGNvbnN0IGlucHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKGlucHV0SWQpOwogIGlucHV0RWxlbWVudC5kaXNhYmxlZCA9IGZhbHNlOwoKICBjb25zdCBvdXRwdXRFbGVtZW50ID0gZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQob3V0cHV0SWQpOwogIG91dHB1dEVsZW1lbnQuaW5uZXJIVE1MID0gJyc7CgogIGNvbnN0IHBpY2tlZFByb21pc2UgPSBuZXcgUHJvbWlzZSgocmVzb2x2ZSkgPT4gewogICAgaW5wdXRFbGVtZW50LmFkZEV2ZW50TGlzdGVuZXIoJ2NoYW5nZScsIChlKSA9PiB7CiAgICAgIHJlc29sdmUoZS50YXJnZXQuZmlsZXMpOwogICAgfSk7CiAgfSk7CgogIGNvbnN0IGNhbmNlbCA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2J1dHRvbicpOwogIGlucHV0RWxlbWVudC5wYXJlbnRFbGVtZW50LmFwcGVuZENoaWxkKGNhbmNlbCk7CiAgY2FuY2VsLnRleHRDb250ZW50ID0gJ0NhbmNlbCB1cGxvYWQnOwogIGNvbnN0IGNhbmNlbFByb21pc2UgPSBuZXcgUHJvbWlzZSgocmVzb2x2ZSkgPT4gewogICAgY2FuY2VsLm9uY2xpY2sgPSAoKSA9PiB7CiAgICAgIHJlc29sdmUobnVsbCk7CiAgICB9OwogIH0pOwoKICAvLyBXYWl0IGZvciB0aGUgdXNlciB0byBwaWNrIHRoZSBmaWxlcy4KICBjb25zdCBmaWxlcyA9IHlpZWxkIHsKICAgIHByb21pc2U6IFByb21pc2UucmFjZShbcGlja2VkUHJvbWlzZSwgY2FuY2VsUHJvbWlzZV0pLAogICAgcmVzcG9uc2U6IHsKICAgICAgYWN0aW9uOiAnc3RhcnRpbmcnLAogICAgfQogIH07CgogIGNhbmNlbC5yZW1vdmUoKTsKCiAgLy8gRGlzYWJsZSB0aGUgaW5wdXQgZWxlbWVudCBzaW5jZSBmdXJ0aGVyIHBpY2tzIGFyZSBub3QgYWxsb3dlZC4KICBpbnB1dEVsZW1lbnQuZGlzYWJsZWQgPSB0cnVlOwoKICBpZiAoIWZpbGVzKSB7CiAgICByZXR1cm4gewogICAgICByZXNwb25zZTogewogICAgICAgIGFjdGlvbjogJ2NvbXBsZXRlJywKICAgICAgfQogICAgfTsKICB9CgogIGZvciAoY29uc3QgZmlsZSBvZiBmaWxlcykgewogICAgY29uc3QgbGkgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaScpOwogICAgbGkuYXBwZW5kKHNwYW4oZmlsZS5uYW1lLCB7Zm9udFdlaWdodDogJ2JvbGQnfSkpOwogICAgbGkuYXBwZW5kKHNwYW4oCiAgICAgICAgYCgke2ZpbGUudHlwZSB8fCAnbi9hJ30pIC0gJHtmaWxlLnNpemV9IGJ5dGVzLCBgICsKICAgICAgICBgbGFzdCBtb2RpZmllZDogJHsKICAgICAgICAgICAgZmlsZS5sYXN0TW9kaWZpZWREYXRlID8gZmlsZS5sYXN0TW9kaWZpZWREYXRlLnRvTG9jYWxlRGF0ZVN0cmluZygpIDoKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgJ24vYSd9IC0gYCkpOwogICAgY29uc3QgcGVyY2VudCA9IHNwYW4oJzAlIGRvbmUnKTsKICAgIGxpLmFwcGVuZENoaWxkKHBlcmNlbnQpOwoKICAgIG91dHB1dEVsZW1lbnQuYXBwZW5kQ2hpbGQobGkpOwoKICAgIGNvbnN0IGZpbGVEYXRhUHJvbWlzZSA9IG5ldyBQcm9taXNlKChyZXNvbHZlKSA9PiB7CiAgICAgIGNvbnN0IHJlYWRlciA9IG5ldyBGaWxlUmVhZGVyKCk7CiAgICAgIHJlYWRlci5vbmxvYWQgPSAoZSkgPT4gewogICAgICAgIHJlc29sdmUoZS50YXJnZXQucmVzdWx0KTsKICAgICAgfTsKICAgICAgcmVhZGVyLnJlYWRBc0FycmF5QnVmZmVyKGZpbGUpOwogICAgfSk7CiAgICAvLyBXYWl0IGZvciB0aGUgZGF0YSB0byBiZSByZWFkeS4KICAgIGxldCBmaWxlRGF0YSA9IHlpZWxkIHsKICAgICAgcHJvbWlzZTogZmlsZURhdGFQcm9taXNlLAogICAgICByZXNwb25zZTogewogICAgICAgIGFjdGlvbjogJ2NvbnRpbnVlJywKICAgICAgfQogICAgfTsKCiAgICAvLyBVc2UgYSBjaHVua2VkIHNlbmRpbmcgdG8gYXZvaWQgbWVzc2FnZSBzaXplIGxpbWl0cy4gU2VlIGIvNjIxMTU2NjAuCiAgICBsZXQgcG9zaXRpb24gPSAwOwogICAgd2hpbGUgKHBvc2l0aW9uIDwgZmlsZURhdGEuYnl0ZUxlbmd0aCkgewogICAgICBjb25zdCBsZW5ndGggPSBNYXRoLm1pbihmaWxlRGF0YS5ieXRlTGVuZ3RoIC0gcG9zaXRpb24sIE1BWF9QQVlMT0FEX1NJWkUpOwogICAgICBjb25zdCBjaHVuayA9IG5ldyBVaW50OEFycmF5KGZpbGVEYXRhLCBwb3NpdGlvbiwgbGVuZ3RoKTsKICAgICAgcG9zaXRpb24gKz0gbGVuZ3RoOwoKICAgICAgY29uc3QgYmFzZTY0ID0gYnRvYShTdHJpbmcuZnJvbUNoYXJDb2RlLmFwcGx5KG51bGwsIGNodW5rKSk7CiAgICAgIHlpZWxkIHsKICAgICAgICByZXNwb25zZTogewogICAgICAgICAgYWN0aW9uOiAnYXBwZW5kJywKICAgICAgICAgIGZpbGU6IGZpbGUubmFtZSwKICAgICAgICAgIGRhdGE6IGJhc2U2NCwKICAgICAgICB9LAogICAgICB9OwogICAgICBwZXJjZW50LnRleHRDb250ZW50ID0KICAgICAgICAgIGAke01hdGgucm91bmQoKHBvc2l0aW9uIC8gZmlsZURhdGEuYnl0ZUxlbmd0aCkgKiAxMDApfSUgZG9uZWA7CiAgICB9CiAgfQoKICAvLyBBbGwgZG9uZS4KICB5aWVsZCB7CiAgICByZXNwb25zZTogewogICAgICBhY3Rpb246ICdjb21wbGV0ZScsCiAgICB9CiAgfTsKfQoKc2NvcGUuZ29vZ2xlID0gc2NvcGUuZ29vZ2xlIHx8IHt9OwpzY29wZS5nb29nbGUuY29sYWIgPSBzY29wZS5nb29nbGUuY29sYWIgfHwge307CnNjb3BlLmdvb2dsZS5jb2xhYi5fZmlsZXMgPSB7CiAgX3VwbG9hZEZpbGVzLAogIF91cGxvYWRGaWxlc0NvbnRpbnVlLAp9Owp9KShzZWxmKTsK",
              "ok": true,
              "headers": [
                [
                  "content-type",
                  "application/javascript"
                ]
              ],
              "status": 200,
              "status_text": ""
            }
          },
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "3542de2e-d139-4995-cf22-0c22949ff1c8"
      },
      "source": [
        "from google.colab import files #Upload the annotated text files from local drive\n",
        "files.upload()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "\n",
              "     <input type=\"file\" id=\"files-a6529687-8bc2-4868-8186-aa187a532dcc\" name=\"files[]\" multiple disabled\n",
              "        style=\"border:none\" />\n",
              "     <output id=\"result-a6529687-8bc2-4868-8186-aa187a532dcc\">\n",
              "      Upload widget is only available when the cell has been executed in the\n",
              "      current browser session. Please rerun this cell to enable.\n",
              "      </output>\n",
              "      <script src=\"/nbextensions/google.colab/files.js\"></script> "
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "Saving a.g3qPkC.UM7xF33mZVi5sVpKNDm-1123316.tsv to a.g3qPkC.UM7xF33mZVi5sVpKNDm-1123316.tsv\n",
            "Saving a_40v3K2FGdNYfaS_6S6Jsri77.C-22635998.tsv to a_40v3K2FGdNYfaS_6S6Jsri77.C-22635998.tsv\n",
            "Saving a_fUcGCWV_jfoorNEfY9ZLrgTdzy-2310183.tsv to a_fUcGCWV_jfoorNEfY9ZLrgTdzy-2310183.tsv\n",
            "Saving a_gwKjTrMLtrwW4Ry3U8VFfZu_Lu-14706746.tsv to a_gwKjTrMLtrwW4Ry3U8VFfZu_Lu-14706746.tsv\n",
            "Saving a_rI55cZbMHdHTM0cNz7wVUGh.YG-15240330.tsv to a_rI55cZbMHdHTM0cNz7wVUGh.YG-15240330.tsv\n",
            "Saving a0G6qKR9kpJ.vYu5sRpUDK56l3Le-21418495.tsv to a0G6qKR9kpJ.vYu5sRpUDK56l3Le-21418495.tsv\n",
            "Saving a0VbJUpV5YemaEUU65sXwnHgUeo0-22392141.tsv to a0VbJUpV5YemaEUU65sXwnHgUeo0-22392141.tsv\n",
            "Saving a1G9_mwl5127bHyxwtrLqRP9Ep.u-20406281.tsv to a1G9_mwl5127bHyxwtrLqRP9Ep.u-20406281.tsv\n",
            "Saving a1O3245an7NEHwTx7mcFWQNFyto4-22991035.tsv to a1O3245an7NEHwTx7mcFWQNFyto4-22991035.tsv\n",
            "Saving a1pDgMCNLrLbKkZAlAdRBt2r.59S-28664326.tsv to a1pDgMCNLrLbKkZAlAdRBt2r.59S-28664326.tsv\n",
            "Saving a2l7vzT6_qWNbgnP8DGLDWCMEgyO-21468717.tsv to a2l7vzT6_qWNbgnP8DGLDWCMEgyO-21468717.tsv\n",
            "Saving a3cjro1JuxuGsMW3YFdMxf9Hv7KK-21531434.tsv to a3cjro1JuxuGsMW3YFdMxf9Hv7KK-21531434.tsv\n",
            "Saving a3HbOFyvJXWX.dfWfO8dHsBvWdxi-1622259.tsv to a3HbOFyvJXWX.dfWfO8dHsBvWdxi-1622259.tsv\n",
            "Saving a3OgoxXXgRa0homAO625jj3NfU2u-25487887.tsv to a3OgoxXXgRa0homAO625jj3NfU2u-25487887.tsv\n",
            "Saving a3Qc5NWvIJi.yuVfsskKvM2Tp4JG-24668245.tsv to a3Qc5NWvIJi.yuVfsskKvM2Tp4JG-24668245.tsv\n",
            "Saving a4.M_gLNftxxNQ_2aZgztZhuJPl8-31071671.tsv to a4.M_gLNftxxNQ_2aZgztZhuJPl8-31071671.tsv\n",
            "Saving a5jgtjZkpBoX6xpCfZU3376Kzlaq-234247.tsv to a5jgtjZkpBoX6xpCfZU3376Kzlaq-234247.tsv\n",
            "Saving a5YzvBm7c31lRjx7z_2nfMDygwl0-1718942.tsv to a5YzvBm7c31lRjx7z_2nfMDygwl0-1718942.tsv\n",
            "Saving a8.btHMLFHQHGdc7nUKzihjPP9y0-1592816.tsv to a8.btHMLFHQHGdc7nUKzihjPP9y0-1592816.tsv\n",
            "Saving a9y0qNOVEp4ztjwkaTBe1Pz6eETa-2067577.tsv to a9y0qNOVEp4ztjwkaTBe1Pz6eETa-2067577.tsv\n",
            "Saving a19opHP85j5SpAVTkODOJ4LNVwxa-1610174.tsv to a19opHP85j5SpAVTkODOJ4LNVwxa-1610174.tsv\n",
            "Saving a64iRnUPdE2jbaKQELV_UNrG2Zxi-1526465.tsv to a64iRnUPdE2jbaKQELV_UNrG2Zxi-1526465.tsv\n",
            "Saving a74YbCmMNjsB.SiEQcIn2iQ3Q70m-8226980.tsv to a74YbCmMNjsB.SiEQcIn2iQ3Q70m-8226980.tsv\n",
            "Saving a97SQMyLZ_YUFwcnZKRlxeuttZyC-16391099.tsv to a97SQMyLZ_YUFwcnZKRlxeuttZyC-16391099.tsv\n",
            "Saving a237AbH6ipbjuFjK_fKc7jngiNwS-25025097.tsv to a237AbH6ipbjuFjK_fKc7jngiNwS-25025097.tsv\n",
            "Saving aAjTJkYkdurOuzdNS2EqvP8DTTfS-1447132.tsv to aAjTJkYkdurOuzdNS2EqvP8DTTfS-1447132.tsv\n",
            "Saving aAMflZPODkJwGSZvpxMEZCCrxhLy-31394342.tsv to aAMflZPODkJwGSZvpxMEZCCrxhLy-31394342.tsv\n",
            "Saving aAR7COUYPwWwooLSEEo7umBWUrEC-22003008.tsv to aAR7COUYPwWwooLSEEo7umBWUrEC-22003008.tsv\n",
            "Saving aArRcX1.yw2C779LPKZEQgkPFd5O-1961727.tsv to aArRcX1.yw2C779LPKZEQgkPFd5O-1961727.tsv\n",
            "Saving aAvpUplV__m4CU1IcWvDMWWuHY0y-18606539.tsv to aAvpUplV__m4CU1IcWvDMWWuHY0y-18606539.tsv\n",
            "Saving aBgzC9zRQ5gI.RRbvzcyxs3v.erG-24123738.tsv to aBgzC9zRQ5gI.RRbvzcyxs3v.erG-24123738.tsv\n",
            "Saving aBlYzaPGSQp3xU8t_IxrAGMCyvrC-23454590.tsv to aBlYzaPGSQp3xU8t_IxrAGMCyvrC-23454590.tsv\n",
            "Saving aBN1Qmxk3Xj19lwi8__Xx43FTPcO-29563.tsv to aBN1Qmxk3Xj19lwi8__Xx43FTPcO-29563.tsv\n",
            "Saving aBP4ogv6rktF6czR15ipvZka18oS-1885512.tsv to aBP4ogv6rktF6czR15ipvZka18oS-1885512.tsv\n",
            "Saving aBRIckae0Mpac9CxsT6pSNnkHqf0-26776948.tsv to aBRIckae0Mpac9CxsT6pSNnkHqf0-26776948.tsv\n",
            "Saving aCdU83YXibjOzSJ8u3BvmyF344eS-31445334.tsv to aCdU83YXibjOzSJ8u3BvmyF344eS-31445334.tsv\n",
            "Saving aCL5B7lodNpiF9VFK56lfoXwR6jK-26308673.tsv to aCL5B7lodNpiF9VFK56lfoXwR6jK-26308673.tsv\n",
            "Saving aD3qB7nGh9V6zv33OQXsumxeJ7OC-1254564.tsv to aD3qB7nGh9V6zv33OQXsumxeJ7OC-1254564.tsv\n",
            "Saving adFPfP5Za2Tqr7GICnCj4szpgbR0-21974851.tsv to adFPfP5Za2Tqr7GICnCj4szpgbR0-21974851.tsv\n",
            "Saving adHR6sQH7bCPevVwArbdhgPTsT58-24676750.tsv to adHR6sQH7bCPevVwArbdhgPTsT58-24676750.tsv\n",
            "Saving aEHh.KmFDmnx4LqVJgZmN2.IU97y-22854895.tsv to aEHh.KmFDmnx4LqVJgZmN2.IU97y-22854895.tsv\n",
            "Saving aeLjOQVjTmU6CxOxaC6IOPkD15ai-29150672.tsv to aeLjOQVjTmU6CxOxaC6IOPkD15ai-29150672.tsv\n",
            "Saving aeRSJ.g51yv44AhUWaZmaS3zrcwS-1416975.tsv to aeRSJ.g51yv44AhUWaZmaS3zrcwS-1416975.tsv\n",
            "Saving af5Vd1_V.IZVv2fy8SRV3TX_iUQ8-28401633.tsv to af5Vd1_V.IZVv2fy8SRV3TX_iUQ8-28401633.tsv\n",
            "Saving afHujGvs_M.1SLW0m4OXadVRJvT8-31549657.tsv to afHujGvs_M.1SLW0m4OXadVRJvT8-31549657.tsv\n",
            "Saving afJ8f.ek_mLp6PuUrK9e4uiRMQc8-2219955.tsv to afJ8f.ek_mLp6PuUrK9e4uiRMQc8-2219955.tsv\n",
            "Saving aFQ5GrPy0N9DBLuocsDuRxtTrpWq-1327782.tsv to aFQ5GrPy0N9DBLuocsDuRxtTrpWq-1327782.tsv\n",
            "Saving aFqlwZAcl3EaVCR._itfw0Bg7ecG-2394680.tsv to aFqlwZAcl3EaVCR._itfw0Bg7ecG-2394680.tsv\n",
            "Saving afYImpCZ9lNrPIWIyzjfgfu2pcbu-26273258.tsv to afYImpCZ9lNrPIWIyzjfgfu2pcbu-26273258.tsv\n",
            "Saving aGGjZft5cKAxMFYpf3D1CHm4eTP4-925012.tsv to aGGjZft5cKAxMFYpf3D1CHm4eTP4-925012.tsv\n",
            "Saving agM5xp_DbJltbmulA8.crpkhzCjC-30647456.tsv to agM5xp_DbJltbmulA8.crpkhzCjC-30647456.tsv\n",
            "Saving ah7ZS_ptJZlahOrm7G4ReJWe5mFy-1429434.tsv to ah7ZS_ptJZlahOrm7G4ReJWe5mFy-1429434.tsv\n",
            "Saving aHg0m7EJoE7Lqw1a3K8dzuvFIwIu-23560662.tsv to aHg0m7EJoE7Lqw1a3K8dzuvFIwIu-23560662.tsv\n",
            "Saving aHlTwI8ihDgYtvTxoIMATAE97Y10-23076592.tsv to aHlTwI8ihDgYtvTxoIMATAE97Y10-23076592.tsv\n",
            "Saving ahsvvoMu84Zjr7Rhu16yMIALmbNW-21615634.tsv to ahsvvoMu84Zjr7Rhu16yMIALmbNW-21615634.tsv\n",
            "Saving aHUtck9wjaHujHIYEMKTrb8lB3zu-8005.tsv to aHUtck9wjaHujHIYEMKTrb8lB3zu-8005.tsv\n",
            "Saving ahwTYdMzdaIWKB3vd5dHo7Gtzk0m-28363960.tsv to ahwTYdMzdaIWKB3vd5dHo7Gtzk0m-28363960.tsv\n",
            "Saving ai.4gEPfAl8ZZ1zaemJCkzjRZIcS-1587481.tsv to ai.4gEPfAl8ZZ1zaemJCkzjRZIcS-1587481.tsv\n",
            "Saving aiJjGYw7ugb4aPZPKqQPn.7BC6Jq-1447127.tsv to aiJjGYw7ugb4aPZPKqQPn.7BC6Jq-1447127.tsv\n",
            "Saving aioL7f.VZae8CICcRDJxPjZZ9i.u-23066709.tsv to aioL7f.VZae8CICcRDJxPjZZ9i.u-23066709.tsv\n",
            "Saving aiyTKDM2vSKVPC57nYa4PXKHMzGm-18451.tsv to aiyTKDM2vSKVPC57nYa4PXKHMzGm-18451.tsv\n",
            "Saving aiZwtrEfgPnqN6Sj4buoSESUOhmK-27005792.tsv to aiZwtrEfgPnqN6Sj4buoSESUOhmK-27005792.tsv\n",
            "Saving ajgdpPcEt7PN4OhUnb8AiAxsqqku-30326388.tsv to ajgdpPcEt7PN4OhUnb8AiAxsqqku-30326388.tsv\n",
            "Saving ajipNizGkakHghlktMYLqC1CDxdO-10919784.tsv to ajipNizGkakHghlktMYLqC1CDxdO-10919784.tsv\n",
            "Saving aJU0mGpHnbYRcwLiK45Pyiv8kPBS-24239819.tsv to aJU0mGpHnbYRcwLiK45Pyiv8kPBS-24239819.tsv\n",
            "Saving aKEgV7EueJsNLXQ4_VvYw9dxQ7yu-1453953.tsv to aKEgV7EueJsNLXQ4_VvYw9dxQ7yu-1453953.tsv\n",
            "Saving akmW.Z1YzVjwZLOuxDEqL8FcuU84-1885526.tsv to akmW.Z1YzVjwZLOuxDEqL8FcuU84-1885526.tsv\n",
            "Saving akNQ3DlOY.5VE2HxxEG85I2F2dhi-19625116.tsv to akNQ3DlOY.5VE2HxxEG85I2F2dhi-19625116.tsv\n",
            "Saving akTMPuJstdvfFzK0APrYk1dzo31q-21396889.tsv to akTMPuJstdvfFzK0APrYk1dzo31q-21396889.tsv\n",
            "Saving alFClOzZRZSztDE_huAmaKx0apz0-19002456.tsv to alFClOzZRZSztDE_huAmaKx0apz0-19002456.tsv\n",
            "Saving aLqTZF8guoL3.vLWZwhCJz03WWyG-22307296.tsv to aLqTZF8guoL3.vLWZwhCJz03WWyG-22307296.tsv\n",
            "Saving alUk0HHVUEOu_iv56ZVCjPLj5B_8-1720168.tsv to alUk0HHVUEOu_iv56ZVCjPLj5B_8-1720168.tsv\n",
            "Saving aM2.ASbdUakhKQsFMULm0m.241eu-24898178.tsv to aM2.ASbdUakhKQsFMULm0m.241eu-24898178.tsv\n",
            "Saving aM4A2nEttHUsrp616fh2R6Qrk.ru-1268229.tsv to aM4A2nEttHUsrp616fh2R6Qrk.ru-1268229.tsv\n",
            "Saving aMecN20D4rUvYkTsLUdOC2E6wagm-27984820.tsv to aMecN20D4rUvYkTsLUdOC2E6wagm-27984820.tsv\n",
            "Saving amqJtPoDnPqhqD2Mz64n9qFnjLnS-30765135.tsv to amqJtPoDnPqhqD2Mz64n9qFnjLnS-30765135.tsv\n",
            "Saving aMWcBtBI2Z2a0njuP81Tr3mX6I0m-28321487.tsv to aMWcBtBI2Z2a0njuP81Tr3mX6I0m-28321487.tsv\n",
            "Saving anEWNcuFgnjOgnlsSzzA3t6wyxxm-11319111.tsv to anEWNcuFgnjOgnlsSzzA3t6wyxxm-11319111.tsv\n",
            "Saving anqXB6uId6MDusV6KvxX_W7x0HEC-22544249.tsv to anqXB6uId6MDusV6KvxX_W7x0HEC-22544249.tsv\n",
            "Saving aNtTIxCXZIoCbW2sw7pg6qma8hbK-15883866.tsv to aNtTIxCXZIoCbW2sw7pg6qma8hbK-15883866.tsv\n",
            "Saving aNUC2kQ9JHXhzRelFXkqKr8nCllq-1577754.tsv to aNUC2kQ9JHXhzRelFXkqKr8nCllq-1577754.tsv\n",
            "Saving aNYIc2VJb1lvg_2sFDoCarzYacLS-17117994.tsv to aNYIc2VJb1lvg_2sFDoCarzYacLS-17117994.tsv\n",
            "Saving ao0KyFqjKjEwaK8YvffA4rnFfU9q-681275.tsv to ao0KyFqjKjEwaK8YvffA4rnFfU9q-681275.tsv\n",
            "Saving ao7_nk1Xq7z9.O6P5YMtRjkcAB7a-22247165.tsv to ao7_nk1Xq7z9.O6P5YMtRjkcAB7a-22247165.tsv\n",
            "Saving aO33b5hse6bt4xZVSrrcQgUo9jO4-28006730.tsv to aO33b5hse6bt4xZVSrrcQgUo9jO4-28006730.tsv\n",
            "Saving ap6QaMTWuCa9XFmImeAixzmBeZZO-23022948.tsv to ap6QaMTWuCa9XFmImeAixzmBeZZO-23022948.tsv\n",
            "Saving aPjCyemv4fUpmXpk7VoVwjyBDs_u-1371658.tsv to aPjCyemv4fUpmXpk7VoVwjyBDs_u-1371658.tsv\n",
            "Saving arDLI8bpK.7etpUqvs5jM.I3rNqu-11831474.tsv to arDLI8bpK.7etpUqvs5jM.I3rNqu-11831474.tsv\n",
            "Saving aS4kJlp8QNdWM003oQeKhHPXfZ34-16329946.tsv to aS4kJlp8QNdWM003oQeKhHPXfZ34-16329946.tsv\n",
            "Saving asarkR8qt4kkCy31TNJmI5eqS0Za-22176786.tsv to asarkR8qt4kkCy31TNJmI5eqS0Za-22176786.tsv\n",
            "Saving aSnQ6wHh4xWTKzoJ8bT4HhzeOVPy-23586778.tsv to aSnQ6wHh4xWTKzoJ8bT4HhzeOVPy-23586778.tsv\n",
            "Saving asrNy8uPI86WRMSZOyIZ15gB_nI8-25582347.tsv to asrNy8uPI86WRMSZOyIZ15gB_nI8-25582347.tsv\n",
            "Saving at2euSJ6UnNz47kXoJpW0dqA1x3y-23186529.tsv to at2euSJ6UnNz47kXoJpW0dqA1x3y-23186529.tsv\n",
            "Saving aTjaxECCPKlNaQUPN9wzGhuTosMG-20305027.tsv to aTjaxECCPKlNaQUPN9wzGhuTosMG-20305027.tsv\n",
            "Saving atSqjJMKCyOI03pI2eklLyIFEk7q-1938949.tsv to atSqjJMKCyOI03pI2eklLyIFEk7q-1938949.tsv\n",
            "Saving aU.B2YOcXQQ0c__tKQlZmVhxL9.W-24862007.tsv to aU.B2YOcXQQ0c__tKQlZmVhxL9.W-24862007.tsv\n",
            "Saving av5KK4Dy7_iLJ7bZjPYS8aW2_utK-1987135.tsv to av5KK4Dy7_iLJ7bZjPYS8aW2_utK-1987135.tsv\n",
            "Saving aV17kDPU_zg1fV8gxsBjgBRUOw_W-1853203.tsv to aV17kDPU_zg1fV8gxsBjgBRUOw_W-1853203.tsv\n",
            "Saving aVCsGoaq69gyqXoQPrYR_Xh_8YHy-19497847.tsv to aVCsGoaq69gyqXoQPrYR_Xh_8YHy-19497847.tsv\n",
            "Saving aVDY7Ev59m70WiI7f_a67hScGBiS-2034671.tsv to aVDY7Ev59m70WiI7f_a67hScGBiS-2034671.tsv\n",
            "Saving aVlBjfk8CtLMfZEIJbKUN7dxO2DW-1892384.tsv to aVlBjfk8CtLMfZEIJbKUN7dxO2DW-1892384.tsv\n",
            "Saving aVpYWI_FIBmNF9Q.wYEuetl8lCLi-1444373.tsv to aVpYWI_FIBmNF9Q.wYEuetl8lCLi-1444373.tsv\n",
            "Saving aVXBoAqrkI_nbpmM_mu.Br9fKHzC-19251892.tsv to aVXBoAqrkI_nbpmM_mu.Br9fKHzC-19251892.tsv\n",
            "Saving aW12Y2HqOS2s.HdUDpjI4tCPW.aK-1444422.tsv to aW12Y2HqOS2s.HdUDpjI4tCPW.aK-1444422.tsv\n",
            "Saving awCM8T0i1WF_qAGxsa.VbPiG6ESG-24928877.tsv to awCM8T0i1WF_qAGxsa.VbPiG6ESG-24928877.tsv\n",
            "Saving aWjQPKPgXqYQt4s0er.XVq9ksHf4-30324341.tsv to aWjQPKPgXqYQt4s0er.XVq9ksHf4-30324341.tsv\n",
            "Saving aXadYOMx.9g1YFCDL6FE4VwsFQ9G-29886364.tsv to aXadYOMx.9g1YFCDL6FE4VwsFQ9G-29886364.tsv\n",
            "Saving axkF9gyJczwVDWPUPFdllX89G2yi-22938719.tsv to axkF9gyJczwVDWPUPFdllX89G2yi-22938719.tsv\n",
            "Saving aXV8WPhnnXaKBDRW.KaFM9h3JK_8-24721596.tsv to aXV8WPhnnXaKBDRW.KaFM9h3JK_8-24721596.tsv\n",
            "Saving ay109tu2e7Y4hDEQbQsupvVzOxqi-28762129.tsv to ay109tu2e7Y4hDEQbQsupvVzOxqi-28762129.tsv\n",
            "Saving aybjIDVZs86m9ZxgZrF128jpHnVy-20401686.tsv to aybjIDVZs86m9ZxgZrF128jpHnVy-20401686.tsv\n",
            "Saving aYcUnBgYZ0CMSJ0ltle2cqxMkSES-23811502.tsv to aYcUnBgYZ0CMSJ0ltle2cqxMkSES-23811502.tsv\n",
            "Saving aYt2ZK7qs9azR0119c8jCx.UWB3C-20088886.tsv to aYt2ZK7qs9azR0119c8jCx.UWB3C-20088886.tsv\n",
            "Saving aYzMIzEZRsCOVevEzvdU9zR6IuCa-19709256.tsv to aYzMIzEZRsCOVevEzvdU9zR6IuCa-19709256.tsv\n",
            "Saving azow9xfl_vua1heIv1oyGayw3.CW-1390687.tsv to azow9xfl_vua1heIv1oyGayw3.CW-1390687.tsv\n",
            "Saving azrOx67dvJt8Z_RDwJlO6UaiI5py-1768085.tsv to azrOx67dvJt8Z_RDwJlO6UaiI5py-1768085.tsv\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'a.g3qPkC.UM7xF33mZVi5sVpKNDm-1123316.tsv': b'Pathways for the degradation of m-cresol and p-cresol by Pseudomonas putida.\\tO\\n\\nAbstract\\tO\\n\\nA comparison of the oxidation rates of various compounds by whole cells of Pseudomonas putida 3, 5 indicated that m-cresol is metabolized by oxidation to 3-hydroxybenzoate followed by hydroxylation to gentisate, the ring-fission substrate, when grown with 3, 5-xylenol. However, when m-cresol was the growth substrate, similar experiments suggested a different pathway involving a methyl-substituted catechol, and ring-fission by meta cleavage. Assays of ring-fission enzymes in cell-free extracts confirmed that different pathways are induced by the two growth substrates. 3, 5-Xylenol-grown cells contained high levels of \\tO\\ngentisate oxygenase\\tenzyme\\n and only very small amounts of \\tO\\ncatechol oxygenase\\tenzyme\\n, whereas gentisate ocygenase could not be detected in m-cresol-grown cells, but levels of \\tO\\ncatechol oxygenase\\tenzyme\\n were greatly increased. Extracts of m-cresol-grown cells also contained \\tO\\n2-hydroxymuconic semialdehyde dehydrogenase\\tenzyme\\n and hydrolase, whose specificities enable them to metabolize the ring-fission products from catechol, 3-methylcatechol, and 4-methylcatechol. This catechol pathway is also used by m-cresol-grown cells for p-cresol metabolism. In contrast, the results for cells grown with p-cresol point to an alternative pathway involving oxidation to 4-hydroxybenzoate and hydrosylation to protocatechuate as ring-fission substrate. Extracts of these cells contained high levels of protocatechuate oxygenase and only small amounts of \\tO\\ncatechol oxygenase\\tenzyme\\n.\\tO\\n\\n',\n",
              " 'a0G6qKR9kpJ.vYu5sRpUDK56l3Le-21418495.tsv': b'(R,S)-dichlorprop herbicide in agricultural soil induces proliferation and expression of multiple dioxygenase-encoding genes in the indigenous microbial community.\\tO\\n\\nAbstract\\tO\\n\\nWe investigated the effect of (R,S)-dichlorprop herbicide addition to soil microcosms on the degrading indigenous microbial community by targeting multiple \\xce\\xb1-ketoglutarate-dependent (\\xce\\xb1-KG) \\tO\\ndioxygenase\\tenzyme\\n-encoding genes (\\tO\\nrdpA\\tgene\\n, \\tO\\nsdpA\\tgene\\n and \\tO\\ntfdA\\tgene\\n group I) at both gene and transcript level. The soil microbial community responded with high growth of potential degraders as measured by the abundance of \\tO\\ndioxygenase\\tenzyme\\n-encoding genes using quantitative real-time PCR (qPCR). \\tO\\nrdpA\\tgene\\n DNA was not detectable in unamended soil but reached over 10\\xe2\\x81\\xb6 copies g\\xe2\\x81\\xbb\\xc2\\xb9 soil after amendment. \\tO\\nsdpA\\tgene\\n and \\tO\\ntfdA\\tgene\\n were both present prior to amendment at levels of ~5 \\xc3\\x97 10\\xe2\\x81\\xb4 and ~ 10\\xc2\\xb2 copies g\\xe2\\x81\\xbb\\xc2\\xb9 soil, respectively, and both reached over 10\\xe2\\x81\\xb5copies g\\xe2\\x81\\xbb\\xc2\\xb9 soil. While expression of all three target genes was detected during two cycles of herbicide degradation, a time-shift occurred between maximum expression of each gene. Gene diversity by denaturing gradient gel electrophoresis (DGGE) uncovered a diversity of \\tO\\nsdpA\\tgene\\n and \\tO\\ntfdA\\tgene\\n genes at the DNA level while \\tO\\nrdpA\\tgene\\n remained highly conserved. However, mRNA profiles indicated that all transcribed \\tO\\ntfdA\\tgene\\n sequences were class III genes while \\tO\\nrdpA\\tgene\\n transcripts shared 100% identity to \\tO\\nrdpA\\tgene\\n of Delftia acidovorans MC1 and \\tO\\nsdpA\\tgene\\n transcripts shared 100% identity to \\tO\\nsdpA\\tgene\\n from Sphingomonas herbicidovorans MH. This is the first report to describe expression dynamics of multiple \\xce\\xb1-KG \\tO\\ndioxygenase\\tenzyme\\n-encoding genes in the indigenous microbial community as related to degradation of a phenoxypropionate herbicide in soil.\\tO\\n\\n',\n",
              " 'a0VbJUpV5YemaEUU65sXwnHgUeo0-22392141.tsv': b'Managing methanogens and homoacetogens to promote reductive dechlorination of trichloroethene with direct delivery of H2 in a membrane biofilm reactor.\\tO\\n\\nAbstract\\tO\\n\\nA study with H(2)-based membrane biofilm reactors (MBfRs) was undertaken to examine the effectiveness of direct H(2) delivery in ex-situ reductive dechlorination of chlorinated ethenes. Trichloroethene (TCE) could be reductively dechlorinated to ethene with up to 95% efficiency as long as the pH-increase effects of methanogens and homoacetogens were managed and dechlorinators were selected for during start-up by creating H(2) limitation. Based on quantitative PCR, the dominant bacterial groups in the biofilm at the end of reactor operation were Dehalococcoides, Geobacter, and homoacetogens. Pyrosequencing confirmed the dominance of the dechlorinators and identified Acetobacterium as the key homoacetogen. Homoacetogens outcompeted methanogens for bicarbonate, based on the effluent concentration of acetate, by suppressing methanogens during batch start-up. This was corroborated by the methanogenesis functional gene mcrA, which was 1-2 orders of magnitude lower than the \\tO\\nFTHFS\\tgene\\n functional gene for homoacetogens. Imaging of the MBfR fibers using scanning electron microscopy showed a distinct Dehalococcoides-like morphology in the fiber biofilm. These results support that direct addition of H(2) can allow for efficient and complete reductive dechlorination, and they shed light into how H(2)-fed biofilms, when operated to manage methanogenic and homoacetogenic activity, can be used for ex-situ bioremediation of chlorinated ethenes.\\tO\\n\\n',\n",
              " 'a19opHP85j5SpAVTkODOJ4LNVwxa-1610174.tsv': b'Biodegradation of the herbicide bromoxynil (3,5-dibromo-4-hydroxybenzonitrile) by purified \\tO\\npentachlorophenol hydroxylase\\tenzyme\\n and whole cells of Flavobacterium sp. strain ATCC 39723 is accompanied by cyanogenesis.\\tO\\n\\nAbstract\\tO\\n\\nA pentachlorophenol (PCP)-degrading Flavobacterium sp. (strain ATCC 39723) degraded bromoxynil with the production of bromide and cyanide. No aromatic intermediates were detected in the spent culture fluid. The cyanide produced upon bromoxynil metabolism was inhibitory to the Flavobacterium sp. Whole cells degraded PCP more rapidly than they did bromoxynil. Bromoxynil metabolism and PCP metabolism were coinduced, either substrate serving as the inducer. Purified \\tO\\nPCP hydroxylase\\tenzyme\\n degraded bromoxynil with stoichiometric accumulation of cyanide and without bromide production. A product accumulated which was more hydrophilic than bromoxynil upon high-pressure liquid chromatographic analysis and which, when analyzed by gas chromatography-mass spectrometry, had a mass spectrum consistent with that expected for dibromohydroquinone. \\tO\\nPCP hydroxylase\\tenzyme\\n consumed NADPH, oxygen, and bromoxynil in a 2:1:1 molar ratio, producing 1 mol of cyanide per mol of bromoxynil degraded. We propose a pathway by which bromoxynil is metabolized by the same enzymes which degrade PCP. The initial step in the pathway is the conversion of bromoxynil to 2,6-dibromohydroquinone by \\tO\\nPCP hydroxylase\\tenzyme\\n. In addition to its utility for decontaminating PCP-polluted sites, the Flavobacterium sp. may be useful for decontaminating bromoxynil spills. This is the first report of cyanide production accompanying the metabolism of a benzonitrile derivative.\\tO\\n\\n',\n",
              " 'a1G9_mwl5127bHyxwtrLqRP9Ep.u-20406281.tsv': b'Novel \\xce\\xb1-ketoglutarate dioxygenase tfdA-related genes are found in soil DNA after exposure to phenoxyalkanoic herbicides.\\tO\\n\\nAbstract\\tO\\n\\nPhenoxyalkanoic herbicides such as 2,4-dichlorophenoxyacetate (2,4-D), 2,4-dichlorophenoxybutyrate (2,4-DB) or mecoprop are widely used to control broad-leaf weeds. Several bacteria have been reported to degrade these herbicides using the \\xce\\xb1-ketoglutarate-dependent, \\tO\\n2,4-dichlorophenoxyacetate dioxygenase\\tenzyme\\n encoded by the \\tO\\ntfdA\\tgene\\n gene, as the enzyme catalysing the first step in the catabolic pathway. The effects of exposure to different phenoxyalkanoic herbicides in the soil bacterial community and in the \\tO\\ntfdA\\tgene\\n genes diversity were assessed using an agricultural soil exposed to these anthropogenic compounds. Total community bacterial DNA was analysed by terminal restriction fragment length polymorphism of the 16S rRNA and the \\tO\\ntfdA\\tgene\\n gene markers, and detection and cloning of \\tO\\ntfdA\\tgene\\n gene related sequences, using PCR primer pairs. After up to 4 months of herbicide exposure, significant changes in the bacterial community structure were detected in soil microcosms treated with mecoprop, 2,4-DB and a mixture of both plus 2,4-D. An impressive variety of novel \\tO\\ntfdA\\tgene\\n gene related sequences were found in these soil microcosms, which cluster in new \\tO\\ntfdA\\tgene\\n gene related sequence groups, unequally abundant depending on the specific herbicide used in soil treatment. Structural analysis of the putative protein products showed small but significant amino acid differences. These \\tO\\ntfdA\\tgene\\n gene sequence variants are, probably, required for degradation of natural substrate(s) structurally related to these herbicides and their presence explains self-remediation of soils exposed to phenoxyalkanoic herbicides.\\tO\\n\\n',\n",
              " 'a1O3245an7NEHwTx7mcFWQNFyto4-22991035.tsv': b'Estimating the biodegradation of pesticide in soils by monitoring pesticide-degrading gene expression.\\tO\\n\\nAbstract\\tO\\n\\nAssessing in situ microbial abilities of soils to degrade pesticides is of great interest giving insight in soil filtering capability, which is a key ecosystem function limiting pollution of groundwater. Quantification of pesticide-degrading gene expression by reverse transcription quantitative PCR (RT-qPCR) was tested as a suitable indicator to monitor pesticide biodegradation performances in soil. RNA extraction protocol was optimized to enhance the yield and quality of RNA recovered from soil samples to perform RT-qPCR assays. As a model, the activity of atrazine-degrading communities was monitored using RT-qPCRs to estimate the level of expression of \\tO\\natzD\\tgene\\n in five agricultural soils showing different atrazine mineralization abilities. Interestingly, the relative abundance of \\tO\\natzD\\tgene\\n mRNA copy numbers was positively correlated to the maximum rate and to the maximal amount of atrazine mineralized. Our findings indicate that the quantification of pesticide-degrading gene expression may be suitable to assess biodegradation performance in soil and monitor natural attenuation of pesticide.\\tO\\n\\n',\n",
              " 'a1pDgMCNLrLbKkZAlAdRBt2r.59S-28664326.tsv': b'Manganese-enhanced degradation of lignocellulosic waste by Phanerochaete chrysosporium: evidence of enzyme activity and gene transcription.\\tO\\n\\nAbstract\\tO\\n\\nLignolytic fungi initiate lignocellulose decay by producing extracellular oxidative enzymes. For better understanding the enzymatic degradation of lignocellulose by white-rot fungi, we investigated the effect of manganese on the organic matter loss, \\tO\\nmanganese peroxidase\\tenzyme\\n (MnP) activity, and \\tO\\nmanganese peroxidase\\tenzyme\\n gene (mnp) transcription levels during solid-state fermentation of rice straw with Phanerochaete chrysosporium. The results showed that the addition of manganese improved MnP activity and made it reach the peak earlier, promoted fungal growth at the early period (0-9 days), and enhanced the degradation of lignocellulosic waste. The total organic matter loss had a good correlation with fungal biomass during 30 days of cultivation, and manganese amendment promoted the ability of P. chrysosporium to degrade lignocellulose. Quantitative real-time RT-PCR revealed the differential expression of \\tO\\nmnp1\\tgene\\n, \\tO\\nmnp2\\tgene\\n, and \\tO\\nmnp3\\tgene\\n: manganese amendment increased the transcription of \\tO\\nmnp1\\tgene\\n and \\tO\\nmnp2\\tgene\\n but not \\tO\\nmnp3\\tgene\\n. The results indicated that manganese stimulated mnp transcription levels and played a post-transcriptional role in MnP production. These findings provide opportunity of development in enzymatic degradation of lignocellulosic waste by P. chrysosporium amended with manganese.\\tO\\n\\n',\n",
              " 'a237AbH6ipbjuFjK_fKc7jngiNwS-25025097.tsv': b'Quantitative and functional dynamics of Dehalococcoides spp. and its \\tO\\ntceA\\tgene\\n and \\tO\\nvcrA\\tgene\\n genes under TCE exposure.\\tO\\n\\nAbstract\\tO\\n\\nThis study aimed at monitoring the dynamics of phylogenetic and catabolic genes of a dechlorinating enrichment culture before, during, and after complete dechlorination of chlorinated compounds. More specifically, the effect of 40 \\xce\\xbcM trichloroethene (TCE) and 5.6 mM lactate on the gene abundance and activity of an enrichment culture was investigated for 40 days. Although \\tO\\ntceA\\tgene\\n and \\tO\\nvcrA\\tgene\\n gene copy numbers were relatively stable in DNA extracts over time, \\tO\\ntceA\\tgene\\n and \\tO\\nvcrA\\tgene\\n mRNA abundances were upregulated from undetectable levels to 2.96 \\xc3\\x97 and 6.33 \\xc3\\x97 10\\xe2\\x81\\xb4 transcripts/mL, respectively, only after exposure to TCE and lactate. While \\tO\\ntceA\\tgene\\n gene transcripts decreased over time with TCE dechlorination, the \\tO\\nvcrA\\tgene\\n gene was expressed steadily even when the concentration of vinyl chloride was at undetectable levels. In addition, ratios between catabolic and phylogenetic genes indicated that \\tO\\ntceA\\tgene\\n and \\tO\\nvcrA\\tgene\\n gene carrying organisms dechlorinated TCE and its produced daughter products, while \\tO\\nvcrA\\tgene\\n gene was mainly responsible for the dechlorination of the lower VC concentrations in a later stage of degradation.\\tO\\n\\n',\n",
              " 'a2l7vzT6_qWNbgnP8DGLDWCMEgyO-21468717.tsv': b'Evidence for taxonomic and functional drift of an atrazine-degrading culture in response to high atrazine input.\\tO\\n\\nAbstract\\tO\\n\\nWe evaluated the effects of variations in atrazine input on the evolution of a bacterial culture adapted to a low atrazine concentration. This initial culture (M3-K) was subjected to weekly subculturing in the presence of a high concentration of atrazine as the only N source (100 mg l(-1)). After four subculturing, M3-K evolved to a new bacterial culture (M3) which exhibited a significant increase in the extent of atrazine mineralization in comparison with the initial culture. Molecular analyses of M3-K and M3 cultures by cloning, restriction analysis, and sequencing of the 16S rRNA genes revealed significant differences in culture structure and composition. M3-K culture comprised mainly Actinobacteria (40%), \\xce\\xb2-Proteobacteria (26%), and Bacteroidetes (16%). After exposure to a high atrazine concentration, the dominance of Actinobacteria decreased (14%), Bacteroidetes increased (27%), and \\xce\\xb2-Proteobacteria were replaced by \\xce\\xb3-Proteobacteria (32%). Quantitative PCR revealed that the abundance of \\tO\\natzB\\tgene\\n and \\tO\\natzC\\tgene\\n genes relative to total bacteria decreased by a factor of 3-4 following the increase in atrazine concentration, while the relative abundance of \\tO\\ntrzD\\tgene\\n increased significantly (\\xe2\\x89\\x88400 times). Presented study shows that variations in atrazine input drive both functional and compositional shifts in the atrazine-degrading bacterial culture.\\tO\\n\\n',\n",
              " 'a3HbOFyvJXWX.dfWfO8dHsBvWdxi-1622259.tsv': b\"Oxidative degradation of phenanthrene by the ligninolytic fungus Phanerochaete chrysosporium.\\tO\\n\\nAbstract\\tO\\n\\nThe ligninolytic fungus Phanerochaete chrysosporium oxidized phenanthrene and phenanthrene-9,10-quinone (PQ) at their C-9 and C-10 positions to give a ring-fission product, 2,2'-diphenic acid (DPA), which was identified in chromatographic and isotope dilution experiments. DPA formation from phenanthrene was somewhat greater in low-nitrogen (ligninolytic) cultures than in high-nitrogen (nonligninolytic) cultures and did not occur in uninoculated cultures. The oxidation of PQ to DPA involved both fungal and abiotic mechanisms, was unaffected by the level of nitrogen added, and was significantly faster than the cleavage of phenanthrene to DPA. Phenanthrene-trans-9,10-dihydrodiol, which was previously shown to be the principal phenanthrene metabolite in nonligninolytic P. chrysosporium cultures, was not formed in the ligninolytic cultures employed here. These results suggest that phenanthrene degradation by ligninolytic P. chrysosporium proceeds in order from phenanthrene----PQ----DPA, involves both ligninolytic and nonligninolytic enzymes, and is not initiated by a classical microsomal cytochrome P-450. The extracellular \\tO\\nlignin peroxidase\\tenzyme\\ns of P. chrysosporium were not able to oxidize phenanthrene in vitro and therefore are also unlikely to catalyze the first step of phenanthrene degradation in vivo. Both phenanthrene and PQ were mineralized to similar extents by the fungus, which supports the intermediacy of PQ in phenanthrene degradation, but both compounds were mineralized significantly less than the structurally related \\tO\\nlignin peroxidase\\tenzyme\\n substrate pyrene was.\\tO\\n\\n\",\n",
              " 'a3OgoxXXgRa0homAO625jj3NfU2u-25487887.tsv': b'Exploring methane-oxidizing communities for the co-metabolic degradation of organic micropollutants.\\tO\\n\\nAbstract\\tO\\n\\nMethane-oxidizing cultures from five different inocula were enriched to be used for co-metabolic degradation of micropollutants. In a first screening, 18 different compounds were tested for degradation with the cultures as well as with four pure methane-oxidizing bacterial (MOB) strains. The tested compounds included pharmaceuticals, chemical additives, pesticides, and their degradation products. All enriched cultures were successful in the degradation of at least four different pollutants, but the compounds degraded most often were sulfamethoxazole (SMX) and benzotriazole (BTZ). Addition of acetylene, a specific \\tO\\nmethane monooxygenase\\tenzyme\\n (MMO) inhibitor, revealed that SMX and BTZ were mainly degraded co-metabolically by the present MOB. The pure MOB cultures exhibited less degradation potential, while SMX and BTZ were also degraded by three of the four tested pure strains. For MOB, copper (Cu(2+)) concentration is often an important factor, as several species have the ability to express a soluble MMO (sMMO) if the Cu(2+) concentration is low. In literature, this enzyme is often described to have a broader compound range for co-metabolic degradation of pollutants, in particular when it comes to aromatic structures. However, this study indicated that co-metabolic degradation of the aromatic compounds SMX and BTZ was possible at high Cu(2+) concentration, most probably catalyzed by pMMO.\\tO\\n\\n',\n",
              " 'a3Qc5NWvIJi.yuVfsskKvM2Tp4JG-24668245.tsv': b'Role of primary substrate composition on microbial community structure and function and trace organic chemical attenuation in managed aquifer recharge systems.\\tO\\n\\nAbstract\\tO\\n\\nThis study was performed to reveal the microbial community characteristics in simulated managed aquifer recharge (MAR), a natural water treatment system, under different concentrations and compositions of biodegradable dissolved organic carbon (BDOC) and further link these to the biotransformation of emerging trace organic chemicals (TOrCs). Two pairs of soil-column setups were established in the laboratory receiving synthetic feed solutions composed of different peptone/humic acid ratios and concentrations. Higher BDOC concentration resulted in lower microbial community diversity and higher relative abundance of Betaproteobacteria. Decreasing the peptone/humic acid ratio resulted in higher diversity of the community and higher relative abundances of Firmicutes, Planctomycetes, and Actinobacteria. The metabolic capabilities of microbiome involved in xenobiotics biodegradation were significantly promoted under lower BDOC concentration and higher humic acid content. \\tO\\nCytochrome P450\\tgene\\n genes were also more abundant under these primary substrate conditions. Lower peptone/humic acid ratios also promoted the attenuation of most TOrCs. These results suggest that the primary substrate characterized by a more refractory character could increase the relative abundances of Firmicutes, Planctomycetes, and Actinobacteria, as well as associated \\tO\\ncytochrome P450\\tgene\\n genes, all of which should play important roles in the biotransformation of TOrCs in this natural treatment system.\\tO\\n\\n',\n",
              " 'a3cjro1JuxuGsMW3YFdMxf9Hv7KK-21531434.tsv': b'Chlorophenols and other related derivatives of environmental concern: properties, distribution and microbial degradation processes.\\tO\\n\\nAbstract\\tO\\n\\nChlorophenols are chlorinated aromatic compound structures and are commonly found in pesticide preparations as well as industrial wastes. They are recalcitrant to biodegradation and consequently persistent in the environment. A variety of chlorophenols derivatives compounds are highly toxic, mutagenic and carcinogenic for living organisms. Biological transformation by microorganisms is one of the key remediation options that can be exploited to solve environmental pollution problems caused by these notorious compounds. The key enzymes in the microbial degradation of chlorophenols are the \\tO\\noxygenase\\tenzyme\\ns and \\tO\\ndioxygenase\\tenzyme\\ns. These enzymes can be engineered for enhanced degradation of highly chlorinated aromatic compounds through directed evolution methods. This review underscores the mechanisms of chlorophenols biodegradation with the view to understanding how bioremediation processes can be optimized for cleaning up chloroaromatic contaminated environments.\\tO\\n\\n',\n",
              " 'a4.M_gLNftxxNQ_2aZgztZhuJPl8-31071671.tsv': b'Influence of different redox conditions and dissolved organic matter on pesticide biodegradation in simulated groundwater systems.\\tO\\n\\nAbstract\\tO\\n\\nInsights into the influence of redox conditions, that is the availability of electron acceptors, and dissolved organic matter (DOM) on pesticide biodegradation in groundwater are key to understanding the environmental fate of pesticides in natural groundwater systems. Here, the influence of redox conditions and supplemental DOM addition on biodegradation of pesticides, 2,4-dichlorophenoxyacetic acid (2,4-D), 2,6-dichlorobenzamide (BAM), mecoprop-p (MCPP) and bentazone, was tested in microcosm and subsequent column experiments. Pesticide degradation, functional genes and changes in specific fractions and quantity of DOM were systematically quantified. In aerobic microcosm experiments, the highest 2,4-D degradation rate was obtained with the presence of more assimilable DOM. In column experiments, minimal pesticide degradation (\\xe2\\x89\\xa433.77%) in any anaerobic redox conditions was observed in the absence of DOM. However, in the presence of DOM, 2,4-D biodegradation was considerably enhanced under nitrate-reducing conditions (from 23.5\\xe2\\x80\\xaf\\xc2\\xb1\\xe2\\x80\\xaf10.2% to 82.3\\xe2\\x80\\xaf\\xc2\\xb1\\xe2\\x80\\xaf11.6%) and in a column without external electron acceptor amendment (from -6.3\\xe2\\x80\\xaf\\xc2\\xb1\\xe2\\x80\\xaf12.6% to 31.1\\xe2\\x80\\xaf\\xc2\\xb1\\xe2\\x80\\xaf36.3%). Observed preferential depletion of the fulvic acid fraction of DOM provides indications for specific functional DOM properties. The qPCR results show an increase in microbial biomass and functional genes (\\tO\\ntfdA\\tgene\\n) in liquid phase after DOM addition. The results of this work provide insights into the interplays among DOM, redox geochemistry, and pesticide biodegradation, and show the potential of a novel approach - DOM addition to groundwater systems - for in situ biostimulation technology to remove pesticides from groundwater systems.\\tO\\n\\n',\n",
              " 'a5YzvBm7c31lRjx7z_2nfMDygwl0-1718942.tsv': b'Molecular cloning of a Pseudomonas paucimobilis gene encoding a 17-kilodalton polypeptide that eliminates HCl molecules from gamma-hexachlorocyclohexane.\\tO\\n\\nAbstract\\tO\\n\\nPseudomonas paucimobilis UT26 is capable of growing on gamma-hexachlorocyclohexane (gamma-HCH). A genomic library of P. paucimobilis UT26 was constructed in Pseudomonas putida by using the broad-host-range cosmid vector pKS13. After 2,300 clones were screened by gas chromatography, 3 clones showing gamma-HCH degradation were detected. A 5-kb fragment from one of the cosmid clones was subcloned into pUC118, and subsequent deletion and gas chromatography-mass spectrometry analyses revealed that a fragment of ca. 500 bp was responsible for the conversion of gamma-HCH to 1,2,4-trichlorobenzene via gamma-pentachlorocyclohexene. Nucleotide sequence analysis revealed an open reading frame (\\tO\\nlinA\\tgene\\n) of 465 bp within the fragment. The nucleotide sequence of the \\tO\\nlinA\\tgene\\n gene and the deduced amino acid sequence showed no similarity to any known sequences. The product of the \\tO\\nlinA\\tgene\\n gene was 16.5 kDa according to sodium dodecyl sulfate-polyacrylamide gel electrophoresis.\\tO\\n\\n',\n",
              " 'a5jgtjZkpBoX6xpCfZU3376Kzlaq-234247.tsv': b'Initial reactions in the oxidation of naphthalene by Pseudomonas putida.\\tO\\n\\nAbstract\\tO\\n\\nA strain of Pseudomonas putida that can utilize naphthalene as its sole source of carbon and energy was isolated from soil. A mutant strain of this organism, P. putida 119, when grown on glucose in the presence of naphthalene, accumulates optically pure (+)-cis-1(R),2(S)-dihydroxy-1,2-dihydronaphthalene in the culture medium. The cis relative stereochemistry in this molecule was established by nuclear magnetic resonance spectrometry. Radiochemical trapping experiments established that this cis dihydrodiol is an intermediate in the metabolism of naphthalene by P. Fluorescens (formerly ATCC, 17483), P. putida (ATCC, 17484), and a Pseudomonas species (NCIB 9816), as well as the parent strain of P. putida described in this report. Formation of the cis dihydrodiol is catalyzed by a dioxygenase which requires either NADH or NADPH as an electron donor. A double label procedure is described for determining the origin of oxygen in the cis dihydrodiol under conditions where this metabolite would not normally accumulate. Several aromatic hydrocarbons are oxidized by cell extracts prepared from naphthalene-grown cells of P. putida. The cis dihydrodiol is converted to 1,2-dihydroxynaphthalene by an \\tO\\nNAD+-dependent dehydrogenase\\tenzyme\\n. This enzyme is specific for the (+) isomer of the dihydrodiol and shows a primary isotope effect when the dihydrodiol is substituted at C-2 with deuterium.\\tO\\n\\n',\n",
              " 'a64iRnUPdE2jbaKQELV_UNrG2Zxi-1526465.tsv': b'Reductive dechlorination of trichloroethylene by the \\tO\\nCO-reduced CO dehydrogenase\\tenzyme\\n enzyme complex from Methanosarcina thermophila.\\tO\\n\\nAbstract\\tO\\n\\nTrichloroethylene (TCE) was reductively dechlorinated to cis-dichloroethylene, trans-dichloroethylene, 1,1-dichloroethylene, vinyl chloride, and ethylene by the \\tO\\nCO-reduced CO dehydrogenase\\tenzyme\\n enzyme complex from Methanosarcina thermophila; the apparent Km and Vmax values were 1.7 +/- 0.3 mM TCE and 26.2 +/- 1.7 mol TCE dechlorinated/min/mmol factor III. Factor III also catalysed the dechlorination of TCE when in the presence of titanium(III) citrate; the apparent Km and Vmax values were 1.2 +/- 0.3 mM TCE and 34.9 +/- 3.6 mol TCE dechlorinated/min/mmol factor III. The enzyme complex was resolved into the two-subunit nickel/iron-sulfur (Ni/Fe-S) component and the two-subunit factor III-containing corrinoid/iron-sulfur (Co/Fe-S) component. The Ni/Fe-S component was unable to dechlorinate TCE in the presence of CO; however, reconstitution with the Co/Fe-S component yielded the same dechlorinated products as with the CO dehydrogenase enzyme complex.\\tO\\n\\n',\n",
              " 'a74YbCmMNjsB.SiEQcIn2iQ3Q70m-8226980.tsv': b'Purification and characterization of 2,4-dichlorophenoxyacetate/alpha-ketoglutarate dioxygenase.\\tO\\n\\nAbstract\\tO\\n\\nThe Alcaligenes eutrophus \\tO\\n2,4-dichlorophenoxyacetate/alpha-ketoglutarate dioxygenase\\tenzyme\\n, encoded by the \\tO\\ntfdA\\tgene\\n gene of plasmid pJP4, is an Fe(II)-dependent enzyme that catalyzes the conversion of 2,4-dichlorophenoxyacetate to 2,4-dichlorophenol and glyoxylate concomitant with the decomposition of alpha-ketoglutarate to form succinate and carbon dioxide (Fukumori, F., and Hausinger, R. P. (1993) J. Bacteriol. 175, 2083-2086). Using recombinant Escherichia coli cells that overexpress the \\tO\\ntfdA\\tgene\\n gene, the thermolabile enzyme (stable only up to 30 degrees C) was purified to apparent homogeneity (specific activity of 16.9 mumol of substrate converted min-1 mg of protein-1) by a simple two-step procedure. The native protein has an apparent M(r) of 50,000 +/- 2,500, consistent with a homodimeric structure. Ferrous ion is absolutely required for activity and cannot be replaced by several other divalent cations tested. Ascorbic acid stimulates dioxygenase activity and reduces the rate of enzyme inactivation by a metal ion-mediated process. The enzyme exhibits maximum activity at pH 6.5-7, however, it is stable over a pH range of 6.5-11. Although capable of hydroxylating a wide range of phenoxyacetates and related compounds, the enzyme exhibits the greatest affinity (Km 17.5 +/- 1.0 microM) and highest catalytic efficiency for 2,4-dichlorophenoxyacetate. Similarly, alpha-ketoglutarate is the preferred co-substrate (Km 3.20 +/- 0.54 microM) for the enzyme, but it can utilize a range of other alpha-ketoacids with lower efficiency. Results from chemical modification studies are consistent with the presence of multiple essential histidine residues in the enzyme.\\tO\\n\\n',\n",
              " 'a8.btHMLFHQHGdc7nUKzihjPP9y0-1592816.tsv': b'Anaerobic degradation of 2-aminobenzoic acid (anthranilic acid) via benzoyl-coenzyme A (CoA) and cyclohex-1-enecarboxyl-CoA in a denitrifying bacterium.\\tO\\n\\nAbstract\\tO\\n\\nThe enzymes catalyzing the initial reactions in the anaerobic degradation of 2-aminobenzoic acid (anthranilic acid) were studied with a denitrifying Pseudomonas sp. anaerobically grown with 2-aminobenzoate and nitrate as the sole carbon and energy sources. Cells grown on 2-aminobenzoate are simultaneously adapted to growth with benzoate, whereas cells grown on benzoate degrade 2-aminobenzoate several times less efficiently than benzoate. Evidence for a new reductive pathway of aromatic metabolism and for four enzymes catalyzing the initial steps is presented. The organism contains \\tO\\n2-aminobenzoate-coenzyme A ligase\\tenzyme\\n (\\tO\\n2-aminobenzoate-CoA ligase\\tenzyme\\n), which forms 2-aminobenzoyl-CoA. 2-Aminobenzoyl-CoA is then reductively deaminated to benzoyl-CoA by an oxygen-sensitive enzyme, 2-aminobenzoyl-CoA reductase (deaminating), which requires a low potential reductant [Ti(III)]. The specific activity is 15 nmol of 2-aminobenzoyl-CoA reduced min-1 mg-1 of protein at an optimal pH of 7. The two enzymes are induced by the substrate under anaerobic conditions only. Benzoyl-CoA is further converted in vitro by reduction with Ti(III) to six products; the same products are formed when benzoyl-CoA or 2-aminobenzoyl-CoA is incubated under reducing conditions. Two of them were identified preliminarily. One product is cyclohex-1-enecarboxyl-CoA, the other is trans-2-hydroxycyclohexane-carboxyl-CoA. The complex transformation of benzoyl-CoA is ascribed to at least two enzymes, \\tO\\nbenzoyl-CoA reductase\\tenzyme\\n (aromatic ring reducing) and \\tO\\ncyclohex-1-enecarboxyl-CoA hydratase\\tenzyme\\n. The reduction of benzoyl-CoA to alicyclic compounds is catalyzed by extracts from cells grown anaerobically on either 2-aminobenzoate or benzoate at almost the same rate (10 to 15 nmol min-1 mg-1 of protein). In contrast, extracts from cells grown anaerobically on acetate or grown aerobically on benzoate or 2-aminobenzoate are inactive. This suggests a sequential induction of the enzymes.\\tO\\n\\n',\n",
              " 'a97SQMyLZ_YUFwcnZKRlxeuttZyC-16391099.tsv': b'Slow-release inoculation allows sustained biodegradation of gamma-hexachlorocyclohexane.\\tO\\n\\nAbstract\\tO\\n\\nThis study investigated the feasibility of a slow-release inoculation approach as a bioaugmentation strategy for the degradation of lindane (gamma-hexachlorocyclohexane [gamma-HCH]). Slow-release inoculation of Sphingomonas sp. gamma 1-7 was established in both liquid and soil slurry microcosms using open-ended silicone tubes in which the bacteria are encapsulated in a protective nutrient-rich matrix. The capacity of the encapsulated cells to degrade lindane under aerobic conditions was evaluated in comparison with inoculation of free-living cells. Encapsulation of cells in tubes caused the removal of lindane by adsorption to the silicone tubes but also ensured prolonged biodegradation activity. Lindane degradation persisted 2.2 and 1.4 times longer for liquid and soil slurry microcosms, respectively, than that for inoculation with free cells. While inoculation of free-living cells led to a loss in lindane-degrading activity in limited time intervals, encapsulation in tubes allowed for a more stable actively degrading community. The loss in degrading activity was linked to the loss of the \\tO\\nlinA\\tgene\\n gene, encoding gamma-\\tO\\nHCH dehydrochlorinase\\tenzyme\\n (\\tO\\nLinA\\tgene\\n), which is involved in the initial steps of the lindane degradation pathway. This work shows that a slow-release inoculation approach using a catabolic strain encapsulated in open-ended tubes is a promising bioaugmentation tool for contaminated sites, as it can enhance pollutant removal and can prolong the degrading activity in comparison with traditional inoculation strategies.\\tO\\n\\n',\n",
              " 'a9y0qNOVEp4ztjwkaTBe1Pz6eETa-2067577.tsv': b'Structure of the detoxification catalyst \\tO\\nmercuric ion reductase\\tenzyme\\n from Bacillus sp. strain RC607.\\tO\\n\\nAbstract\\tO\\n\\nSeveral hundred million tons of toxic mercurials are dispersed in the biosphere. Microbes can detoxify organo-mercurials and mercury salts through sequential action of two enzymes, organomercury lyase and \\tO\\nmercuric ion reductase\\tenzyme\\n (\\tO\\nMerA\\tgene\\n). The latter, a homodimer with homology to the \\tO\\nFAD-dependent disulphide oxidoreductases\\tenzyme\\n, catalyses the reaction NADPH + Hg(II)----NADP+ + H+ + Hg(0), one of the very rare enzymic reactions with metal substrates. Human glutathione reductase serves as a reference molecule for \\tO\\nFAD-dependent disulphide reductases\\tenzyme\\n and between its primary structure and that of \\tO\\nMerA\\tgene\\n from Tn501 (Pseudomonas), Tn21 (Shigella), p1258 (Staphylococcus) and Bacillus, 25-30% of the residues have been conserved. All MerAs have a C-terminal extension about 15 residues long but have very varied N termini. Although the enzyme from Streptomyces lividans has no addition, from Pseudomonas aeruginosa Tn501 and Bacillus sp. strain RC607 it has one and two copies respectively of a domain of 80-85 residues, highly homologous to MerP, the periplasmic component of proteins encoded by the mer operon. These domains can be proteolytically cleaved off without changing the catalytic efficiency. We report here the crystal structure of \\tO\\nMerA\\tgene\\n from the Gram-positive bacterium Bacillus sp. strain RC607. Analysis of its complexes with nicotinamide dinucleotide substrates and the inhibitor Cd(II) reveals how limited structural changes enable an enzyme to accept as substrate what used to be a dangerous inhibitor. Knowledge of the mode of mercury ligation is a prerequisite for understanding this unique detoxification mechanism.\\tO\\n\\n',\n",
              " 'aAMflZPODkJwGSZvpxMEZCCrxhLy-31394342.tsv': b'LDPE microplastic films alter microbial community composition and enzymatic activities in soil.\\tO\\n\\nAbstract\\tO\\n\\nConcerns regarding microplastic contamination have spread from aquatic environments to terrestrial systems with a growing number of studies have been reported. Notwithstanding, the potential effects on soil ecosystems remain largely unexplored. In this study, the effects of polyethylene microplastics on soil enzymatic activities and the bacterial community were evaluated, and the microbiota colonizing on microplastics were also investigated. Microplastic amendment (2000 fragments per kg soil) significantly increased the \\tO\\nurease\\tenzyme\\n and \\tO\\ncatalase\\tenzyme\\n activities in soil after 15 days, and no discernible alteration of \\tO\\ninvertase\\tenzyme\\n activities was detected. Results from high-throughput sequencing of 16S rRNA revealed that the alpha diversities (richness, evenness, and diversity) of the microbiota in soil were not obviously changed by the PE amendment, whereas the diversity indexes of microbiota on plastic fragments were significantly lower than those in the control and amended soils. Different taxonomic composition was observed in between the control and amended soils after 90 days of incubation. Bacterial assemblages with distinct community structure colonized the PE microplastics. Additionally, several taxa including plastic-degrading bacteria and pathogens were more abundant on microplastics. Simultaneously, the predicted functional profiles showed that the pathways of amino acid metabolism and xenobiotics biodegradation and metabolism were higher on the microplastics. These results indicated that microplastics in soil, compared with those in aquatic environments, can also act as a distinct microbial habitat, potentially altering the ecological functions of soil ecosystems.\\tO\\n\\n',\n",
              " 'aAR7COUYPwWwooLSEEo7umBWUrEC-22003008.tsv': b'A novel hydrolase identified by genomic-proteomic analysis of phenylurea herbicide mineralization by Variovorax sp. strain SRS16.\\tO\\n\\nAbstract\\tO\\n\\nThe soil bacterial isolate Variovorax sp. strain SRS16 mineralizes the phenylurea herbicide linuron. The proposed pathway initiates with hydrolysis of linuron to 3,4-dichloroaniline (DCA) and N,O-dimethylhydroxylamine, followed by conversion of DCA to Krebs cycle intermediates. Differential proteomic analysis showed a linuron-dependent upregulation of several enzymes that fit into this pathway, including an \\tO\\namidase\\tenzyme\\n (\\tO\\nLibA\\tgene\\n), a multicomponent chloroaniline dioxygenase, and enzymes associated with a modified chlorocatechol ortho-cleavage pathway. Purified \\tO\\nLibA\\tgene\\n is a monomeric \\tO\\nlinuron hydrolase\\tenzyme\\n of \\xe2\\x88\\xbc55 kDa with a K(m) and a V(max) for linuron of 5.8 \\xce\\xbcM and 0.16 nmol min\\xe2\\x81\\xbb\\xc2\\xb9, respectively. This novel member of the \\tO\\namidase\\tenzyme\\n signature family is unrelated to phenylurea-hydrolyzing enzymes from Gram-positive bacteria and lacks activity toward other tested phenylurea herbicides. Orthologues of \\tO\\nlibA\\tgene\\n are present in all other tested linuron-degrading Variovorax strains with the exception of Variovorax strains WDL1 and PBS-H4, suggesting divergent evolution of the linuron catabolic pathway in different Variovorax strains. The organization of the linuron degradation genes identified in the draft SRS16 genome sequence indicates that gene patchwork assembly is at the origin of the pathway. Transcription analysis suggests that a catabolic intermediate, rather than linuron itself, acts as effector in activation of the pathway. Our study provides the first report on the genetic organization of a bacterial pathway for complete mineralization of a phenylurea herbicide and the first report on a \\tO\\nlinuron hydrolase\\tenzyme\\n in Gram-negative bacteria.\\tO\\n\\n',\n",
              " 'aAjTJkYkdurOuzdNS2EqvP8DTTfS-1447132.tsv': b'Resolution and some properties of enzymes involved in enantioselective transformation of 1,3-dichloro-2-propanol to (R)-3-chloro-1,2-propanediol by Corynebacterium sp. strain N-1074.\\tO\\n\\nAbstract\\tO\\n\\nDuring the course of the transformation of 1,3-dichloro-2-propanol (DCP) into (R)-3-chloro-1,2-propanediol [(R)-MCP] with the cell extract of Corynebacterium sp. strain N-1074, epichlorohydrin (ECH) was transiently formed. The cell extract was fractionated into two DCP-dechlorinating activities (fractions Ia and Ib) and two ECH-hydrolyzing activities (fractions IIa and IIb) by TSKgel DEAE-5PW column chromatography. Fractions Ia and Ib catalyzed the interconversion of DCP to ECH, and fractions IIa and IIb catalyzed the transformation of ECH into MCP. Fractions Ia and IIa showed only low enantioselectivity for each reaction, whereas fractions Ib and IIb exhibited considerable enantioselectivity, yielding R-rich ECH and MCP, respectively. Enzymes Ia and Ib were isolated from fractions Ia and Ib, respectively. Enzyme Ia had a molecular mass of about 108 kDa and consisted of four subunits identical in molecular mass (about 28 kDa). Enzyme Ib was a protein of 115 kDa, composed of two different polypeptides (about 35 and 32 kDa). The specific activity of enzyme Ib for DCP was about 30-fold higher than that of enzyme Ia. Both enzymes catalyzed the transformation of several halohydrins into the corresponding epoxides with liberation of halides and its reverse reaction. Their substrate specificities and immunological properties differed from each other. Enzyme Ia seemed to be \\tO\\nhalohydrin hydrogen-halide-lyase\\tenzyme\\n which was already purified from Escherichia coli carrying a gene from Corynebacterium sp. strain N-1074.\\tO\\n\\n',\n",
              " 'aArRcX1.yw2C779LPKZEQgkPFd5O-1961727.tsv': b'Ring fission of anthracene by a eukaryote.\\tO\\n\\nAbstract\\tO\\n\\nLigninolytic fungi are unique among eukaryotes in their ability to degrade polycyclic aromatic hydrocarbons (PAHs), but the mechanism for this process is unknown. Although certain PAHs are oxidized in vitro by the fungal \\tO\\nlignin peroxidase\\tenzyme\\ns (LiPs) that catalyze ligninolysis, it has never been shown that LiPs initiate PAH degradation in vivo. To address these problems, the metabolism of anthracene (AC) and its in vitro oxidation product, 9,10-anthraquinone (AQ), was examined by chromatographic and isotope dilution techniques in Phanerochaete chrysosporium. The fungal oxidation of AC to AQ was rapid, and both AC and AQ were significantly mineralized. Both compounds were cleaved by the fungus to give the same ring-fission metabolite, phthalic acid, and phthalate production from AQ was shown to occur only under ligninolytic culture conditions. These results show that the major pathway for AC degradation in Phanerochaete proceeds AC----AQ----phthalate + CO2 and that it is probably mediated by LiPs and other enzymes of ligninolytic metabolism.\\tO\\n\\n',\n",
              " 'aAvpUplV__m4CU1IcWvDMWWuHY0y-18606539.tsv': b'alpha-L-Arabinofuranosidase\\tenzyme\\n from Streptomyces sp. PC22: purification, characterization and its synergistic action with xylanolytic enzymes in the degradation of xylan and agricultural residues.\\tO\\n\\nAbstract\\tO\\n\\nalpha-l-Arabinofuranosidase\\tenzyme\\n was purified from culture filtrates of the thermoalkaliphilic Streptomyces sp. PC22 to about 108-fold purity by (NH(4))(2)SO(4) precipitation followed by column chromatography. Its approximate molecular weight was 404kDa, with a subunit mass of approximately 79kDa. The evaluated K(m) and V(max) values with p-nitrophenyl-alpha-l-arabinofuranoside as substrate were 0.23mM and 124 U.mg(-1), respectively. The purified enzyme was optimally active at 65 degrees C and pH 6.0 and showed a mild but significant synergistic effect in combination with other xylanolytic enzymes, including \\tO\\nxylanase\\tenzyme\\n, \\tO\\nbeta-xylosidase\\tenzyme\\n and \\tO\\nacetyl esterase\\tenzyme\\n, on the degradation of oat-spelt xylan, corn cob and corn husk substrates with a 1.25, 1.32 and 1.21-fold increase in the amount of reducing sugar released, respectively, compared to the expected (additive) amounts for the individual enzymes acting alone. Sequential reactions using two xylan-backbone degrading enzymes (\\tO\\nxylanase\\tenzyme\\n/\\tO\\nbeta-xylosidase\\tenzyme\\n) and two debranching enzymes (\\tO\\nalpha-l-arabinofuranosidase\\tenzyme\\n/\\tO\\nacetyl esterase\\tenzyme\\n) were also determined. The highest degree of synergy was obtained in sequential reactions with the debranching enzyme digestion preceding the xylan-backbone degrading enzymes.\\tO\\n\\n',\n",
              " 'aBN1Qmxk3Xj19lwi8__Xx43FTPcO-29563.tsv': b'Phthalate and 4-hydroxyphthalate metabolism in Pseudomonas testosteroni: purification and properties of \\tO\\n4,5-dihydroxyphthalate decarboxylase\\tenzyme\\n.\\tO\\n\\nAbstract\\tO\\n\\nPhthalate is degraded through 4,5-dihydroxyphthalate and protocatechuate in Pseudomonas testosteroni NH1000. The ezyme \\tO\\n4,5-dihydroxyphthalate decarboxylase\\tenzyme\\n, catalyzing the conversion of 4,5-dihydroxyphthalate to protocatechuate and carbon dioxide, was purified approximately 130-fold from phthalate-induced cells of a protocatechuate 4,5-dioxygenase-deficient mutant of P. testosteroni. The most purified preparation showed a single protein band on sodium dodecyl sulfate-acrylamide disc gel electrophoresis with a molecular weight of 38,000. The apparent molecular weight of the native enzyme determined by Sephadex G-200 column chromatography was 150,000. Among the substrate analogs tested, only 4-hydroxyphthalate served as a substrate, which was decarboxylated to form m-hydroxybenzoate. The apparent Km values for 4,5-dihydroxyphthalate and 4-hydroxyphthalate were estimated to be 10.5 micrometer and 1.25 mM, respectively, and the Vmax for the former was 10 times larger than that for the latter. Whereas the wild-type strain could utilize 4-hydroxyphthalate as a sole source of carbon, none of the following could grow with the compound: \\tO\\n4,5-dihydroxyphthalate decarboxylase\\tenzyme\\n-deficient, m-hydroxybenzoate-nondegradable, and protocatechuate 4,5-dioxygenase-deficient mutants. Since one-step revertants of these mutants could utilize 4-hydroxyphthalate, the compound appears to be metabolized through m-hydroxybenzoate and protocatechuate in P. testosteroni NH1000.\\tO\\n\\n',\n",
              " 'aBP4ogv6rktF6czR15ipvZka18oS-1885512.tsv': b'Cloning and characterization of a Pseudomonas mendocina KR1 gene cluster encoding \\tO\\ntoluene-4-monooxygenase\\tenzyme\\n.\\tO\\n\\nAbstract\\tO\\n\\nPseudomonas mendocina KR1 metabolizes toluene as a carbon source by a previously unknown pathway. The initial step of the pathway is hydroxylation of toluene to form p-cresol by a multicomponent \\tO\\ntoluene-4-monooxygenase\\tenzyme\\n (T4MO) system. The T4MO enzyme system has broad substrate specificity and provides a new opportunity for biodegradation of toxic compounds and bioconversions. Its known activities include conversion of a variety of phenyl compounds into the phenolic derivatives and the complete degradation of trichloroethylene. We have cloned and characterized a gene cluster from KR1 that determines the offO activity. To clone the T4MO genes, KR1 DNA libraries were constructed in Escherichia coli HB101 by using a broad-host-range vector and transferred to a KR1 mutant able to grow on p-cresol but not on toluene. An insert consisting of two SacI fragments of identical size (10.2 kb) was shown to complement the mutant for growth on toluene. One of the SacI fragments, when cloned into the E. coli vector pUC19, was found to direct the synthesis of indigo dye. The indigo-forming property was correlated with the presence of T4MO activity. The T4MO genes were mapped to a 3.6-kb region, and the direction of transcription was determined. DNA sequencing and N-terminal amino acid determination identified a five-gene cluster, \\tO\\ntmoABCDE\\tgene\\n, within this region. Expression of this cluster carrying a single mutation in each gene demonstrated that each of the five genes is essential for T4MO activity. Other evidence presented indicated that none of the tmo genes was involved in the regulation of the tmo gene cluster, in the control of substrate transport for the T4MO system, or in major processing of the products of the tmo genes. It was tentatively concluded that the \\tO\\ntmoABCDE\\tgene\\n genes encode structural polypeptides of the T4MO enzyme system. One of the tmo genes was tentatively identified as a ferredoxin gene.\\tO\\n\\n',\n",
              " 'aBRIckae0Mpac9CxsT6pSNnkHqf0-26776948.tsv': b'A novel \\tO\\narsenic methyltransferase\\tenzyme\\n gene of Westerdykella aurantiaca isolated from arsenic contaminated soil: phylogenetic, physiological, and biochemical studies and its role in arsenic bioremediation.\\tO\\n\\nAbstract\\tO\\n\\nElevated arsenic concentration in the environment and agricultural soil is a serious concern to crop production and human health. Among different detoxification mechanisms, the methylation of arsenic is a widespread phenomenon in nature. A number of microorganisms are able to methylate arsenic, but less is known about the arsenic metabolism in fungi. We identified a novel \\tO\\narsenic methyltransferase\\tenzyme\\n (\\tO\\nWaarsM\\tenzyme\\n) gene from a soil fungus, Westerdykella aurantiaca. \\tO\\nWaarsM\\tenzyme\\n showed sequence homology with all known \\tO\\narsenic methyltransferase\\tenzyme\\ns having three conserved SAM binding motifs. The expression of \\tO\\nWaarsM\\tenzyme\\n enhanced arsenic resistance in E. coli (\\xce\\x94ars) and S. cerevisiae (\\xce\\x94acr2) strains by biomethylation and required endogenous reductants, preferably GSH, for methyltransferase activity. The purified \\tO\\nWaarsM\\tenzyme\\n catalyzes the production of methylated arsenicals from both AsIII and AsV, and also displays AsV reductase activity. It displayed higher methyltransferase activity and lower KM 0.1945 \\xc2\\xb1 0.021 mM and KM 0.4034 \\xc2\\xb1 0.078 mM for AsIII and AsV, respectively. S. cerevisiae (\\xce\\x94acr2) cells expressing \\tO\\nWaarsM\\tenzyme\\n produced 2.2 ppm volatile arsenic and 0.64 ppm DMA(v) with 0.58 ppm volatile arsenicals when exposed to 20 ppm AsV and 2 ppm AsIII, respectively. Arsenic tolerance in rice after co-culture with genetically engineered yeast suggested its potential role in arsenic bioremediation. Thus, characterization of \\tO\\nWaarsM\\tenzyme\\n provides a potential strategy to reduce arsenic concentration in soil with reduced arsenic accumulation in crops grown in arsenic contaminated areas, and thereby alleviating human health risks.\\tO\\n\\n',\n",
              " 'aBgzC9zRQ5gI.RRbvzcyxs3v.erG-24123738.tsv': b'The novel bacterial N-demethylase PdmAB is responsible for the initial step of N,N-dimethyl-substituted phenylurea herbicide degradation.\\tO\\n\\nAbstract\\tO\\n\\nThe environmental fate of phenylurea herbicides has received considerable attention in recent decades. The microbial metabolism of N,N-dimethyl-substituted phenylurea herbicides can generally be initiated by mono-N-demethylation. In this study, the molecular basis for this process was revealed. The \\tO\\npdmAB\\tgene\\n genes in Sphingobium sp. strain YBL2 were shown to be responsible for the initial mono-N-demethylation of commonly used N,N-dimethyl-substituted phenylurea herbicides. \\tO\\nPdmAB\\tgene\\n is the oxygenase component of a bacterial Rieske non-heme iron oxygenase (RO) system. The genes \\tO\\npdmAB\\tgene\\n, encoding the \\xce\\xb1 subunit \\tO\\nPdmA\\tgene\\n and the \\xce\\xb2 subunit \\tO\\nPdmB\\tgene\\n, are organized in a transposable element flanked by two direct repeats of an insertion element resembling ISRh1. Furthermore, this transposable element is highly conserved among phenylurea herbicide-degrading sphingomonads originating from different areas of the world. However, there was no evidence of a gene for an electron carrier (a ferredoxin or a reductase) located in the immediate vicinity of \\tO\\npdmAB\\tgene\\n. Without its cognate electron transport components, expression of \\tO\\nPdmAB\\tgene\\n in Escherichia coli, Pseudomonas putida, and other sphingomonads resulted in a functional enzyme. Moreover, coexpression of a putative [3Fe-4S]-type ferredoxin from Sphingomonas sp. strain RW1 greatly enhanced the catalytic activity of \\tO\\nPdmAB\\tgene\\n in E. coli. These data suggested that \\tO\\nPdmAB\\tgene\\n has a low specificity for electron transport components and that its optimal ferredoxin may be the [3Fe-4S] type. \\tO\\nPdmA\\tgene\\n exhibited low homology to the \\xce\\xb1 subunits of previously characterized ROs (less than 37% identity) and did not cluster with the RO group involved in O- or N-demethylation reactions, indicating that \\tO\\nPdmAB\\tgene\\n is a distinct bacterial RO \\tO\\nN-demethylase\\tenzyme\\n.\\tO\\n\\n',\n",
              " 'aBlYzaPGSQp3xU8t_IxrAGMCyvrC-23454590.tsv': b'Discharge of landfill leachate to streambed sediments impacts the mineralization potential of phenoxy acid herbicides depending on the initial abundance of \\tO\\ntfdA\\tgene\\n gene classes.\\tO\\n\\nAbstract\\tO\\n\\nTo understand the role of abundance of \\tO\\ntfdA\\tgene\\n gene classes belonging to \\xce\\xb2- and \\xce\\xb3-proteobacteria on phenoxy acid herbicide degradation, streambed sediments were sampled around three seepage meters (SMs) installed in a landfill-impacted groundwater-surface water interface. Highest herbicide mass discharge to SM3, and lower herbicide mass discharges to SM1 and SM2 were determined due to groundwater discharge rates and herbicide concentrations. SM1-sediment with the lowest abundance of \\tO\\ntfdA\\tgene\\n gene classes had the slowest mineralization, whereas SM2- and SM3-sediments with more abundant \\tO\\ntfdA\\tgene\\n genes had faster mineralization. The observed difference in mineralization rates between discharge zones was simulated by a Monod-based kinetic model, which confirmed the role of abundance of \\tO\\ntfdA\\tgene\\n gene classes. This study suggests presence of specific degraders adapted to slow growth rate and high yield strategy due to long-term herbicide exposure; and thus groundwater-surface water interface could act as a natural biological filter and protect stream water quality.\\tO\\n\\n',\n",
              " 'aCL5B7lodNpiF9VFK56lfoXwR6jK-26308673.tsv': b\"Identification of the \\tO\\nAmidase\\tenzyme\\n \\tO\\nBbdA\\tgene\\n That Initiates Biodegradation of the Groundwater Micropollutant 2,6-dichlorobenzamide (BAM) in Aminobacter sp. MSH1.\\tO\\n\\nAbstract\\tO\\n\\n2,6-dichlorobenzamide (BAM) is a recalcitrant groundwater micropollutant that poses a major problem for drinking water production in European countries. Aminobacter sp. MSH1 and related strains have the unique ability to mineralize BAM at micropollutant concentrations but no information exists on the genetics of BAM biodegradation. An \\tO\\namidase\\tenzyme\\n-\\tO\\nBbdA\\tgene\\n-converting BAM to 2,6-dichlorobenzoic acid (DCBA) was purified from Aminobacter sp. MSH1. Heterologous expression of the corresponding \\tO\\nbbdA\\tgene\\n gene and its absence in MSH1 mutants defective in BAM degradation, confirmed its BAM degrading function. \\tO\\nBbdA\\tgene\\n shows low amino acid sequence identity with reported amidases and is encoded by an IncP1-\\xce\\xb2 plasmid (pBAM1, 40.6 kb) that lacks several genes for conjugation. \\tO\\nBbdA\\tgene\\n has a remarkably low KM for BAM (0.71 \\xce\\xbcM) and also shows activity against benzamide and ortho-chlorobenzamide (OBAM). Differential proteomics and transcriptional reporter analysis suggest the constitutive expression of \\tO\\nbbdA\\tgene\\n in MSH1. Also in other BAM mineralizing Aminobacter sp. strains, \\tO\\nbbdA\\tgene\\n and pBAM1 appear to be involved in BAM degradation. \\tO\\nBbdA\\tgene\\n's high affinity for BAM and its constitutive expression are of interest for using strain MSH1 in treatment of groundwater containing micropollutant concentrations of BAM for drinking water production.\\tO\\n\\n\",\n",
              " 'aCdU83YXibjOzSJ8u3BvmyF344eS-31445334.tsv': b'Exogenous Zn2+ enhance the biodegradation of atrazine by regulating the chlorohydrolase gene \\tO\\ntrzN\\tgene\\n transcription and membrane permeability of the degrader Arthrobacter sp. DNS10.\\tO\\n\\nAbstract\\tO\\n\\nEnhancing the biodegradation efficiency of atrazine, a kind of commonly applied herbicide, has been attracted much more concern. Here, Zn2+ which has long been considered essential in adjusting cell physiological status was selected to investigate its role on the biodegradation of atrazine by Arthrobacter sp. DNS10 as well as the transmembrane transport of atrazine during the biodegradation period. The results of gas chromatography showed that the atrazine removal percentages (initial concentration was 100\\xe2\\x80\\xafmg\\xe2\\x80\\xafL-1) in 0.05\\xe2\\x80\\xafmM Zn2+ and 1.0\\xe2\\x80\\xafmM Zn2+ treatments were 94.42% and 86.02% respectively at 48\\xe2\\x80\\xafh, while there was also 66.43% of atrazine left in the treatment without exogenous Zn2+ existence. The expression of \\tO\\natrazine chlorohydrolase\\tenzyme\\n gene \\tO\\ntrzN\\tgene\\n in the strain DNS10 cultured with 0.05\\xe2\\x80\\xafmM and 1.0\\xe2\\x80\\xafmM Zn2+ was 7.30- and 4.67- times respectively compared with that of the non-zinc treatment. In addition, the flow cytometry test suggests that 0.05\\xe2\\x80\\xafmM of Zn2+ could better adjust the membrane permeability of strain DNS10, meanwhile, the amount of atrazine accumulation in the strain DNS10 co-cultured with this level Zn2+ was 2.21 times of that of the strain without Zn2+. This study may facilitate a better understanding of the mechanisms that exogenous Zn2+ enhances the biodegradation of atrazine by Arthrobacter sp. DNS10.\\tO\\n\\n',\n",
              " 'aD3qB7nGh9V6zv33OQXsumxeJ7OC-1254564.tsv': b'Metabolism of resorcinylic compounds by bacteria: orcinol pathway in Pseudomonas putida.\\tO\\n\\nAbstract\\tO\\n\\nEnrichment cultures yielded two strains of Pseudomonas putida capable of growth with orcinol (3,5-dihydroxytoluene) as the sole source of carbon. Experiments with cell suspensions and cell extracts indicate that orcinol is metabolized by hydroxylation of the benzene ring followed successively by ring cleavage and hydrolyses to give 2 mol of acetate and 1 mol of pyruvate per mol of orcinol as shown: orcinol leads to 2,3,5-trihydroxytoluene leads to 2,4,6-trioxoheptanoate leads to acetate + acetylpyruvate leads to acetate + pyruvate. Evidence for this pathway is based on: (i) high respiratory activities of orcinol-grown cells towards 2,3,5-trihydroxytoluene; (ii) transient accumulation of a quinone, probably 2-hydroxy-6-methyl-1,4-benzoquinone, during grouth with orcinol; (iii) formation of pyruvate and acetate from orcinol, 2,3,5-trihydroxytoluene, and acetylpyruvate catalyzed by extracts of orcinol, but not by succinate-grown cells; (iv) characterization of the product of oxidation of 3-methylcatechol (an analogue of 2,3,5-trihydroxytoluene) showing that oxygenative cleavage occurs between carbons bearing methyl and hydroxyl substituents; (v) transient appearance of a compound having spectral properties similar to those of acetylpyruvate during 2,3,5-trihydroxytoluene oxidation by extracts of orcinol-grown cells. \\tO\\nOrcinol hydroxylase\\tenzyme\\n exhibits catalytic activity when resorcinol or m-cresol is substituted for orcinol; hydroxyquinol and 3-methylcatechol are substrates for the ring cleavage enzyme \\tO\\n2,3,5-trihydroxytoluene-1,2-oxygenase\\tenzyme\\n. The enzymes of this pathway are induced by growth with orcinol but not with glucose or succinate.\\tO\\n\\n',\n",
              " 'aEHh.KmFDmnx4LqVJgZmN2.IU97y-22854895.tsv': b\"Response of a diuron-degrading community to diuron exposure assessed by real-time quantitative PCR monitoring of phenylurea hydrolase A and B encoding genes.\\tO\\n\\nAbstract\\tO\\n\\nA real-time quantitative PCR method was developed to detect and quantify phenlylurea hydrolase genes' (\\tO\\npuhA\\tgene\\n and \\tO\\npuhB\\tgene\\n) sequences from environmental DNA samples to assess diuron-degrading genetic potential in some soil and sediment microbial communities. In the soil communities, mineralization rates (determined with [ring-\\xc2\\xb9\\xe2\\x81\\xb4C]-labeled diuron) were linked to diuron-degrading genetic potentials estimated from \\tO\\npuhB\\tgene\\n number copies, which increased following repeated diuron treatments. In the sediment communities, mineralization potential did not depend solely on the quantity of \\tO\\npuhB\\tgene\\n copies, underlining the need to assess gene expression. In the sediment samples, both \\tO\\npuhB\\tgene\\n copy numbers and mineralization capacities were highly conditioned by whether or not diuron-treated soil was added. This points to transfers of degradative potential from soils to sediments. No \\tO\\npuhA\\tgene\\n gene was detected in soil and sediment DNA extracts. Moreover, some sediments exhibited high diuron mineralization potential even though \\tO\\npuhB\\tgene\\n genes were not detected, suggesting the existence of alternative diuron degradation pathways.\\tO\\n\\n\",\n",
              " 'aFQ5GrPy0N9DBLuocsDuRxtTrpWq-1327782.tsv': b'Purification and characterisation of the NADH:acceptor reductase component of \\tO\\nxylene monooxygenase\\tenzyme\\n encoded by the TOL plasmid pWW0 of Pseudomonas putida mt-2.\\tO\\n\\nAbstract\\tO\\n\\nThe \\tO\\nxylene monooxygenase\\tenzyme\\n system encoded by the TOL plasmid pWW0 of Pseudomonas putida catalyses the hydroxylation of a methyl side-chain of toluene and xylenes. Genetic studies have suggested that this monooxygenase consists of two different proteins, products of the \\tO\\nxylA\\tgene\\n and \\tO\\nxylM\\tgene\\n genes, which function as an electron-transfer protein and a terminal hydroxylase, respectively. In this study, the electron-transfer component of \\tO\\nxylene monooxygenase\\tenzyme\\n, the product of \\tO\\nxylA\\tgene\\n, was purified to homogeneity. Fractions containing the \\tO\\nxylA\\tgene\\n gene product were identified by its NADH:cytochrome c reductase activity. The molecular mass of the enzyme was determined to be 40 kDa by SDS/PAGE, and 42 kDa by gel filtration. The enzyme was found to contain 1 mol/mol of tightly but not covalently bound FAD, as well as 2 mol/mol of non-haem iron and 2 mol/mol of acid-labile sulfide, suggesting the presence of two redox centers, one FAD and one [2Fe-2S] cluster/protein molecule. The oxidised form of the protein had absorbance maxima at 457 nm and 390 nm, with shoulders at 350 nm and 550 nm. These absorbance maxima disappeared upon reduction of the protein by NADH or dithionite. The NADH:acceptor reductase was capable of reducing either one- or two-electron acceptors, such as horse heart cytochrome c or 2,6-dichloroindophenol, at an optimal pH of 8.5. The reductase was found to have a Km value for NADH of 22 microM. The oxidation of NADH was determined to be stereospecific; the enzyme is pro-R (class A enzyme). The titration of the reductase with NADH or dithionite yielded three distinct reduced forms of the enzyme: the reduction of the [2Fe-2S] center occurred with a midpoint redox potential of -171 mV; and the reduction of FAD to FAD. (semiquinone form), with a calculated midpoint redox potential of -244 mV. The reduction of FAD. to FAD.. (dihydroquinone form), the last stage of the titration, occurred with a midpoint redox potential of -297 mV. The [2Fe-2S] center could be removed from the protein by treatment with an excess of mersalyl acid. The [2Fe-2S]-depleted protein was still reduced by NADH, giving rise to the formation of the anionic flavin semiquinone observed in the native enzyme, thus suggesting that the electron flow was NADH --> FAD --> [2Fe-2S] in this reductase. The resulting protein could no longer reduce cytochrome c, but could reduce 2,6-dichloroindophenol at a reduced rate.\\tO\\n\\n',\n",
              " 'aFqlwZAcl3EaVCR._itfw0Bg7ecG-2394680.tsv': b'Enzymatic formation, stability, and spontaneous reactions of 4-fluoromuconolactone, a metabolite of the bacterial degradation of 4-fluorobenzoate.\\tO\\n\\nAbstract\\tO\\n\\nEnzymatic conversion of 4-fluorocatechol in the simultaneous presence of partially purified preparations of \\tO\\ncatechol 1,2-dioxygenase\\tenzyme\\n from Pseudomonas cepacia and muconate cycloisomerase from Alcaligenes eutrophus 335 yielded a product that was unambiguously identified as (+)-4-fluoromuconolactone [(+)-4-carboxymethyl-4-fluoro-but-2-en-4-olide]. This compound was shown to be the only major product formed from 3-fluoro-cis,cis-muconate by the action of \\tO\\nmuconate cycloisomerases\\tenzyme\\n from A. eutrophus 335, A. eutrophus JMP134, and P. cepacia as well as by the action of \\tO\\ndichloromuconate cycloisomerase\\tenzyme\\n from A. eutrophus JMP134. This finding implies that \\tO\\ndichloromuconate cycloisomerase\\tenzyme\\n, like the \\tO\\nmuconate cycloisomerases\\tenzyme\\n, catalyzes primarily a cycloisomerization reaction, which only in the case of chloro- and bromo-substituted substrates is connected to a dehalogenation. 4-Fluoromuconolactone at pH 7 decomposes by spontaneous reactions mainly to maleylacetate, which then decarboxylates to give cis-acetylacrylate. Although significant amounts of an unidentified compound are also formed from the fluorolactone, HF elimination to the two isomeric dienelactones (4-carboxymethylenebut-2-en-4-olides) is negligible. However, all spontaneous reactions proceed so slowly that an enzymatic conversion of 4-fluoromuconolactone must be assumed. Participation of \\tO\\ndienelactone hydrolases\\tenzyme\\n in this reaction is indicated by their induction during growth of various strains with 4-fluorobenzoate. However, experiments with cell extracts of P. putida A3.12 suggest that at least one other hydrolytic enzyme is able to contribute to 4-fluoromuconolactone conversion. In light of these observations, earlier proposals for a 4-fluorobenzoate degradative pathway are discussed.\\tO\\n\\n',\n",
              " 'aGGjZft5cKAxMFYpf3D1CHm4eTP4-925012.tsv': b'A novel \\tO\\nketone monooxygenase\\tenzyme\\n from Pseudomonas cepacia. Purification and properties.\\tO\\n\\nAbstract\\tO\\n\\nA \\tO\\nketone monooxygenase\\tenzyme\\n was purified from cells of Pseudomonas cepacia grown on 2-tridecanone as sole carbon source. Enzyme stability is maintained by the addition of ethanol, EDTA, and dithiothreitol. Stoichiometric studies show that for 1 mol of undecyl acetate formed, 1 mol of O2 is consumed and 1 mol of NADPH is oxidized. The monooxygenase, purified to homogeneity, has a molecular weight of approximately 123,000 and consists of two equal subunits with molecular weights of 55,000. The enzyme contains FAD and exhibits absorption maxima at 375 and 488 nm. Enzyme activity is inhibited by thiol-active reagents and the inhibition by the cations, cadmium, copper, zinc, and mercury, is reversed by dithiothreitol, indicating the presence of essential sulfhydryl groups. Substrate specificity tests show that acetate esters are formed from methyl ketones from C-7 through C-14. The oxygenase is also active on isomers of 2-tridecanone forming esters from 3- through 7-tridecanone. With 6-tridecanone, two esters are formed, heptyl hexanoate and pentyl octanoate, indicating that oxygen is inserted on either side of the carbonyl group. In addition, the enzyme catalyzes the lactonization of the cyclic ketone, cyclopentanone, with the formation of 5-valerolactone.\\tO\\n\\n',\n",
              " 'aHUtck9wjaHujHIYEMKTrb8lB3zu-8005.tsv': b'Pathways of microbial metabolism of parathion.\\tO\\n\\nAbstract\\tO\\n\\nA mixed bacterial culture, consisting of a minimum of nine isolates, was adapted to growth on technical parathion (PAR) as a sole carbon and energy source. The primary oxidative pathway for PAR metabolism involved an initial hydrolysis to yield diethylthiophosphoric acid and p-nitrophenol. A secondary oxidative pathway involved the oxidation of PAR to paraoxon and then hydrolysis to yield p-nitrophenol and diethylphosphoric acid. Under low oxgen conditions PAR was reduced via a third pathway to p-aminoparathion and subsequently hydrolyzed to p-aminophenol and diethylthiophosphoric acid. \\tO\\nPAR hydrolase\\tenzyme\\n, an enzyme produced by an isolate from the mixed culture, rapidly hydrolyzed PAR and paraoxon (6.0 mumol/mg per min). This enzyme was inducible and stable at room temperature and retained 100% of its activity when heated for 55 C for 10 min.\\tO\\n\\n',\n",
              " 'aHg0m7EJoE7Lqw1a3K8dzuvFIwIu-23560662.tsv': b'Succession of bacterial and fungal 4-chloro-2-methylphenoxyacetic acid degraders at the soil-litter interface.\\tO\\n\\nAbstract\\tO\\n\\nPhenoxyacetic acids can be degraded by diverse soil microorganisms. Nevertheless, we miss information about the succession of 4-chloro-2-methylphenoxyacetic acid (MCPA) degraders in micro-environments of soils as well as specific functions of different microbial groups during MCPA degradation. We studied MCPA degradation at the soil-litter interface in a microcosm experiment and followed the succession of different degrader populations by quantifying the abundance of 16S rRNA genes as well as, the fungal ITS fragment and the functional genes \\tO\\ntfdA\\tgene\\n (in total and divided into three classes) and \\tO\\ncadA\\tgene\\n. Adjacent to the litter layer, a dynamic depletion zone of MCPA indicated that the litter effect on MCPA degradation depends on substrate availability and the affected soil volume. The increase of the \\tO\\ntfdA\\tgene\\n class III and \\tO\\ncadA\\tgene\\n genes was linked to MCPA mineralisation. Total abundance of \\tO\\ntfdA\\tgene\\n genes was dominated by class I MCPA degraders and did not reflect MCPA degradation potential of the soil. Litter addition induced the development of pioneer and late-stage fungal communities, which were probably both involved in MCPA degradation. The results underline the importance of the ecological behaviour of different degrader populations for the understanding of herbicide degradation in soils.\\tO\\n\\n',\n",
              " 'aHlTwI8ihDgYtvTxoIMATAE97Y10-23076592.tsv': b'Evolution of atrazine-degrading capabilities in the environment.\\tO\\n\\nAbstract\\tO\\n\\nSince their first introduction in the mid 1950s, man-made s-triazine herbicides such as atrazine have extensively been used in agriculture to control broadleaf weed growth in different crops, and thus contributed to improving crop yield and quality. Atrazine is the most widely used s-triazine herbicide for the control of weeds in crops such as corn and sorghum. Although atrazine was initially found to be slowly and partially biodegradable, predominantly by nonspecific \\tO\\nP450 monoxygenase\\tenzyme\\ns which do not sustain microbial growth, microorganisms gradually evolved as a result of repeated exposure, started using it as a growth substrate and eventually succeeded in mineralizing it. Within three decades, an entirely new hydrolase-dependent pathway for atrazine mineralization emerged and rapidly spread worldwide among genetically different bacteria. This review focuses on the enzymes involved in atrazine mineralization and their evolutionary histories, the genetic composition of microbial populations involved in atrazine degradation and the biotechnologies that have been developed, based on these systems, for the bioremediation of atrazine contamination in the environment.\\tO\\n\\n',\n",
              " 'aJU0mGpHnbYRcwLiK45Pyiv8kPBS-24239819.tsv': b'Metagenomic analysis reveals potential biodegradation pathways of persistent pesticides in freshwater and marine sediments.\\tO\\n\\nAbstract\\tO\\n\\nThe abundance and diversity of biodegradation genes (BDGs) and potential degradation pathways of dichlorodiphenyltrichloroethane (DDT), hexachlorocyclohexane (HCH), and atrazine (ATZ) in freshwater and marine sediments were investigated by metagenomic analysis using 6 datasets (16Gb in total). The datasets were derived using Illumina high-throughput sequencing and were based on BLAST against self-established databases of BDGs, DDT degradation genes (DDGs), HCH degradation genes (HDGs), and ATZ degradation genes (ADGs). The results showed that the abundance and diversity of BDGs, DDGs, HDGs, and ADGs varied with sample source and locations. The \\tO\\nlip\\tgene\\n and \\tO\\nmnp\\tgene\\n genes, which encode for \\tO\\nperoxidase\\tenzyme\\n, and the \\tO\\ncarA\\tgene\\n gene, which encodes for \\tO\\nlaccase\\tenzyme\\n, were detected as the dominant genes for degradation of organic pollutants. The \\tO\\nhdt\\tgene\\n, \\tO\\nhdg\\tgene\\n, and \\tO\\natzB\\tgene\\n genes, which encode for \\tO\\nhydratase\\tenzyme\\n, \\tO\\ndehalogenase\\tenzyme\\n, and \\tO\\nethylaminohydrolase\\tenzyme\\n, were found to be the most abundant genes involved in DDT, HCH, and ATZ degradation, respectively. The identified 69 genera capable of degrading organic pollutants were mostly affiliated with Proteobacteria (49.3%) and Actinobacteria (21.7%). Four genera, including Plesiocystis, Anaerolinea, Jannaschia, and Mycobacterium, were the major biodegradation populations in all sediments. In this study, the nearly complete biodegradation pathways of DDT and ATZ were found, and the partial degradation pathway of HCH was detected in all sediments.\\tO\\n\\n',\n",
              " 'aKEgV7EueJsNLXQ4_VvYw9dxQ7yu-1453953.tsv': b'DNA sequence determination and functional characterization of the OCT-plasmid-encoded alkJKL genes of Pseudomonas oleovorans.\\tO\\n\\nAbstract\\tO\\n\\nThe \\tO\\nalkBFGHJKL\\tgene\\n and \\tO\\nalkST\\tgene\\n operons encode enzymes that allow Pseudomonas putida (oleovorans) to metabolize alkanes. In this paper we report the nucleotide sequence of a 4592 bp region of the \\tO\\nalkBFGHJKL\\tgene\\n operon encoding the \\tO\\nAlkJ\\tgene\\n, \\tO\\nAlkK\\tgene\\n and \\tO\\nAlkL\\tgene\\n polypeptides. The \\tO\\nalkJ\\tgene\\n gene encodes a protein of 59 kilodaltons. The predicted amino acid sequence shows significant homology with four flavin proteins: \\tO\\ncholine dehydrogenase\\tenzyme\\n, a \\tO\\nglucose dehydrogenase\\tenzyme\\n and two oxidases. \\tO\\nAlkJ\\tgene\\n is membrane-bound and converts aliphatic medium-chain-length alcohols into aldehydes. The properties of \\tO\\nAlkJ\\tgene\\n suggest that it is linked to the electron transfer chain. \\tO\\nAlkJ\\tgene\\n is necessary for growth on alkanes only in P. putida \\tO\\nalcohol dehydrogenase\\tenzyme\\n (\\tO\\nAlcA\\tgene\\n) mutants. \\tO\\nAlkK\\tgene\\n is homologous to a range of proteins which act by an ATP-dependent covalent binding of AMP to their substrate. This list includes the acetate, coumarate and long-chain fatty acid CoA ligases. The \\tO\\nalkK\\tgene\\n gene complements a fadD mutation in Escherichia coli, which shows that it indeed encodes an \\tO\\nacyl-CoA synthetase\\tenzyme\\n. \\tO\\nAlkK\\tgene\\n is a 60 kilodalton protein located in the cytoplasm. \\tO\\nAlkL\\tgene\\n is homologous to OmpW, a Vibrio cholerae outer membrane protein of unknown function, and a hypothetical polypeptide encoded by ytt4 in E. coli. \\tO\\nAlkL\\tgene\\n, OmpW and Ytt4 all have a signal peptide and end with a sequence characteristic of outer membrane proteins. The \\tO\\nalkL\\tgene\\n gene product was found in the outer membrane of E. coli W3110 containing the alk-genes. The \\tO\\nalkL\\tgene\\n gene can be deleted without a clear effect on growth rate. Its function remains unknown. The G+C content of the alkJKL genes is 45%, identical to that of the alkBFGH genes, and significantly lower than the G+C content of the OCT-plasmid and the P. putida chromosome.\\tO\\n\\n',\n",
              " 'aLqTZF8guoL3.vLWZwhCJz03WWyG-22307296.tsv': b'Dynamics of the \\tO\\nlinuron hydrolase\\tenzyme\\n \\tO\\nlibA\\tgene\\n gene pool size in response to linuron application and environmental perturbations in agricultural soil and on-farm biopurification systems.\\tO\\n\\nAbstract\\tO\\n\\nlibA\\tgene\\n, a gene encoding a novel type of \\tO\\nlinuron hydrolase\\tenzyme\\n, was recently identified in the linuron-mineralizing Variovorax sp. strain SRS16. In order to assess the contribution of \\tO\\nlibA\\tgene\\n to linuron degradation in environmental settings, \\tO\\nlibA\\tgene\\n abundance was monitored in response to the application of linuron and to environmental perturbations in agricultural soil microcosms and microcosms simulating the matrix of on-farm biopurification systems. \\tO\\nlibA\\tgene\\n numbers were measured by real-time PCR and linked to reported data of Variovorax community composition and linuron mineralization capacity. In the soil microcosms and one biopurification system setup, \\tO\\nlibA\\tgene\\n numbers responded to the application of linuron and environmental changes in congruency with the modulation of linuron mineralization capacity and the occurrence of a particular Variovorax phylotype (phylotype A). However, in another biopurification system setup, no such correlations were found. Our data suggest that in the simulated environmental settings, the occurrence of \\tO\\nlibA\\tgene\\n can be linked to the linuron mineralization capacity and that \\tO\\nlibA\\tgene\\n is primarily hosted by Variovorax phylotype A strains. However, the results also suggest that, apart from \\tO\\nlibA\\tgene\\n, other, as-yet-unknown isofunctional genes play an important role in linuron mineralization in the environment.\\tO\\n\\n',\n",
              " 'aM2.ASbdUakhKQsFMULm0m.241eu-24898178.tsv': b'16S rRNA gene phylogeny and tfdA gene analysis of 2,4-D-degrading bacteria isolated in China.\\tO\\n\\nAbstract\\tO\\n\\nTwenty-two 2,4-dichlorophenoxyacetic acid (2,4-D)-degrading bacterial isolates were collected from agricultural soils at three sites in China. Sequence analysis of the 16S rRNA genes indicated that the isolates were phylogenetically grouped into four categories: Ochrobactrum anthropi, in the Alpha- class of the phylum Proteobacteria (3 out of 22 isolates), Cupriavidus sp., of the Betaproteobacteria (3 out of 22), Pseudomonas sp. and Stenotrophomonas sp., which are Gammaproteobacteria (7 out of 22), and Bacillus sp., of the phylum Firmicutes (9 out of 22). Primers were designed to amplify the conserved domain of \\tO\\ntfdA\\tgene\\n, which is known to be involved in the degradation of 2,4-D. Results showed that the \\tO\\ntfdA\\tgene\\n genes of all 22 strains were most similar to that of Cupriavidus necator JMP134, which belongs to the 2,4-D/\\xce\\xb1-ketoglutarate dioxygenase \\tO\\nTfdA\\tgene\\n protein family, indicating that the JMP134-type \\tO\\ntfdA\\tgene\\n gene is likely to be almost universal among the 2,4-D-degrading bacteria isolated from China. Degradation abilities of these 22 strains were investigated in assays using 2,4-D as the sole source of carbon and energy. Thirteen strains degraded >60 % of the available 2,4-D (500 mg l(-1)) over a 1-week incubation period, while a further nine Bacillus sp. strains degraded 50-81 % of the available 2,4-D. None of these nine strains degraded other selected herbicides, such as mecoprop, 2-methyl-4-chlorophenoxyacetic acid, quizalofop, and fluroxypyr. This is the first report of 2,4-D-degradation by Bacilli.\\tO\\n\\n',\n",
              " 'aM4A2nEttHUsrp616fh2R6Qrk.ru-1268229.tsv': b'Trimethylamine dehydrogenase\\tenzyme\\n from a methylotrophic bacterium. I. Isolation and steady-state kinetics.\\tO\\n\\nAbstract\\tO\\n\\n1. The isolation of \\tO\\ntrimethylamine dehydrogenase\\tenzyme\\n (EC 1.5.99.7) from a restricted facultative methylotroph to electrophoretic homogeneity is described. 2. The molecular weight and subunit molecular weights were found to be 146800 for the enzyme by sedimentation equilibrium ultracentrifugation and 70000-80000 for the two non-identical subunits by sodium dodecyl sulphate gel electrophoresis. 3. Initial velocity studies indicate that the enzymatic reaction proceeds by a Ping-Pong mechanism. 4. Further kinetic evidence was obtained by analysis of product inhibition patterns using the alternate substrate diethylamine and the products acetaldehyde and ethylamine as product inhibitors, for the release of ethylamine before the addition of phenazine methosulphate and for the existence of an enzyme-two-carbon unit complex as a stable form of the enzyme. 5. Some properties of the unusual prosthetic group of \\tO\\ntrimethylamine dehydrogenase\\tenzyme\\n and its photodegradation product are described in preliminary form.\\tO\\n\\n',\n",
              " 'aMWcBtBI2Z2a0njuP81Tr3mX6I0m-28321487.tsv': b'Benzene degradation in a denitrifying biofilm reactor: activity and microbial community composition.\\tO\\n\\nAbstract\\tO\\n\\nBenzene is an aromatic compound and harmful for the environment. Biodegradation of benzene can reduce the toxicological risk after accidental or controlled release of this chemical in the environment. In this study, we further characterized an anaerobic continuous biofilm culture grown for more than 14 years on benzene with nitrate as electron acceptor. We determined steady state degradation rates, microbial community composition dynamics in the biofilm, and the initial anaerobic benzene degradation reactions. Benzene was degraded at a rate of 0.15 \\xce\\xbcmol/mg protein/day and a first-order rate constant of 3.04/day which was fourfold higher than rates reported previously. Bacteria belonging to the Peptococcaceae were found to play an important role in this anaerobic benzene-degrading biofilm culture, but also members of the Anaerolineaceae were predicted to be involved in benzene degradation or benzene metabolite degradation based on Illumina MiSeq analysis of 16S ribosomal RNA genes. Biomass retention in the reactor using a filtration finger resulted in reduction of benzene degradation capacity. Detection of the benzene carboxylase encoding gene, \\tO\\nabcA\\tgene\\n, and benzoic acid in the culture vessel indicated that benzene degradation proceeds through an initial carboxylation step.\\tO\\n\\n',\n",
              " 'aMecN20D4rUvYkTsLUdOC2E6wagm-27984820.tsv': b'Comparative proteomics reveal the mechanism of Tween80 enhanced phenanthrene biodegradation by Sphingomonas sp. GY2B.\\tO\\n\\nAbstract\\tO\\n\\nPrevious study concerning the effects of surfactants on phenanthrene biodegradation focused on observing the changes of cell characteristics of Sphingomonas sp. GY2B. However, the impact of surfactants on the expression of bacterial proteins, controlling phenanthrene transport and catabolism, remains obscure. To overcome the knowledge gap, comparative proteomic approaches were used to investigate protein expressions of Sphingomonas sp. GY2B during phenanthrene biodegradation in the presence and absence of a nonionic surfactant, Tween80. A total of 23 up-regulated and 19 down-regulated proteins were detected upon Tween80 treatment. Tween80 could regulate ion transport (e.g. H+) in cell membrane to provide driving force (ATP) for the transmembrane transport of phenanthrene thus increasing its uptake and biodegradation by GY2B. Moreover, Tween80 probably increased GY2B vitality and growth by inducing the expression of \\tO\\npeptidylprolyl isomerase\\tenzyme\\n to stabilize cell membrane, increasing the abundances of proteins involved in intracellular metabolic pathways (e.g. TCA cycle), as well as decreasing the abundances of translation/transcription-related proteins and cysteine desulfurase, thereby facilitating phenanthrene biodegradation. This study may facilitate a better understanding of the mechanisms that regulate surfactants-enhanced biodegradation of PAHs at the proteomic level.\\tO\\n\\n',\n",
              " 'aNUC2kQ9JHXhzRelFXkqKr8nCllq-1577754.tsv': b'A \\tO\\nthiocyanate hydrolase\\tenzyme\\n of Thiobacillus thioparus. A novel enzyme catalyzing the formation of carbonyl sulfide from thiocyanate.\\tO\\n\\nAbstract\\tO\\n\\nA \\tO\\nthiocyanate hydrolase\\tenzyme\\n that catalyzes the first step in thiocyanate degradation was purified to homogeneity from Thiobacillus thioparus, an obligate chemolithotrophic eubacterium metabolizing thiocyanate to sulfate as an energy source. The \\tO\\nthiocyanate hydrolase\\tenzyme\\n was purified 52-fold by steps involving ammonium sulfate precipitation, DEAE-Sephacel column chromatography, and hydroxylapatite column chromatography. The enzyme hydrolyzed 1 mol of thiocyanate to form 1 mol of carbonyl sulfide and 1 mol of ammonia as follows: SCN- + 2H2O----COS + NH3 + OH-. This is the first report describing the hydrolysis of thiocyanate to carbonyl sulfide by an enzyme. The enzyme had a molecular mass of 126 kDa and was composed of three different subunits: alpha (19 kDa), beta (23 kDa), and gamma (32 kDa). The enzyme exhibited optimal activities at pH 7.5-8.0 and at temperatures ranging from 30 to 40 degrees C. The Km value for thiocyanate was approximately 11 mM. Immunoblot analysis with polyclonal antibodies against the purified enzyme suggested that it was induced in T. thioparus cells when the cells were grown with thiocyanate.\\tO\\n\\n',\n",
              " 'aNYIc2VJb1lvg_2sFDoCarzYacLS-17117994.tsv': b'2,4-D impact on bacterial communities, and the activity and genetic potential of 2,4-D degrading communities in soil.\\tO\\n\\nAbstract\\tO\\n\\nThe key role of telluric microorganisms in pesticide degradation is well recognized but the possible relationships between the biodiversity of soil microbial communities and their functions still remain poorly documented. If microorganisms influence the fate of pesticides, pesticide application may reciprocally affect soil microorganisms. The objective of our work was to estimate the impact of 2,4-D application on the genetic structure of bacterial communities and the 2,4-D-degrading genetic potential in relation to 2,4-D mineralization. Experiments combined isotope measurements with molecular analyses. The impact of 2,4-D on soil bacterial populations was followed with ribosomal intergenic spacer analysis. The 2,4-D degrading genetic potential was estimated by real-time PCR targeted on \\tO\\ntfdA\\tgene\\n sequences coding an enzyme specifically involved in 2,4-D mineralization. The genetic structure of bacterial communities was significantly modified in response to 2,4-D application, but only during the intense phase of 2,4-D biodegradation. This effect disappeared 7 days after the treatment. The 2,4-D degrading genetic potential increased rapidly following 2,4-D application. There was a concomitant increase between the \\tO\\ntfdA\\tgene\\n copy number and the 14C microbial biomass. The maximum of \\tO\\ntfdA\\tgene\\n sequences corresponded to the maximum rate of 2,4-D mineralization. In this soil, 2,4-D degrading microbial communities seem preferentially to use the tfd pathway to degrade 2,4-D.\\tO\\n\\n',\n",
              " 'aNtTIxCXZIoCbW2sw7pg6qma8hbK-15883866.tsv': b'Molecular analysis of bacterial community based on 16S rDNA and functional genes in activated sludge enriched with 2,4-dichlorophenoxyacetic acid (2,4-D) under different cultural conditions.\\tO\\n\\nAbstract\\tO\\n\\nDifferential emergence and diversity of bacterial communities from activated sludge in response to varied cultural conditions using 2,4-dichlorophenoxyacetic acid (2,4-D) were investigated by coupling molecular analyses based on 16S rDNA with functional genes. We employed three different cultural conditions: (1) a culture sequentially fed a high concentration (300 mg/L) of 2,4-D (HS); (2) a culture continuously fed a low concentration (10 mg/L) of 2,4-D (LC); and (3) a serial batch culture in which 1% (v/v) of culture was transferred to a fresh medium containing a high concentration (300 mg/L) of 2,4-D (HB). The HS and LC bioreactors were operated for 3 months and HB was repeatedly transferred for 1 month. The 2,4-D was stably degraded under all the cultural conditions tested. PCR amplification and cloning-based analysis of functional genes using community DNAs from the cultures revealed five different \\tO\\noxygenase\\tenzyme\\n genes that may be involved in the initial step of 2,4-D degradation. All five gene-types were present in HS, while one of the five genes, type V (\\tO\\ntftA\\tgene\\n) was not detected in LC. Quantitative PCR analysis showed that in HS, Ralstonia eutropha JMP 134 type-\\tO\\ntfdA4\\tgene\\n (type I) was the most abundant in copy number (2.0 +/- 0.1 x 10(7) copies/microg DNA) followed by RASC type-\\tO\\ntfdA\\tgene\\n (type II) (1.8 +/- 1.0 x 10(6) copies/microg DNA), putative \\tO\\ncadA\\tgene\\n-like gene (type IV) (2.6 +/- 0.8 x 10(5) copies/microg DNA), \\tO\\ncadA\\tgene\\n gene (type III) (1.3 +/- 1.0 x 10(4) copies/microg DNA), and \\tO\\ntftA\\tgene\\n gene (type V) (3.5 +/- 1.1 x 10(3) copies/microg DNA). Similar results were obtained in LC. In contrast, HB contained only type I and type III genes, and the type I gene was five orders of magnitude greater in copy number than the type III gene. Denaturing gel gradient electrophoresis (DGGE) analysis of PCR, amplified 16S rDNA fragments of bacterial communities in the three different cultures showed low similarity coefficient values (< or =0.35) when compared to the original activated sludge, suggesting that 2,4-D amendment caused a drastic change in the bacterial community. Particularly, HB showed only six bands (16-18 bands in the other cultures) and very low similarity coefficient values when compared to the other communities (0.10 to HS, 0.17 to LC, and 0.0 to original sludge). These results indicated that serial batch culturing (HB) resulted in a phylogenetically limited number of 2,4-D degrading bacteria carrying limited catabolic genes whereas more diverse 2,4-D degraders and catabolic genes were present in HS and LC. Therefore, the approach used for monitoring should be taken into account when one evaluates the population dynamics of contaminant-degrading bacteria at bioremediation sites.\\tO\\n\\n',\n",
              " 'aO33b5hse6bt4xZVSrrcQgUo9jO4-28006730.tsv': b'Enhancement of microbial 2,4,6-trinitrotoluene transformation with increased toxicity by exogenous nutrient amendment.\\tO\\n\\nAbstract\\tO\\n\\nIn this study, the bacterial strain Citrobacter youngae strain E4 was isolated from 2,4,6-trinitrotoluene (TNT)-contaminated soil and used to assess the capacity of TNT transformation with/without exogenous nutrient amendments. C. youngae E4 poorly degraded TNT without an exogenous amino nitrogen source, whereas the addition of an amino nitrogen source considerably increased the efficacy of TNT transformation in a dose-dependent manner. The enhanced TNT transformation of C. youngae E4 was mediated by increased cell growth and up-regulation of \\tO\\nTNT nitroreductase\\tenzyme\\ns, including NemA, NfsA and NfsB. This result indicates that the increase in TNT transformation by C. youngae E4 via nitrogen nutrient stimulation is a cometabolism process. Consistently, TNT transformation was effectively enhanced when C. youngae E4 was subjected to a TNT-contaminated soil slurry in the presence of an exogenous amino nitrogen amendment. Thus, effective enhancement of TNT transformation via the coordinated inoculation of the nutrient-responsive C. youngae E4 and an exogenous nitrogen amendment might be applicable for the remediation of TNT-contaminated soil. Although the TNT transformation was significantly enhanced by C. youngae E4 in concert with biostimulation, the 96-h LC50 value of the TNT transformation product mixture on the aquatic invertebrate Tigriopus japonicas was higher than the LC50 value of TNT alone. Our results suggest that exogenous nutrient amendment can enhance microbial TNT transformation; however, additional detoxification processes may be needed due to the increased toxicity after reduced TNT transformation.\\tO\\n\\n',\n",
              " 'aPjCyemv4fUpmXpk7VoVwjyBDs_u-1371658.tsv': b'Degradation of 2-methylbenzoic acid by Pseudomonas cepacia MB2.\\tO\\n\\nAbstract\\tO\\n\\nWe report the isolation of Pseudomonas cepacia MB2, believed to be the first microorganism to utilize 2-methylbenzoic acid as the sole carbon source. Its growth range included all mono- and dimethylbenzoates (with the exception of 2,5- and 2,6-dimethylbenzoates) and 3-chloro-2-methylbenzoate (but not 4- or 5-chloro-2-methylbenzoate) but not chlorobenzoates lacking a methyl group. 2-Chlorobenzoate, 3-chlorobenzoate, and 2,3-, 2,4-, and 3,4-dichlorobenzoates inhibited growth of MB2 on 2-methylbenzoate as a result of cometabolism to the corresponding chlorinated catechols which blocked the key enzyme \\tO\\ncatechol 2,3-dioxygenase\\tenzyme\\n. A \\tO\\nmetapyrocatechase\\tenzyme\\n-negative mutant, MB2-G5, showed accumulation of dimethylcatechols from 2,3- and 3,4-dimethylbenzoates, and phenols were detected in resting-cell transformation extracts bearing the same substitution pattern as the original substrate, presumably following thermal degradation of the intermediate dihydrodiol. 2-Methylphenol was also found in extracts of the mutant cells with 2-methylbenzoate. These observations suggested a major route of methylbenzoate metabolism to be dioxygenation to a carboxy-hydrodiol which then forms a catechol derivative. In addition, the methyl group of 2-methylbenzoate was oxidized to isobenzofuranone (by cells of MB2-G5) and to phthalate (by cells of a separate mutant that could not utilize phthalate, MB2-D2). This pathway also generated a chlorinated isobenzofuranone from 3-chloro-2-methylbenzoate.\\tO\\n\\n',\n",
              " 'aS4kJlp8QNdWM003oQeKhHPXfZ34-16329946.tsv': b'Cooperative catabolic pathways within an atrazine-degrading enrichment culture isolated from soil.\\tO\\n\\nAbstract\\tO\\n\\nAtrazine degradation previously has been shown to be carried out by individual bacterial species or by relatively simple consortia that have been isolated using enrichment cultures. Here, the degradative pathway for atrazine was examined for a complex 8-membered enrichment culture. The species composition of the culture was determined by PCR-DGGE. The bacterial species included Agrobacterium tumefaciens, Caulobacter crescentus, Pseudomonas putida, Sphingomonas yaniokuyae, Nocardia sp., Rhizobium sp., Flavobacterium oryzihabitans, and Variovorax paradoxus. All of the isolates were screened for the presence of known genes that function for atrazine degradation including \\tO\\natzA\\tgene\\n,-B,-C,-D,-E,-F and \\tO\\ntrzD\\tgene\\n,-N. Dechlorination of atrazine, which was obligatory for complete mineralization, was carried out exclusively by Nocardia sp., which contained the trzN gene. Following dechlorination, the resulting product, hydroxyatrazine was further degraded via two separate pathways. In one pathway Nocardia converted hydroxyatrazine to N-ethylammelide via an unidentified gene product. In the second pathway, hydroxyatrazine generated by Nocardia sp. was hydrolyzed to N-isopropylammelide by Rhizobium sp., which contained the atzB gene. Each member of the enrichment culture contained \\tO\\natzC\\tgene\\n, which is responsible for ring cleavage, but none of the isolates carried the \\tO\\natzD\\tgene\\n,-E, or -F genes. Each member further contained either \\tO\\ntrzD\\tgene\\n or exhibited urease activity. The enrichment culture was destabilized by loss of Nocardia sp. when grown on ethylamine, ethylammelide, and cyanuric acid, after which the consortium was no longer able to degrade atrazine. The analysis of this enrichment culture highlights the broad level bacterial community interactions that may be involved in atrazine degradation in nature.\\tO\\n\\n',\n",
              " 'aSnQ6wHh4xWTKzoJ8bT4HhzeOVPy-23586778.tsv': b'In situ response of the linuron degradation potential to linuron application in an agricultural field.\\tO\\n\\nAbstract\\tO\\n\\nTo assess the involvement of the genus Variovorax and the \\tO\\nlinuron hydrolase\\tenzyme\\n gene \\tO\\nlibA\\tgene\\n in in situ linuron degradation in agricultural fields, changes in Variovorax community size and composition, in \\tO\\nlibA\\tgene\\n abundance and in linuron mineralization capacity were monitored in field soil plots either treated or not with a linuron-containing herbicide mixture. Changes in Variovorax community composition, due to the proliferation of a hereto unknown Variovorax phylotype D, and increases in \\tO\\nlibA\\tgene\\n numbers occurred concomitant to increases in linuron mineralization capacity in the plot treated with the herbicide mixture. The observations suggest that Variovorax and \\tO\\nlibA\\tgene\\n proliferated as a response to linuron and hence their contribution to in situ linuron degradation. The involvement of Variovorax phylotype D and \\tO\\nlibA\\tgene\\n in linuron degradation in the examined soil was supported by laboratory soil microcosm experiments. Attempts to enrich in suspended cultures and isolate the organism corresponding to phylotype D from the soil were unsuccessful as the enrichment resulted in replacement of Variovorax phylotype D by other Variovorax phylotypes. This illustrates that linuron-degrading strains isolated by liquid enrichment cultures are not always representatives of those responsive to linuron in the field, although the genus specificity of linuron degradation was retained.\\tO\\n\\n',\n",
              " 'aTjaxECCPKlNaQUPN9wzGhuTosMG-20305027.tsv': b'Abundance and expression of enantioselective rdpA and sdpA dioxygenase genes during degradation of the racemic herbicide (R,S)-2-(2,4-dichlorophenoxy)propionate in soil.\\tO\\n\\nAbstract\\tO\\n\\nThe \\tO\\nrdpA\\tgene\\n and \\tO\\nsdpA\\tgene\\n genes encode two enantioselective \\tO\\nalpha-ketoglutarate-dependent dioxygenases\\tenzyme\\n catalyzing the initial step of microbial degradation of the chiral herbicide (R,S)-2-(2,4-dichlorophenoxy)propionate (R,S-dichlorprop). Primers were designed to assess abundance and transcription dynamics of \\tO\\nrdpA\\tgene\\n and \\tO\\nsdpA\\tgene\\n genes in a natural agricultural soil. No indigenous \\tO\\nrdpA\\tgene\\n genes were detected, but \\tO\\nsdpA\\tgene\\n genes were present at levels of approximately 10(3) copies g of soil(-1). Cloning and sequencing of partial \\tO\\nsdpA\\tgene\\n genes revealed a high diversity within the natural \\tO\\nsdpA\\tgene\\n gene pool that could be divided into four clusters by phylogenetic analysis. BLASTp analysis of deduced amino acids revealed that members of cluster I shared 68 to 69% identity, cluster II shared 78 to 85% identity, cluster III shared 58 to 64% identity, and cluster IV shared 55% identity to their closest \\tO\\nSdpA\\tgene\\n relative in GenBank. Expression of \\tO\\nrdpA\\tgene\\n and \\tO\\nsdpA\\tgene\\n in Delftia acidovorans MC1 inoculated in soil was monitored by reverse transcription quantitative real-time PCR (qPCR) during in situ degradation of 2 and 50 mg kg(-1) of (R,S)-dichlorprop. (R,S)-Dichlorprop amendment created a clear upregulation of both \\tO\\nrdpA\\tgene\\n and \\tO\\nsdpA\\tgene\\n gene expression during the active phase of (14)C-labeled (R,S)-dichlorprop mineralization, particularly following the second dose of 50 mg kg(-1) herbicide. Expression of both genes was maintained at a low constitutive level in nonamended soil microcosms. This study is the first to report the presence of indigenous \\tO\\nsdpA\\tgene\\n genes recovered directly from natural soil and also comprises the first investigation into the transcription dynamics of two enantioselective dioxygenase genes during the in situ degradation of the herbicide (R,S)-dichlorprop in soil.\\tO\\n\\n',\n",
              " 'aU.B2YOcXQQ0c__tKQlZmVhxL9.W-24862007.tsv': b'Bacteria induced degradation of fluoranthene in minimal salt medium mediated by catabolic enzymes in vitro condition.\\tO\\n\\nAbstract\\tO\\n\\nFluoranthene is highly toxic and ubiquitous in the environment. A study on degradation of 200 ppm of fluoranthene in MSM by two bacterial strains Pseudomonas aeruginosa PSA5, Rhodococcus sp. NJ2 and their consortium revealed that fluoranthene was degraded 74% by Rhodococcus sp. NJ2, 61% by Pseudomonas sp. PSA5 and 97% by their consortium. Higher degradation in the consortium may be attributed to synergistic action between two bacteria. It was also observed that several degradative enzymes \\tO\\ncatechol 1,2 dioxygenase\\tenzyme\\n, \\tO\\ncatechol 2,3 dioxygenase\\tenzyme\\n, \\tO\\nprotocatechuate 2,3 dioxygenase\\tenzyme\\n, \\tO\\nprotocatechuate 3,4 dioxygenase\\tenzyme\\n, \\tO\\nprotocatechuate 4,5 dioxygenase\\tenzyme\\n, \\tO\\nsalicylate hydroxylase\\tenzyme\\n and \\tO\\n2-carboxybenzaldehyde dehydrogenase\\tenzyme\\n were differentially induced at different stages of fluoranthene degradation. Biodegradation kinetics indicated half life period of fluoranthene degradation. Besides, glycolipid, as a biosurfactant, was induced to facilitate the degradation process. Hence, both the bacteria may be used individually or in combination for effective decontamination of oil and sludge contaminated soil.\\tO\\n\\n',\n",
              " 'aV17kDPU_zg1fV8gxsBjgBRUOw_W-1853203.tsv': b'Novel enzymic hydrolytic dehalogenation of a chlorinated aromatic.\\tO\\n\\nAbstract\\tO\\n\\nMicrobial enzyme systems may be used in the biodegradation of persistent environmental pollutants. The three polypeptide components of one such system, the \\tO\\n4-chlorobenzoate dehalogenase\\tenzyme\\n system, have been isolated, and the chemical steps of the 4-hydroxybenzoate-forming reaction that they catalyze have been identified. The genes contained within a 4.5-kilobase Pseudomonas sp. strain CBS3 chromosomal DNA fragment that encode dehalogenase activity were selectively expressed in transformed Escherichia coli. Oligonucleotide sequencing revealed a stretch of homology between the 57-kilodalton (kD) polypeptide and several magnesium adenosine triphosphate (MgATP)-cleaving enzymes that allowed MgATP and coenzyme A (CoA) to be identified as the dehalogenase cosubstrate and cofactor, respectively. The dehalogenase activity arises from two components, a 4-chlorobenzoate:\\tO\\nCoA ligase-dehalogenase\\tenzyme\\n (an alpha beta dimer of the 57- and 30-kD polypeptides) and a \\tO\\nthioesterase\\tenzyme\\n (the 16-kD polypeptide).\\tO\\n\\n',\n",
              " 'aVCsGoaq69gyqXoQPrYR_Xh_8YHy-19497847.tsv': b'Differential membrane proteome analysis reveals novel proteins involved in the degradation of aromatic compounds in Geobacter metallireducens.\\tO\\n\\nAbstract\\tO\\n\\nAromatic compounds comprise a large class of natural and man-made compounds, many of which are of considerable concern for the environment and human health. In aromatic compound-degrading anaerobic bacteria the central intermediate of aromatic catabolism, benzoyl coenzyme A, is attacked by dearomatizing \\tO\\nbenzoyl-CoA reductase\\tenzyme\\ns (BCRs). An ATP-dependent BCR has been characterized in facultative anaerobes. In contrast, a previous analysis of the soluble proteome from the obligately anaerobic model organism Geobacter metallireducens identified genes putatively coding for a completely different dearomatizing BCR. The corresponding BamBCDEFGHI complex is predicted to comprise soluble molybdenum or tungsten, selenocysteine, and FeS cluster-containing components. To elucidate key processes involved in the degradation of aromatic compounds in obligately anaerobic bacteria, differential membrane protein abundance levels from G. metallireducens grown on benzoate and acetate were determined by the MS-based spectral counting approach. A total of 931 proteins were identified by combining one-dimensional sodium dodecyl sulfate-polyacrylamide gel electrophoresis with liquid chromatography-tandem mass spectrometry. Several membrane-associated proteins involved in the degradation of aromatic compounds were newly identified including proteins with similarities to modules of NiFe/heme b-containing and energy-converting hydrogenases, \\tO\\ncytochrome bd oxidase\\tenzyme\\ns, \\tO\\ndissimilatory nitrate reductase\\tenzyme\\ns, and a tungstate ATP-binding cassette transporter system. The transcriptional regulation of differentially expressed genes was analyzed by quantitative reverse transcription-PCR; in addition benzoate-induced in vitro activities of hydrogenase and nitrate reductase were determined. The results obtained provide novel insights into the poorly understood degradation of aromatic compounds in obligately anaerobic bacteria.\\tO\\n\\n',\n",
              " 'aVDY7Ev59m70WiI7f_a67hScGBiS-2034671.tsv': b'Isolation and properties of a \\tO\\nnitrile hydratase\\tenzyme\\n from the soil fungus Myrothecium verrucaria that is highly specific for the fertilizer cyanamide and cloning of its gene.\\tO\\n\\nAbstract\\tO\\n\\nA protein was purified from crude extracts of the soil fungus Myrothecium verrucaria by gel filtration and hydrophobic chromatography to homogeneity; this protein catalyzed the stoichiometric hydration of the fertilizer cyanamide to urea with high substrate specificity. This cyanamide hydratase (urea hydro-lyase; EC 4.2.1.69) contained zinc and consisted of six identical subunits with Mr = 27,700. It was partially sequenced. The protein was detectable only when the fungus was grown on cyanamide as the sole nitrogen source. Genomic DNA from the fungus was cloned, and the gene encoding the enzyme was mapped with an oligonucleotide probe derived from the amino acid sequence within a 25,800-base-pair DNA region. The subunit of the enzyme is encoded by a 795-base-pair DNA sequence containing a 63-base-pair intron. A cDNA clone containing the intronless gene with an open reading frame encoding a sequence of 244 amino acids expressed the enzyme in active form in Escherichia coli with excellent yield.\\tO\\n\\n',\n",
              " 'aVXBoAqrkI_nbpmM_mu.Br9fKHzC-19251892.tsv': b'TaqMan probe-based real-time PCR assay for detection and discrimination of class I, II, and III \\tO\\ntfdA\\tgene\\n genes in soils treated with phenoxy acid herbicides.\\tO\\n\\nAbstract\\tO\\n\\nSeparate quantification of three classes of \\tO\\ntfdA\\tgene\\n genes was performed using TaqMan quantitative real-time PCR for 13 different soils subsequent to mineralization of three phenoxy acids. Class III \\tO\\ntfdA\\tgene\\n genes were found to be involved in mineralization more often than class I and II \\tO\\ntfdA\\tgene\\n genes.\\tO\\n\\n',\n",
              " 'aVlBjfk8CtLMfZEIJbKUN7dxO2DW-1892384.tsv': b'Mutants of Pseudomonas cepacia G4 defective in catabolism of aromatic compounds and trichloroethylene.\\tO\\n\\nAbstract\\tO\\n\\nPseudomonas cepacia G4 possesses a novel pathway of toluene catabolism that is shown to be responsible for the degradation of trichloroethylene (TCE). This pathway involves conversion of toluene via o-cresol to 3-methylcatechol. In order to determine the enzyme of toluene degradation that is responsible for TCE degradation, chemically induced mutants, blocked in the toluene \\tO\\northo-monooxygenase\\tenzyme\\n (TOM) pathway of G4, were examined. Mutants of the phenotypic class designated TOM A- were all defective in their ability to oxidize toluene, o-cresol, m-cresol, and phenol, suggesting that a single enzyme is responsible for conversion of these compounds to their hydroxylated products (3-methylcatechol from toluene, o-cresol, and m-cresol and catechol from phenol) in the wild type. Mutants of this class did not degrade TCE. Two other mutant classes which were blocked in toluene catabolism, TOM B-, which lacked \\tO\\ncatechol-2,3-dioxygenase\\tenzyme\\n, and TOM C-, which lacked \\tO\\n2-hydroxy-6-oxoheptadienoic acid hydrolase\\tenzyme\\n activity, were fully capable of TCE degradation. Therefore, TCE degradation is directly associated with the monooxygenation capability responsible for toluene, cresol, and phenol hydroxylation.\\tO\\n\\n',\n",
              " 'aVpYWI_FIBmNF9Q.wYEuetl8lCLi-1444373.tsv': b'Microbial transformation of benzocyclobutene to benzocyclobutene-1-ol and benzocyclobutene-1-one.\\tO\\n\\nAbstract\\tO\\n\\nOxidation of benzocyclobutene by intact cells of Pseudomonas fluorescens 127-68 XVII, containing \\tO\\nnaphthalene dioxygenase\\tenzyme\\n, yielded exclusively benzylic oxidation products; benzocyclobutene-1-ol and benzocyclobutene-1-one. No evidence for incorporation of oxygen into the aromatic ring was found.\\tO\\n\\n',\n",
              " 'aW12Y2HqOS2s.HdUDpjI4tCPW.aK-1444422.tsv': b'Metabolism of hexadecyltrimethylammonium chloride in Pseudomonas strain B1.\\tO\\n\\nAbstract\\tO\\n\\nA bacterium (strain B1) utilizing hexadecyltrimethylammonium chloride as a carbon and energy source was isolated from activated sludge and tentatively identified as a Pseudomonas sp. This bacterium only grew on alkyltrimethylammonium salts (C12 to C22) and possible intermediates of hexadecyltrimethylammonium chloride breakdown such as hexadecanoate and acetate. Pseudomonas strain B1 did not grow on amines. Simultaneous adaptation studies suggested that the bacterium oxidized only the alkyl chain of hexadecyltrimethylammonium chloride. This was confirmed by the stoichiometric formation of trimethylamine from hexadecyltrimethylammonium chloride. The initial \\tO\\nhexadecyltrimethylammonium chloride oxygenase\\tenzyme\\n activity, measured by its ability to form trimethylamine, was NAD(P)H and O2 dependent. Finally, assays of \\tO\\naldehyde dehydrogenase\\tenzyme\\n, \\tO\\nhexadecanoyl-coenzyme A dehydrogenase\\tenzyme\\n, and \\tO\\nisocitrate lyase\\tenzyme\\n in cell extracts revealed the potential of Pseudomonas strain B1 to metabolize the alkyl chain via beta-oxidation.\\tO\\n\\n',\n",
              " 'aWjQPKPgXqYQt4s0er.XVq9ksHf4-30324341.tsv': b'Impact of additives of commercial rubber compounds on the microbial and enzymatic degradation of poly(cis-1,4-isoprene).\\tO\\n\\nAbstract\\tO\\n\\nMuch fundamental research has already been performed to understand the mechanism of microbial rubber degradation. Due to the increasing amount of rubber waste, biotechnical methods to degrade that particular waste are strongly needed. The present study evaluates whether a microbial or an enzymatic process is more suitable for efficient biodegradation, due to less sensitivity towards rubber additives. Therefore we investigated the impact of 15 different frequently used rubber additives on cells of the potent rubber degrader Gordonia polyisoprenivorans VH2 and the enzyme \\tO\\nLcp1VH2\\tenzyme\\n. For this, cells were grown on poly(cis-1,4-isoprene) in presence of these rubber additives. Furthermore, the effect of those additives on the enzymatic cleavage of poly(cis-1,4-isoprene) by \\tO\\nLcp1VH2\\tenzyme\\n was determined by in vitro studies. It was observed that additives, used to accelerate the vulcanization process, like N-cyclohexyl-2-benzothiazolesulfenamide and zinc-bis(N,N-dibenzyl-dithiocarbamate), are diminishing the growth of the microorganism depending on their concentration-higher toxicity with increasing concentration. In contrast, sulfur prevents cell growth, but does not affect \\tO\\nLcp1VH2\\tenzyme\\n. Stearic acid and paraffin wax were found to be consumed by G. polyisoprenivorans VH2. Plasticizers mainly prevent growth, but do not interfere with the enzyme activity. This study identified antioxidants as the most interfering group of additives for microbial and enzymatic rubber degradation. It was found that the in vitro degradation by \\tO\\nLcp1VH2\\tenzyme\\n is much more resistant and less sensitive towards the investigated rubber additives, when compared to the in vivo approach. Therefore, an enzymatic process might be a promising method to enhance rubber degradation.\\tO\\n\\n',\n",
              " 'aXV8WPhnnXaKBDRW.KaFM9h3JK_8-24721596.tsv': b'Soil mesocosm studies on atrazine bioremediation.\\tO\\n\\nAbstract\\tO\\n\\nAccumulation of pesticides in the environment causes serious issues of contamination and toxicity. Bioremediation is an ecologically sound method to manage soil pollution, but the bottleneck here, is the successful scale-up of lab-scale experiments to field applications. This study demonstrates pilot-scale bioremediation in tropical soil using atrazine as model pollutant. Mimicking field conditions, three different bioremediation strategies for atrazine degradation were explored. 100 kg soil mesocosms were set-up, with or without atrazine application history. Natural attenuation and enhanced bioremediation were tested, where augmentation with an atrazine degrading consortium demonstrated best pollutant removal. 90% atrazine degradation was observed in six days in soil previously exposed to atrazine, while soil without history of atrazine use, needed 15 days to remove the same amount of amended atrazine. The bacterial consortium comprised of 3 novel bacterial strains with different genetic atrazine degrading potential. The progress of bioremediation was monitored by measuring the levels of atrazine and its intermediate, cyanuric acid. Genes from the atrazine degradation pathway, namely, \\tO\\natzA\\tgene\\n, \\tO\\natzB\\tgene\\n, \\tO\\natzD\\tgene\\n, \\tO\\ntrzN\\tgene\\n and \\tO\\ntrzD\\tgene\\n were quantified in all mesocosms for 60 days. The highest abundance of all target genes was observed on the 6th day of treatment. \\tO\\ntrzD\\tgene\\n was observed in the bioaugmented mesocosms only. The bacterial community profile in all mesocosms was monitored by LH-PCR over a period of two months. Results indicate that the communities changed rapidly after inoculation, but there was no drastic change in microbial community profile after 1 month. Results indicated that efficient bioremediation of atrazine using a microbial consortium could be successfully up-scaled to pilot scale.\\tO\\n\\n',\n",
              " 'aXadYOMx.9g1YFCDL6FE4VwsFQ9G-29886364.tsv': b'Effects of secondary carbon supplement on biofilm-mediated biodegradation of naphthalene by mutated naphthalene 1, 2-dioxygenase encoded by Pseudomonas putida strain KD9.\\tO\\n\\nAbstract\\tO\\n\\nPolycyclic aromatic hydrocarbons (PAHs) belong to a diverse group of environmental pollutants distributed ubiquitously in the environment. The carcinogenic properties of PAHs are the main causes of harm to human health. The green technology, biodegradation have become convenient options to address the environmental pollution. In this study, we analyzed the biodegradation potential of naphthalene with secondary carbon supplements (SCSs) in carbon deficient media (CSM) by Pseudomonas putida strain KD9 isolated from oil refinerary waste. The rigid-flexible molecular docking method revealed that the mutated naphthalene \\tO\\n1,2-dioxygenase\\tenzyme\\n had lower affinity for naphthalene than that found in wild type strain. Moreover, analytical methods (HPLC, qRT-PCR) and soft agar chemotaxis suggest sucrose (0.5\\xe2\\x80\\xafwt%) to be the best chemo-attractant and it unequivocally caused enhanced biodegradation of naphthalene (500\\xe2\\x80\\xafmg\\xe2\\x80\\xafL-1) in both biofilm-mediated and shake-flask biodegradation methods. In addition, the morphological analysis detected from microscopy clearly showed KD9 to change its size and shape (rod to pointed) during biodegradation of naphthalene in CSM as sole source of carbon and energy. The forward versus side light scatter plot of the singlet cells obtained from flow cytometry suggests smaller cell size in CSM and lower florescence intensity of the total DNA content of cells. This study concludes that sucrose may be used as potential bio-stimulation agent.\\tO\\n\\n',\n",
              " 'aYcUnBgYZ0CMSJ0ltle2cqxMkSES-23811502.tsv': b'HylA\\tgene\\n, an alternative hydrolase for initiation of catabolism of the phenylurea herbicide linuron in Variovorax sp. strains.\\tO\\n\\nAbstract\\tO\\n\\nVariovorax sp. strain WDL1, which mineralizes the phenylurea herbicide linuron, expresses a novel linuron-hydrolyzing enzyme, \\tO\\nHylA\\tgene\\n, that converts linuron to 3,4-dichloroaniline (DCA). The enzyme is distinct from the linuron hydrolase \\tO\\nLibA\\tgene\\n enzyme recently identified in other linuron-mineralizing Variovorax strains and from phenylurea-hydrolyzing enzymes (\\tO\\nPuhA\\tgene\\n, \\tO\\nPuhB\\tgene\\n) found in Gram-positive bacteria. The dimeric enzyme belongs to a separate family of hydrolases and differs in Km, temperature optimum, and phenylurea herbicide substrate range. Within the metal-dependent amidohydrolase superfamily, \\tO\\nHylA\\tgene\\n and \\tO\\nPuhA\\tgene\\n/\\tO\\nPuhB\\tgene\\n belong to two distinct protein families, while \\tO\\nLibA\\tgene\\n is a member of the unrelated \\tO\\namidase\\tenzyme\\n signature family. The \\tO\\nhylA\\tgene\\n gene was identified in a draft genome sequence of strain WDL1. The involvement of \\tO\\nhylA\\tgene\\n in linuron degradation by strain WDL1 is inferred from its absence in spontaneous WDL1 mutants defective in linuron hydrolysis and its presence in linuron-degrading Variovorax strains that lack \\tO\\nlibA\\tgene\\n. In strain WDL1, the \\tO\\nhylA\\tgene\\n gene is combined with catabolic gene modules encoding the downstream pathways for DCA degradation, which are very similar to those present in Variovorax sp. SRS16, which contains \\tO\\nlibA\\tgene\\n. Our results show that the expansion of a DCA catabolic pathway toward linuron degradation in Variovorax can involve different but isofunctional linuron hydrolysis genes encoding proteins that belong to evolutionary unrelated hydrolase families. This may be explained by divergent evolution and the independent acquisition of the corresponding genetic modules.\\tO\\n\\n',\n",
              " 'aYt2ZK7qs9azR0119c8jCx.UWB3C-20088886.tsv': b'atz gene expressions during atrazine degradation in the soil drilosphere.\\tO\\n\\nAbstract\\tO\\n\\nOne of the various ecosystemic services sustained by soil is pollutant degradation mediated by adapted soil bacteria. The pathways of atrazine biodegradation have been elucidated but in situ expression of the genes involved in atrazine degradation has yet to be demonstrated in soil. Expression of the \\tO\\natzA\\tgene\\n and \\tO\\natzD\\tgene\\n genes involved in atrazine dechlorination and s-triazine ring cleavage, respectively, was investigated during in situ degradation of atrazine in the soil drilosphere and bulked samples from two agricultural soils that differed in their ability to mineralize atrazine. Interestingly, expression of the \\tO\\natzA\\tgene\\n gene, although present in both soils, was not detected. Atrazine mineralization was greatest in Epoisses soil, where a larger pool of \\tO\\natzD\\tgene\\n mRNA was consistently measured 7 days after atrazine treatment, compared with Vezin soil (146 vs. 49 mRNA per 10(6)16S rRNA, respectively). Expression of the \\tO\\natzD\\tgene\\n gene varied along the degradation time course and was profoundly modified in soil bioturbated by earthworms. The \\tO\\natzD\\tgene\\n mRNA pool was the highest in the soil drilosphere (casts and burrow-linings) and it was significantly different in burrow-linings compared with bulk soil (e.g. 363 vs. 146 mRNA per 10(6)16S rRNA, 7 days after atrazine treatment in Epoisses soil). Thus, consistent differences in atrazine mineralization were demonstrated between the soil drilosphere and bulk soil. However, the impact of bioturbation on atrazine mineralization depended on soil type. Mineralization was enhanced in casts, compared with bulk soil, from Epoisses soil but in burrow-linings from Vezin soil. This study is the first to report the effects of soil bioturbation by earthworms on s-triazine ring cleavage and its spatial variability in soil.\\tO\\n\\n',\n",
              " 'aYzMIzEZRsCOVevEzvdU9zR6IuCa-19709256.tsv': b'Quantification of the atrazine-degrading Pseudomonas sp. strain ADP in aquifer sediment by quantitative competitive polymerase chain reaction.\\tO\\n\\nAbstract\\tO\\n\\nThe widely used herbicide atrazine and some of its degradation products are among the most commonly found xenobiotics in groundwater in Europe as well as in the USA. The bacterium Pseudomonas sp. strain ADP (P. ADP) possesses genes encoding atrazine mineralization on the self-transmissible plasmid pADP-1. In the present study, this ability of the strain to mineralize atrazine in aquifer sediment under both aerobic and denitrifying conditions at 10 degrees C was studied. P. ADP was able to mineralize more than 50% of 2.8 muM atrazine within 14 days under both growth conditions. Counts of degraders as colony forming units (CFU) on atrazine plates and counts of \\tO\\natzA\\tgene\\n gene copies as determined by quantitative competitive polymerase chain reaction (cPCR) were performed. The \\tO\\natzA\\tgene\\n gene encodes the enzyme which catalyzes the first step of atrazine mineralization by the strain. Quantification of the \\tO\\natzA\\tgene\\n gene gave rise to higher numbers than did counts of CFU. High nitrate concentrations inhibited atrazine mineralization and culturability on agar plates, but \\tO\\natzA\\tgene\\n copy numbers remained stable throughout the experiment. The results show a potential for bioaugmentation using P. ADP at both aerobic and denitrifying conditions and the use of cPCR as a tool for monitoring the bacteria independent of culturability.\\tO\\n\\n',\n",
              " 'a_40v3K2FGdNYfaS_6S6Jsri77.C-22635998.tsv': b'Modeling of phenoxy acid herbicide mineralization and growth of microbial degraders in 15 soils monitored by quantitative real-time PCR of the functional \\tO\\ntfdA\\tgene\\n gene.\\tO\\n\\nAbstract\\tO\\n\\nMineralization potentials, rates, and kinetics of the three phenoxy acid (PA) herbicides, 2,4-dichlorophenoxyacetic acid (2,4-D), 4-chloro-2-methylphenoxyacetic acid (MCPA), and 2-(4-chloro-2-methylphenoxy)propanoic acid (MCPP), were investigated and compared in 15 soils collected from five continents. The mineralization patterns were fitted by zero/linear or exponential growth forms of the three-half-order models and by logarithmic (log), first-order, or zero-order kinetic models. Prior and subsequent to the mineralization event, \\tO\\ntfdA\\tgene\\n genes were quantified using real-time PCR to estimate the genetic potential for degrading PA in the soils. In 25 of the 45 mineralization scenarios, \\xe2\\x88\\xbc60% mineralization was observed within 118 days. Elevated concentrations of \\tO\\ntfdA\\tgene\\n in the range 1 \\xc3\\x97 10(5) to 5 \\xc3\\x97 10(7) gene copies g(-1) of soil were observed in soils where mineralization could be described by using growth-linked kinetic models. A clear trend was observed that the mineralization rates of the three PAs occurred in the order 2,4-D > MCPA > MCPP, and a correlation was observed between rapid mineralization and soils exposed to PA previously. Finally, for 2,4-D mineralization, all seven mineralization patterns which were best fitted by the exponential model yielded a higher \\tO\\ntfdA\\tgene\\n gene potential after mineralization had occurred than the three mineralization patterns best fitted by the Lin model.\\tO\\n\\n',\n",
              " 'a_fUcGCWV_jfoorNEfY9ZLrgTdzy-2310183.tsv': b'New pathway for the biodegradation of indole in Aspergillus niger.\\tO\\n\\nAbstract\\tO\\n\\nIndole and its derivatives form a class of toxic recalcitrant environmental pollutants. The growth of Aspergillus niger was inhibited by very low concentrations (0.005 to 0.02%) of indole, even when 125- to 500-fold excess glucose was present in the medium. When 0.02% indole was added, the fungus showed a lag phase for about 30 h and the uptake of glucose was inhibited. Indole was metabolized by a new pathway via indoxyl (3-hydroxyindole), N-formylanthranilic acid, anthranilic acid, 2,3-dihydroxybenzoic acid, and catechol, which was further degraded by ortho cleavage. The enzymes \\tO\\nN-formylanthranilate deformylase\\tenzyme\\n, \\tO\\nanthranilate hydroxylase\\tenzyme\\n, \\tO\\n2,3-dihydroxybenzoate decarboxylase\\tenzyme\\n, and \\tO\\ncatechol dioxygenase\\tenzyme\\n were induced by indole as early as after 5 h of growth, and their activities were demonstrated in a cell-free system.\\tO\\n\\n',\n",
              " 'a_gwKjTrMLtrwW4Ry3U8VFfZu_Lu-14706746.tsv': b'Real-time reverse transcription PCR analysis of expression of atrazine catabolism genes in two bacterial strains isolated from soil.\\tO\\n\\nAbstract\\tO\\n\\nThe level of expression of highly conserved, plasmid-borne, and widely dispersed atrazine catabolic genes (atz) was studied by RT-qPCR in two telluric atrazine-degrading microbes. RT-qPCR assays, based on the use of real-time PCR, were developed in order to quantify atzABCDEF mRNAs in Pseudomonas sp. ADP and atzABC mRNAs in Chelatobacter heintzii. atz gene expression was expressed as mRNA copy number per 10(6) 16S rRNA. In Pseudomonas sp. ADP, atz genes were basally expressed. It confirmed atrazine-degrading kinetics indicating that catabolic activity starts immediately after adding the herbicide. atz gene expression increased transitorily in response to atrazine treatment. This increase was only observed while low amount of atrazine remained in the medium. In C. heintzii, only \\tO\\natzA\\tgene\\n was basally expressed. \\tO\\natzA\\tgene\\n and \\tO\\natzB\\tgene\\n expression levels were similarly and significantly increased in response to atrazine treatment. \\tO\\natzC\\tgene\\n was not expressed even in the presence of high amounts of atrazine. This study showed that atz genes are basally expressed and up-regulated in response to atrazine treatment. atz gene expression patterns are different in Pseudomonas ADP and C. heintzii suggesting that the host may influence the expression of plasmid-borne atrazine-catabolic potential.\\tO\\n\\n',\n",
              " 'a_rI55cZbMHdHTM0cNz7wVUGh.YG-15240330.tsv': b'Arthrobacter aurescens TC1 atrazine catabolism genes trzN, atzB, and atzC are linked on a 160-kilobase region and are functional in Escherichia coli.\\tO\\n\\nAbstract\\tO\\n\\nArthrobacter aurescens strain TC1 metabolizes atrazine to cyanuric acid via \\tO\\nTrzN\\tgene\\n, \\tO\\nAtzB\\tgene\\n, and \\tO\\nAtzC\\tgene\\n. The complete sequence of a 160-kb bacterial artificial chromosome clone indicated that \\tO\\ntrzN\\tgene\\n, \\tO\\natzB\\tgene\\n, and \\tO\\natzC\\tgene\\n are linked on the A. aurescens genome. \\tO\\nTrzN\\tgene\\n, \\tO\\nAtzB\\tgene\\n, and \\tO\\nAtzC\\tgene\\n were shown to be functional in Escherichia coli. Hybridization studies localized \\tO\\ntrzN\\tgene\\n, \\tO\\natzB\\tgene\\n, and \\tO\\natzC\\tgene\\n to a 380-kb plasmid in A. aurescens strain TC1.\\tO\\n\\n',\n",
              " 'adFPfP5Za2Tqr7GICnCj4szpgbR0-21974851.tsv': b'Characterization of the propanil biodegradation pathway in Sphingomonas sp. Y57 and cloning of the \\tO\\npropanil hydrolase\\tenzyme\\n gene \\tO\\nprpH\\tgene\\n.\\tO\\n\\nAbstract\\tO\\n\\nIn our previous study, the isoproturon-degrading strain Sphingomonas sp. Y57 was isolated from the wastewater treatment system of an herbicide factory. Interestingly, this strain also showed the ability to degrade propanil (3,4-dichloropropionamilide). The present work reveals that Y57 degrades propanil via the following pathway: propanil was initially hydrolyzed to 3,4-dichloroaniline (3,4-DCA) and then converted to 4,5-dichlorocatechol, which was then subjected to aromatic ring cleavage and further processing. N-acylation and N-deacylation of 3,4-DCA also occurred, and among N-acylation products, 3,4-dichloropropionanilide was found for the first time. The gene encoding the \\tO\\npropanil hydrolase\\tenzyme\\n responsible for transforming propanil into 3,4-DCA was cloned from Y57 and was designated as \\tO\\nprpH\\tgene\\n. \\tO\\nPrpH\\tgene\\n was expressed in Escherichia coli BL21 and purified using Ni-nitrilotriacetic acid affinity chromatography. \\tO\\nPrpH\\tgene\\n displayed the highest activity against propanil at 40\\xc2\\xb0C and at pH 7.0. The effect of metal ions on the propanil-degrading activity of \\tO\\nPrpH\\tgene\\n was also determined. To our knowledge, this is the first report of a strain that can degrade both propanil and 3,4-DCA and the first identification of a gene encoding a \\tO\\npropanil hydrolase\\tenzyme\\n in bacteria.\\tO\\n\\n',\n",
              " 'adHR6sQH7bCPevVwArbdhgPTsT58-24676750.tsv': b'Degradation of atrazine by Frankia alni ACN14a: gene regulation, dealkylation, and dechlorination.\\tO\\n\\nAbstract\\tO\\n\\nAtrazine is transformed to N-isopropylammelide through hydroxyatrazine as an intermediate as indicated by high-performance liquid chromatography/mass spectroscopy in culture filtrates of Frankia alni ACN14a and Frankia sp. EuI1c. Both Frankia strains have the ability to degrade atrazine via dechlorination and dealkylation and, subsequently, may be using it as a nitrogen and carbon source as detected here by increasing their growth patterns. Bioinformatic analysis of the Frankia genomes revealed that a potential gene cluster involved in atrazine decomposition contains three genes, namely, \\tO\\ntrzN\\tgene\\n (FRAAL1474 and FraEuI1c_5874), \\tO\\natzB\\tgene\\n (FRAAL1473 and FraEuI1c_5875), and \\tO\\natzR\\tgene\\n (FRAAL1471). The relative messenger RNA gene expression of the former genes was examined by qRT-PCR. The LysR-type transcriptional regulator \\tO\\natzR\\tgene\\n (FRAAL1471), which is expected to control the cluster expression, showed a 13-fold increase in the expression level under atrazine stress. Moreover, the putative \\tO\\nadenosine aminohydrolase 3\\tenzyme\\n \\tO\\natzB\\tgene\\n (FRAAL1473), which is expected to dealkylate the N-ethyl group of atrazine, showed also an increased expression by factor 16 with increased exposure. Eventually, the \\tO\\ntrzN\\tgene\\n (FRAAL1474) gene, which is predicted to encode a putative amidohydrolase catalyzing atrazine dechlorination, exhibited 31-fold increased expression. To our best knowledge, this is the first report about \\tO\\nadenosine aminohydrolase 3\\tenzyme\\n function in the dealkylation of the N-ethyl group from atrazine.\\tO\\n\\n',\n",
              " 'aeLjOQVjTmU6CxOxaC6IOPkD15ai-29150672.tsv': b'FMNH2-dependent monooxygenases initiate catabolism of sulfonamides in Microbacterium sp. strain BR1 subsisting on sulfonamide antibiotics.\\tO\\n\\nAbstract\\tO\\n\\nWe report a cluster of genes encoding two \\tO\\nmonooxygenase\\tenzyme\\ns (\\tO\\nSadA\\tgene\\n and \\tO\\nSadB\\tgene\\n) and one \\tO\\nFMN reductase\\tenzyme\\n (\\tO\\nSadC\\tgene\\n) that enable Microbacterium sp. strain BR1 and other Actinomycetes to inactivate sulfonamide antibiotics. Our results show that \\tO\\nSadA\\tgene\\n and \\tO\\nSadC\\tgene\\n are responsible for the initial attack of sulfonamide molecules resulting in the release of 4-aminophenol. The latter is further transformed into 1,2,4-trihydroxybenzene by \\tO\\nSadB\\tgene\\n and \\tO\\nSadC\\tgene\\n prior to mineralization and concomitant production of biomass. As the degradation products lack antibiotic activity, the presence of \\tO\\nSadA\\tgene\\n will result in an alleviated bacteriostatic effect of sulfonamides. In addition to the relief from antibiotic stress this bacterium gains access to an additional carbon source when this gene cluster is expressed. As degradation of sulfonamides was also observed when Microbacterium sp. strain BR1 was grown on artificial urine medium, colonization with such strains may impede common sulfonamide treatment during co-infections with pathogens of the urinary tract. This case of biodegradation exemplifies the evolving catabolic capacity of bacteria, given that sulfonamide bacteriostatic are purely of synthetic origin. The wide distribution of this cluster in Actinomycetes and the presence of traA encoding a relaxase in its vicinity suggest that this cluster is mobile and that is rather alarming.\\tO\\n\\n',\n",
              " 'aeRSJ.g51yv44AhUWaZmaS3zrcwS-1416975.tsv': b'Kinetics of \\tO\\nsuperoxide dismutase\\tenzyme\\n- and iron-catalyzed nitration of phenolics by peroxynitrite.\\tO\\n\\nAbstract\\tO\\n\\nSuperoxide dismutase\\tenzyme\\n and Fe3+EDTA catalyzed the nitration by peroxynitrite (ONOO-) of a wide range of phenolics including tyrosine in proteins. Nitration was not mediated by a free radical mechanism because hydroxyl radical scavengers did not reduce either \\tO\\nsuperoxide dismutase\\tenzyme\\n or Fe3+EDTA-catalyzed nitration and nitrogen dioxide was not a significant product from either catalyst. Rather, metal ions appear to catalyze the heterolytic cleavage of peroxynitrite to form a nitronium-like species (NO2+). The calculated energy for separating peroxynitrous acid into hydroxide ion and nitronium ion is 13 kcal.mol-1 at pH 7.0. Fe3+EDTA catalyzed nitration with an activation energy of 12 kcal.mol-1 at a rate of 5700 M-1.s-1 at 37 degrees C and pH 7.5. The reaction rate of peroxynitrite with bovine Cu,Zn \\tO\\nsuperoxide dismutase\\tenzyme\\n was 10(5) M-1.s-1 at low \\tO\\nsuperoxide dismutase\\tenzyme\\n concentrations, but the rate of nitration became independent of \\tO\\nsuperoxide dismutase\\tenzyme\\n concentration above 10 microM with only 9% of added peroxynitrite yielding nitrophenol. We propose that peroxynitrite anion is more stable in the cis conformation, whereas only a higher energy species in the trans conformation can fit in the active site of Cu,Zn \\tO\\nsuperoxide dismutase\\tenzyme\\n. At high \\tO\\nsuperoxide dismutase\\tenzyme\\n concentrations, phenolic nitration may be limited by the rate of isomerization from the cis to trans conformations of peroxynitrite as well as by competing pathways for peroxynitrite decomposition. In contrast, Fe3+EDTA appears to react directly with the cis anion, resulting in greater nitration yields.\\tO\\n\\n',\n",
              " 'af5Vd1_V.IZVv2fy8SRV3TX_iUQ8-28401633.tsv': b'Functional soil metagenomics: elucidation of polycyclic aromatic hydrocarbon degradation potential following 12 years of in situ bioremediation.\\tO\\n\\nAbstract\\tO\\n\\nA culture-independent function-based screening approach was used to assess the microbial aerobic catabolome for polycyclic aromatic hydrocarbons degradation of a soil subjected to 12 years of in situ bioremediation. A total of 422 750 fosmid clones were screened for key aromatic ring-cleavage activities using 2,3-dihydroxybiphenyl as substrate. Most of the genes encoding ring-cleavage enzymes on the 768 retrieved positive fosmids could not be identified using primer-based approaches and, thus, 205 fosmid inserts were sequenced. Nearly two hundred extradiol dioxygenase encoding genes of three different superfamilies could be identified. Additional key genes of aromatic metabolic pathways were identified, including a high abundance of Rieske \\tO\\nnon-heme iron oxygenase\\tenzyme\\ns that provided detailed information on enzymes activating aromatic compounds and enzymes involved in activation of the side chain of methylsubstituted aromatics. The gained insights indicated a complex microbial network acting at the site under study, which comprises organisms similar to recently identified Immundisolibacter cernigliae TR3.2 and Rugosibacter aromaticivorans Ca6 and underlined the great potential of an approach that combines an activity-screening, a cost-effective high-throughput sequencing of fosmid clones and a phylogenomic-routed and manually curated database to carefully identify key proteins dedicated to aerobic degradation of aromatic compounds.\\tO\\n\\n',\n",
              " 'afHujGvs_M.1SLW0m4OXadVRJvT8-31549657.tsv': b'Beyond N and P: The impact of Ni on crude oil biodegradation.\\tO\\n\\nAbstract\\tO\\n\\nN and P are the key limiting nutrients considered most important for the stimulation of crude oil degradation but other trace nutrients may also be important. Experimental soil microcosms were setup to investigate crude oil degradation in the context of Ni amendments. Amended Nickel as NiO, NiCl2, or, a porphyrin complex either inhibited, had no effect, or, enhanced aerobic hydrocarbon degradation in an oil-contaminated soil. Biodegradation was significantly (95% confidence) enhanced (70%) with low levels of Ni-Porph (12\\xe2\\x80\\xafmg/kg) relative to an oil-only control; whereas, NiO (200 and 350\\xe2\\x80\\xafmg/kg) significantly inhibited (36 and 87%) biodegradation consistent with oxide particle induced reactive oxygen stress. Microbial community compositions were also significantly affected by Ni. In 16S rRNA sequence libraries, the enriched hydrocarbon degrading genus, Rhodococcus, was partially replaced by a Nocardia sp. in the presence of low levels of NiO (12 and 50\\xe2\\x80\\xafmg/kg). In contrast, the highest relative and absolute Rhodococcus abundances were coincident with the maximal rates of oil degradation observed in the Ni-Porph-amended soils. Growth dependent constitutive requirements for \\tO\\nNi-dependent urease\\tenzyme\\n or perhaps \\tO\\nNi-dependent superoxide dismutase\\tenzyme\\n enzymes (found in Rhodococcus genomes) provided a mechanistic explanation for stimulation. These results suggest biostimulation technologies, in addition to N and P, should also consider trace nutrients such as Ni tacitly considered adequately supplied and available in a typical soil.\\tO\\n\\n',\n",
              " 'afJ8f.ek_mLp6PuUrK9e4uiRMQc8-2219955.tsv': b'Dissimilation of 2,4-dichlorophenoxyacetic acid by Azotobacter chroococcum.\\tO\\n\\nAbstract\\tO\\n\\n1. A strain of Azotobacter chroococcum which could use 2,4-dichlorophenoxyacetic acid (2,4-D) as sole carbon source was isolated. 2. The strain metabolized 2,4-D via p-chlorophenoxyacetic acid, p-chlorophenol and 4-chlorocatechol; the last metabolite was cleaved by \\tO\\ncatechol 1,2-dioxygenase\\tenzyme\\n. 3. The enzyme exhibited broad substrate specificity.\\tO\\n\\n',\n",
              " 'afYImpCZ9lNrPIWIyzjfgfu2pcbu-26273258.tsv': b'Isolation and 2,4-D-degrading characteristics of Cupriavidus campinensis BJ71.\\tO\\n\\nAbstract\\tO\\n\\nAn indigenous bacterial strain capable of utilizing 2,4-dichlorophenoxyacetic acid as the sole carbon and energy source was isolated from a soil used for grown wheat with a long-term history of herbicide use in Beijing, China. The strain BJ71 was identified as Cupriavidus campinensis based on its 16S rRNA sequence analysis and morphological, physiological, and biochemical characteristics. The degradation characteristics of strain BJ71 were evaluated. The optimal conditions for 2,4-D degradation were as follows: pH 7.0, 30 \\xc2\\xb0C, 3% (v/v) inoculum size, and an initial 2,4-D concentration of 350 mg L(-1). Up to 99.57% of the 2,4-D was degraded under optimal conditions after 6 days of incubation. Strain BJ71 was also able to degrade quizalofop and fluroxypyr. This is the first report of a 2,4-D-degrader containing \\tO\\ntfdA\\tgene\\n gene that can utilize these two herbicides. In a biodegradation experiment, 87.13% and 42.53% of 2,4-D (initial concentration, 350 mg kg(-1)) was degraded in non-sterile and sterilized soil inoculated with BJ71, respectively, after 14 days. The 2,4-D degradation was more rapid in a soil microcosm including BJ71 than in a soil microcosm without BJ71. These results indicate that strain BJ71 is a potential candidate for the bioremediation of soil contaminated with the herbicide 2,4-D.\\tO\\n\\n',\n",
              " 'agM5xp_DbJltbmulA8.crpkhzCjC-30647456.tsv': b'Anaerobic degradation of xenobiotic isophthalate by the fermenting bacterium Syntrophorhabdus aromaticivorans.\\tO\\n\\nAbstract\\tO\\n\\nSyntrophorhabdus aromaticivorans is a syntrophically fermenting bacterium that can degrade isophthalate (3-carboxybenzoate). It is a xenobiotic compound which has accumulated in the environment for more than 50 years due to its global industrial usage and can cause negative effects on the environment. Isophthalate degradation by the strictly anaerobic S. aromaticivorans was investigated to advance our understanding of the degradation of xenobiotics introduced into nature, and to identify enzymes that might have ecological significance for bioremediation. Differential proteome analysis of isophthalate- vs benzoate-grown cells revealed over 400 differentially expressed proteins of which only four were unique to isophthalate-grown cells. The isophthalate-induced proteins include a phenylacetate:\\tO\\nCoA ligase\\tenzyme\\n, a \\tO\\nUbiD-like decarboxylase\\tenzyme\\n, a \\tO\\nUbiX-like flavin prenyltransferase\\tenzyme\\n, and a hypothetical protein. These proteins are encoded by genes forming a single gene cluster that putatively codes for anaerobic conversion of isophthalate to benzoyl-CoA. Subsequently, benzoyl-CoA is metabolized by the enzymes of the anaerobic benzoate degradation pathway that were identified in the proteomic analysis. In vitro enzyme assays with cell-free extracts of isophthalate-grown cells indicated that isophthalate is activated to isophthalyl-CoA by an ATP-dependent isophthalate:\\tO\\nCoA ligase\\tenzyme\\n (IPCL), and subsequently decarboxylated to benzoyl-CoA by a \\tO\\nUbiD family isophthalyl-CoA decarboxylase\\tenzyme\\n (IPCD) that requires a prenylated flavin mononucleotide (prFMN) cofactor supplied by \\tO\\nUbiX\\tgene\\n to effect decarboxylation. Phylogenetic analysis revealed that IPCD is a novel member of the functionally diverse \\tO\\nUbiD family (de)carboxylases\\tenzyme\\n. Homologs of the IPCD encoding genes are found in several other bacteria, such as aromatic compound-degrading denitrifiers, marine sulfate-reducers, and methanogenic communities in a terephthalate-degrading reactor. These results suggest that metabolic strategies adapted for degradation of isophthalate and other phthalate are conserved between microorganisms that are involved in the anaerobic degradation of environmentally relevant aromatic compounds.\\tO\\n\\n',\n",
              " 'ah7ZS_ptJZlahOrm7G4ReJWe5mFy-1429434.tsv': b'Phenol hydroxylase\\tenzyme\\n from Trichosporon cutaneum: gene cloning, sequence analysis, and functional expression in Escherichia coli.\\tO\\n\\nAbstract\\tO\\n\\nA cDNA clone encoding \\tO\\nphenol hydroxylase\\tenzyme\\n from the soil yeast Trichosporon cutaneum was isolated and characterized. The clone was identified by hybridization screening of a bacteriophage lambda ZAP-based cDNA library with an oligonucleotide probe which corresponded to the N-terminal amino acid sequence of the purified enzyme. The cDNA encodes a protein consisting of 664 amino acids. Amino acid sequences of a number of peptides obtained by Edman degradation of various cleavage products of the purified enzyme were identified in the cDNA-derived sequence. The \\tO\\nphenol hydroxylase\\tenzyme\\n cDNA was expressed in Escherichia coli to yield high levels of active enzyme. The E. coli-derived \\tO\\nphenol hydroxylase\\tenzyme\\n is very similar to the T. cutaneum enzyme with respect to the range of substrates acted upon, inhibition by excess phenol, and the order of magnitude of kinetic parameters in the overall reaction. Southern blot analysis revealed the presence of \\tO\\nphenol hydroxylase\\tenzyme\\n gene-related sequences in a number of T. cutaneum and Trichosporon beigelii strains and in Cryptococcus elinovii but not in Trichosporon pullulans, Trichosporon penicillatum, or Candida tropicalis.\\tO\\n\\n',\n",
              " 'ahsvvoMu84Zjr7Rhu16yMIALmbNW-21615634.tsv': b'The presence of atrazine and atrazine-degrading bacteria in the residential, cattle farming, forested and golf course regions of Lake Oconee.\\tO\\n\\nAbstract\\tO\\n\\nAIMS\\tO\\n\\nTo assess the concentration of atrazine in Lake Oconee and develop a qPCR assay as a potential marker for the presence of atrazine-degrading bacteria indicating atrazine contamination.\\tO\\n\\nMETHODS AND RESULTS\\tO\\n\\nWater and sediment samples were collected from the Oconee Lake at four golf course sites, two residential sites, one cattle farming site and a forested site. Atrazine concentration at the study sites was determined using an ELISA kit and indicated the presence of atrazine from 0\\xc2\\xb772 ppb at the forested sites to 1\\xc2\\xb784 ppb at the golf course sites. QPCR results indicate the presence of \\tO\\natzA\\tgene\\n gene (\\tO\\natrazine chlorohydrolase\\tenzyme\\n) from 1\\xc2\\xb751 \\xc3\\x97 10(2) gene copies at the residential sites to 3\\xc2\\xb731 \\xc3\\x97 10(5) gene copies per 100 ml of water at the golf course regions of the lake and correlated (r = 0\\xc2\\xb764) with atrazine concentration. Sediment samples had higher \\tO\\natzA\\tgene\\n gene copies compared with the water samples (P < 0\\xc2\\xb705).\\tO\\n\\nCONCLUSIONS\\tO\\n\\nAtrazine concentration and the highest quantity of \\tO\\natzA\\tgene\\n gene were detected in the golf course regions of the lake. Overall, atrazine concentration monitored in Lake Oconee was below the Environment Protection Agency (EPA) regulatory standards.\\tO\\n\\nSIGNIFICANCE AND IMPACT OF THE STUDY\\tO\\n\\nQuantitative PCR is an efficient technique for assessing the presence of atrazine catabolism gene as a functional marker for atrazine-degrading bacteria and the presence of atrazine contamination.\\tO\\n\\n',\n",
              " 'ahwTYdMzdaIWKB3vd5dHo7Gtzk0m-28363960.tsv': b'Genetic (In)stability of 2,6-Dichlorobenzamide Catabolism in Aminobacter sp. Strain MSH1 Biofilms under Carbon Starvation Conditions.\\tO\\n\\nAbstract\\tO\\n\\nAminobacter sp. strain MSH1 grows on and mineralizes the groundwater micropollutant 2,6-dichlorobenzamide (BAM) and is of interest for BAM removal in drinking water treatment plants (DWTPs). The BAM-catabolic genes in MSH1 are located on plasmid \\tO\\npBAM1\\tgene\\n, carrying \\tO\\nbbdA\\tgene\\n, which encodes the conversion of BAM to 2,6-dichlorobenzoic acid (2,6-DCBA) (\\tO\\nBbdA\\tgene\\n+ phenotype), and plasmid \\tO\\npBAM2\\tgene\\n, carrying gene clusters encoding the conversion of 2,6-DCBA to tricarboxylic acid (TCA) cycle intermediates (Dcba+ phenotype). There are indications that MSH1 easily loses its BAM-catabolic phenotype. We obtained evidence that MSH1 rapidly develops a population that lacks the ability to mineralize BAM when grown on nonselective (R2B medium) and semiselective (R2B medium with BAM) media. Lack of mineralization was explained by loss of the Dcba+ phenotype and corresponding genes. The ecological significance of this instability for the use of MSH1 for BAM removal in the oligotrophic environment of DWTPs was explored in lab and pilot systems. A higher incidence of \\tO\\nBbdA\\tgene\\n+ Dcba- MSH1 cells was also observed when MSH1 was grown as a biofilm in flow chambers under C and N starvation conditions due to growth on nonselective residual assimilable organic carbon. Similar observations were made in experiments with a pilot sand filter reactor bioaugmented with MSH1. BAM conversion to 2,6-DCBA was not affected by loss of the DCBA-catabolic genes. Our results show that MSH1 is prone to BAM-catabolic instability under the conditions occurring in a DWTP. While conversion of BAM to 2,6-DCBA remains unaffected, BAM mineralization activity is at risk, and monitoring of metabolites is warranted.IMPORTANCE Bioaugmentation of dedicated biofiltration units with bacterial strains that grow on and mineralize micropollutants was suggested as an alternative for treating micropollutant-contaminated water in drinking water treatment plants (DWTPs). Organic-pollutant-catabolic genes in bacteria are often easily lost, especially under nonselective conditions, which affects the bioaugmentation success. In this study, we provide evidence that Aminobacter sp. strain MSH1, which uses the common groundwater micropollutant 2,6-dichlorobenzamide (BAM) as a C source, shows a high frequency of loss of its BAM-mineralizing phenotype due to the loss of genes that convert 2,6-DCBA to Krebs cycle intermediates when nonselective conditions occur. Moreover, we show that catabolic-gene loss also occurs in the oligotrophic environment of DWTPs, where growth of MSH1 depends mainly on the high fluxes of low concentrations of assimilable organic carbon, and hence show the ecological relevance of catabolic instability for using strain MSH1 for BAM removal in DWTPs.\\tO\\n\\n',\n",
              " 'ai.4gEPfAl8ZZ1zaemJCkzjRZIcS-1587481.tsv': b'Cloning and sequencing of Pseudomonas genes determining sodium dodecyl sulfate biodegradation.\\tO\\n\\nAbstract\\tO\\n\\nThe nucleotide sequences of two genes involved in sodium dodecyl sulfate (SDS) degradation, by Pseudomonas, have been determined. One of these, \\tO\\nsdsA\\tgene\\n, codes for an \\tO\\nalkyl sulfatase\\tenzyme\\n (58,957 Da) and has similarity (31.8% identity over a 201-amino acid stretch) to the N terminus of a predicted protein of unknown function from Mycobacterium tuberculosis. The other gene, \\tO\\nsdsB\\tgene\\n, codes for a positive activator protein (33,600 Da) that has extensive similarity with the lysR family of helix-turn-helix DNA-binding activator proteins.\\tO\\n\\n',\n",
              " 'aiJjGYw7ugb4aPZPKqQPn.7BC6Jq-1447127.tsv': b'Bacterial metabolism of naphthalene: construction and use of recombinant bacteria to study ring cleavage of 1,2-dihydroxynaphthalene and subsequent reactions.\\tO\\n\\nAbstract\\tO\\n\\nThe reactions involved in the bacterial metabolism of naphthalene to salicylate have been reinvestigated by using recombinant bacteria carrying genes cloned from plasmid \\tO\\nNAH7\\tgene\\n. When intact cells of Pseudomonas aeruginosa PAO1 carrying DNA fragments encoding the first three enzymes of the pathway were incubated with naphthalene, they formed products of the dioxygenase-catalyzed ring cleavage of 1,2-dihydroxynaphthalene. These products were separated by chromatography on Sephadex G-25 and were identified by 1H and 13C nuclear magnetic resonance spectroscopy and gas chromatography-mass spectrometry as 2-hydroxychromene-2-carboxylate (HCCA) and trans-o-hydroxybenzylidenepyruvate (tHBPA). HCCA was detected as the first reaction product in these incubation mixtures by its characteristic UV spectrum, which slowly changed to a spectrum indicative of an equilibrium mixture of HCCA and tHBPA. Isomerization of either purified product occurred slowly and spontaneously to give an equilibrium mixture of essentially the same composition. tHBPA is also formed from HCCA by the action of an isomerase enzyme encoded by plasmid \\tO\\nNAH7\\tgene\\n. The gene encoding this enzyme, \\tO\\nnahD\\tgene\\n, was cloned on a 1.95-kb KpnI-BglII fragment. Extracts of Escherichia coli JM109 carrying this fragment catalyzed the rapid equilibration of HCCA and tHBPA. Metabolism of tHBPA to salicylaldehyde by hydration and aldol cleavage is catalyzed by a single enzyme encoded by a 1-kb MluI-StuI restriction fragment. A mechanism for the hydratase-aldolase-catalyzed reaction is proposed. The \\tO\\nsalicylaldehyde dehydrogenase\\tenzyme\\n gene, \\tO\\nnahF\\tgene\\n, was cloned on a 2.75-kb BamHI fragment which also carries the \\tO\\nnaphthalene dihydrodiol dehydrogenase\\tenzyme\\n gene, \\tO\\nnahB\\tgene\\n. On the basis of the identification of the enzymes encoded by various clones, the gene order for the nah operon was shown to be p, A, B, F, C, E, D.\\tO\\n\\n',\n",
              " 'aiZwtrEfgPnqN6Sj4buoSESUOhmK-27005792.tsv': b'Microbial biodegradation of proteinaceous tannery solid waste and production of a novel value added product - Metalloprotease.\\tO\\n\\nAbstract\\tO\\n\\nIn this study, animal fleshing (ANFL) was utilized as a substrate for the production of extracellular protease by Clostridium limosum through central composite rotatable design (CCRD) and response surface methodology (RSM). Optimum protease production of 433U/ml was achieved and the purified enzyme was identified as \\tO\\nacidic metalloprotease\\tenzyme\\n, a monomeric protein. The molecular weight of the enzyme was 71kDa, whose activity was enhanced by bivalent metals such as Zn(2+) and Mg(2+). Scanning electron microscopy (SEM) examination also revealed the hydrolysis/microbial degradation of ANFL through protease activity in the anaerobic fermentation process. Simultaneous hydrolysis of ANFL and production of an enzyme with the potential for different industrial applications provide an attractive methodology for the disposal of tannery solid waste.\\tO\\n\\n',\n",
              " 'aioL7f.VZae8CICcRDJxPjZZ9i.u-23066709.tsv': b'Enrichment and dynamics of novel syntrophs in a methanogenic hexadecane-degrading culture from a Chinese oilfield.\\tO\\n\\nAbstract\\tO\\n\\nMethanogenic communities that degrade alkanes have been reported. However, little is known about the key players involved in the process. Methanogenic hexadecane-degrading consortia were enriched from an oilfield (Shengli, China). The microbial dynamics during the transfer incubations were monitored using terminal restriction fragment length polymorphism (T-RFLP) fingerprinting of 16S rRNA genes in combination with cloning and sequencing. The archaeal community shifted from a predominance of aceticlastic Methanosaeta during early cultivation to a substantial increase in hydrogenotrophic Methanoculleus in the highly enriched culture. Bacterial T-RFs 161 and 164 bp were consistently detected during the incubation and became dominant in the highly enriched culture. T-RF 161 bp primarily represented uncultured Waste Water of Evry 1 bacterium, which was possibly associated with Candidatus Cloacamonas acidaminovorans (99.7% sequence similarity). T-RF 164 bp could be assigned to both Thermotogaceae, with the closest relative being Candidatus Mesotoga sulfurreducens (similarity of 97%) and Syntrophaceae, with Smithella propionica as the closest relative (similarity of 96-97%). These bacterial lineages were potentially capable of syntrophic interactions with methanogen partners during hexadecane degradation. Partial \\tO\\nassA\\tgene\\n genes (encoding the \\xce\\xb1-subunit of \\tO\\nalkylsuccinate synthase\\tenzyme\\n) were also detected, implying that the mechanism of fumarate addition may function in the hexadecane activation.\\tO\\n\\n',\n",
              " 'aiyTKDM2vSKVPC57nYa4PXKHMzGm-18451.tsv': b'Purification and some properties of \\tO\\ncyclohexylamine oxidase\\tenzyme\\n from a Pseudomonas sp.\\tO\\n\\nAbstract\\tO\\n\\nCyclohexylamine oxidase\\tenzyme\\n was purified 90-fold from cell-free extracts of Pseudomonas sp. capable of assimilating sodium cyclamate. The purified enzyme was homogeneous in disc electrophoresis, and the molecular weight was found to be approximately 80,000 by gel filtration. The enzyme catalyzed the following reaction: cyclohexylamine+O2+H2O leads to cyclohexanone+NH3+H2O2. The enzyme thus can be classified as an amine oxidase; it utilized oxygen as the ultimate electron acceptor. The pH optimum of the reaction was 6.8 and the apparent Km value for cyclohexylamine was 2.5 X 10(-4) M. The enzyme was highly specific for the deamination of alicyclic primary amines such as cyclohexylamine, but was found to be inactive toward ordinary amines used as substrates for amine oxidases. The enzyme solution was yellow in color and showed a typical flavoprotein spectrum; the addition of cyclohexylamine under anaerobic conditions caused reduction of the flavin in the native enzyme. The flavin of the prosthetic group was identified as FAD by thin layer chromatography. The participation of sulfhydryl groups in the enzymic action was also suggested by the observation that the enzyme activity was inhibited in the presence of PCMB and could be recovered by the addition of glutathione.\\tO\\n\\n',\n",
              " 'ajgdpPcEt7PN4OhUnb8AiAxsqqku-30326388.tsv': b'Multi-method assessment of the intrinsic biodegradation potential of an aquifer contaminated with chlorinated ethenes at an industrial area in Barcelona (Spain).\\tO\\n\\nAbstract\\tO\\n\\nThe bioremediation potential of an aquifer contaminated with tetrachloroethene (PCE) was assessed by combining hydrogeochemical data of the site, microcosm studies, metabolites concentrations, compound specific-stable carbon isotope analysis and the identification of selected reductive dechlorination biomarker genes. The characterization of the site through 10 monitoring wells evidenced that leaked PCE was transformed to TCE and cis-DCE via hydrogenolysis. Carbon isotopic mass balance of chlorinated ethenes pointed to two distinct sources of contamination and discarded relevant alternate degradation pathways in the aquifer. Application of specific-genus primers targeting Dehalococcoides mccartyi species and the \\tO\\nvinyl chloride-to-ethene reductive dehalogenase\\tenzyme\\n \\tO\\nvcrA\\tgene\\n indicated the presence of autochthonous bacteria capable of the complete dechlorination of PCE. The observed cis-DCE stall was consistent with the aquifer geochemistry (positive redox potentials; presence of dissolved oxygen, nitrate, and sulphate; absence of ferrous iron), which was thermodynamically favourable to dechlorinate highly chlorinated ethenes but required lower redox potentials to evolve beyond cis-DCE to the innocuous end product ethene. Accordingly, the addition of lactate or a mixture of ethanol plus methanol as electron donor sources in parallel field-derived anoxic microcosms accelerated dechlorination of PCE and passed cis-DCE up to ethene, unlike the controls (without amendments, representative of field natural attenuation). Lactate fermentation produced acetate at near-stoichiometric amounts. The array of techniques used in this study provided complementary lines of evidence to suggest that enhanced anaerobic bioremediation using lactate as electron donor source is a feasible strategy to successfully decontaminate this site.\\tO\\n\\n',\n",
              " 'ajipNizGkakHghlktMYLqC1CDxdO-10919784.tsv': b'Effect of dissemination of 2,4-dichlorophenoxyacetic acid (2,4-D) degradation plasmids on 2,4-D degradation and on bacterial community structure in two different soil horizons.\\tO\\n\\nAbstract\\tO\\n\\nTransfer of the 2,4-dichlorophenoxyacetic acid (2,4-D) degradation plasmids \\tO\\npEMT1\\tgene\\n and pJP4 from an introduced donor strain, Pseudomonas putida UWC3, to the indigenous bacteria of two different horizons (A horizon, depth of 0 to 30 cm; B horizon, depth of 30 to 60 cm) of a 2,4-D-contaminated soil was investigated as a means of bioaugmentation. When the soil was amended with nutrients, plasmid transfer and enhanced degradation of 2,4-D were observed. These findings were most striking in the B horizon, where the indigenous bacteria were unable to degrade any of the 2,4-D (100 mg/kg of soil) during at least 22 days but where inoculation with either of the two plasmid donors resulted in complete 2,4-D degradation within 14 days. In contrast, in soils not amended with nutrients, inoculation of donors in the A horizon and subsequent formation of transconjugants (10(5) CFU/g of soil) could not increase the 2,4-D degradation rate compared to that of the noninoculated soil. However, donor inoculation in the nonamended B-horizon soil resulted in complete degradation of 2,4-D within 19 days, while no degradation at all was observed in noninoculated soil during 89 days. With plasmid \\tO\\npEMT1\\tgene\\n, this enhanced degradation seemed to be due only to transconjugants (10(5) CFU/g of soil), since the donor was already undetectable when degradation started. Denaturing gradient gel electrophoresis (DGGE) of 16S rRNA genes showed that inoculation of the donors was followed by a shift in the microbial community structure of the nonamended B-horizon soils. The new 16S rRNA gene fragments in the DGGE profile corresponded with the 16S rRNA genes of 2,4-D-degrading transconjugant colonies isolated on agar plates. This result indicates that the observed change in the community was due to proliferation of transconjugants formed in soil. Overall, this work clearly demonstrates that bioaugmentation can constitute an effective strategy for cleanup of soils which are poor in nutrients and microbial activity, such as those of the B horizon.\\tO\\n\\n',\n",
              " 'akNQ3DlOY.5VE2HxxEG85I2F2dhi-19625116.tsv': b'Microbial transformation of synthetic estrogen 17alpha-ethinylestradiol.\\tO\\n\\nAbstract\\tO\\n\\nNatural estrogens such as estrone, 17beta-estradiol, estriol, and the particularly recalcitrant synthetic estrogen 17alpha-ethinylestradiol used as oral contraceptive, accumulate in the environment and may give rise to health problems. The processes participating in their removal from soil, wastewater, water-sediments, groundwater-aquifer material, and wastewater or sewage treatment plant effluents may involve the action of bacterial and microbial consortia, and in some cases fungi and algae. This review discusses the different efficiencies of bacterial degradation of 17alpha-ethinylestradiol under aerobic and anaerobic conditions, the role of sulfate-, nitrate-, and iron-reducing conditions in anaerobic degradation, and the role of sorption. The participation of autotrophic ammonia oxidizing bacteria and heterotrophic bacteria in cometabolic degradation of estrogens, the estrogen-degrading action of ligninolytic fungi and their extracellular enzymes (\\tO\\nlignin peroxidase\\tenzyme\\n, \\tO\\nmanganese-dependent peroxidase\\tenzyme\\n, \\tO\\nversatile peroxidase\\tenzyme\\n, \\tO\\nlaccase\\tenzyme\\n), and of algae are discussed in detail.\\tO\\n\\n',\n",
              " 'akTMPuJstdvfFzK0APrYk1dzo31q-21396889.tsv': b'Identification and characterization of another 4-nitrophenol degradation gene cluster, nps, in Rhodococcus sp. strain PN1.\\tO\\n\\nAbstract\\tO\\n\\n4-Nitrophenol (4-NP) is a toxic compound formed in soil by the hydrolysis of organophosphorous pesticides, such as parathion. We previously reported the presence of the 4-NP degradation gene cluster (\\tO\\nnphRA1A2\\tgene\\n) in Rhodococcus sp. strain PN1, which encodes a two-component \\tO\\n4-NP hydroxylase\\tenzyme\\n system that oxidizes 4-NP into 4-nitrocatechol. In the current study, another gene cluster (\\tO\\nnpsC\\tgene\\n and \\tO\\nnpsRA2A1B\\tgene\\n) encoding a similar \\tO\\n4-NP hydroxylase\\tenzyme\\n system was cloned from strain PN1. The enzymes from this \\tO\\n4-NP hydroxylase\\tenzyme\\n system (\\tO\\nNpsA1\\tenzyme\\n and \\tO\\nNpsA2\\tenzyme\\n) were purified as histidine-tagged (His-) proteins and then characterized. His-\\tO\\nNpsA2\\tenzyme\\n showed NADH/FAD oxidoreductase activity, and His-\\tO\\nNpsA1\\tenzyme\\n showed 4-NP oxidizing activity in the presence of His-\\tO\\nNpsA2\\tenzyme\\n. In the 4-NP oxidation using the reconstituted enzyme system (His-\\tO\\nNpsA1\\tenzyme\\n and His-\\tO\\nNpsA2\\tenzyme\\n), hydroquinone (35% of 4-NP disappeared) and hydroxyquinol (59% of 4-NP disappeared) were detected in the presence of ascorbic acid as a reducing reagent, suggesting that, without the reducing reagent, 4-NP was converted into their oxidized forms, 1,4-benzoquinone and 2-hydroxy-1,4-benzoquinone. In addition, in the cell extract of recombinant Escherichia coli expressing \\tO\\nnpsB\\tgene\\n, a typical spectral change showing conversion of hydroxyquinol into maleylacetate was observed. These results indicate that this nps gene cluster, in addition to the nph gene cluster, is also involved in 4-NP degradation in strain PN1.\\tO\\n\\n',\n",
              " 'akmW.Z1YzVjwZLOuxDEqL8FcuU84-1885526.tsv': b'Purification and characterization of \\tO\\nbenzoate-coenzyme A ligase\\tenzyme\\n and 2-\\tO\\naminobenzoate-coenzyme A ligase\\tenzyme\\ns from a denitrifying Pseudomonas sp.\\tO\\n\\nAbstract\\tO\\n\\nThe enzymes catalyzing the formation of coenzyme A (CoA) thioesters of benzoate and 2-aminobenzoate were studied in a denitrifying Pseudomonas sp. anaerobically grown with these aromatic acids and nitrate as sole carbon and energy sources. Three different rather specific aromatic \\tO\\nacyl-CoA ligase\\tenzyme\\ns, E1, E2, and E3, were found which catalyze the formation of CoA thioesters of benzoate, fluorobenzoates, and 2-aminobenzoate. ATP is cleaved into AMP and pyrophosphate. The enzymes were purified, their N-terminal amino acid sequences were determined, and their catalytic and molecular properties were studied. Cells anaerobically grown on benzoate and nitrate contain one \\tO\\nCoA ligase\\tenzyme\\n (AMP forming) for benzoic acid (E1). It is a homodimer of Mr 120,000 which prefers benzoate as a substrate but shows some activity also with 2-aminobenzoate and fluorobenzoates, although with lower Km. Cells anaerobically grown on 2-aminobenzoate and nitrate contain three different CoA ligases for aromatic acids. The first one is identical with \\tO\\nbenzoate-CoA ligase\\tenzyme\\n (E1). The second enzyme is a \\tO\\n2-aminobenzoate-CoA ligase\\tenzyme\\n (E2). It is a monomer of Mr 60,000 which prefers 2-aminobenzoate but also activates benzoate, fluorobenzoates and, less effectively, 2-methylbenzoate, with lower affinities to the latter substrates. The enzymes E1 and E2 have similar activity levels; a third minor \\tO\\nCoA ligase\\tenzyme\\n activity is due to a different \\tO\\n2-aminobenzoate-CoA ligase\\tenzyme\\n. The enzyme (E3) is a monomer of Mr, 65,000 which 2-aminobenzoate pathway (U. Altenschmidt, C. Eckerskorn, and G. Fuchs, Eur. J. Biochem. 194:647-653, 1990); apparently, it is not completely repressed under anaerobic conditions and therefore also is induced to a small extent by 2-aminobenzoate under anaerobic growth conditions.\\tO\\n\\n',\n",
              " 'alFClOzZRZSztDE_huAmaKx0apz0-19002456.tsv': b'Molecular characteristics of xenobiotic-degrading sphingomonads.\\tO\\n\\nAbstract\\tO\\n\\nThe genus Sphingomonas (sensu latu) belongs to the alpha-Proteobacteria and comprises strictly aerobic chemoheterotrophic bacteria that are widespread in various aquatic and terrestrial environments. The members of this genus are often isolated and studied because of their ability to degrade recalcitrant natural and anthropogenic compounds, such as (substituted) biphenyl(s) and naphthalene(s), fluorene, (substituted) phenanthrene(s), pyrene, (chlorinated) diphenylether(s), (chlorinated) furan(s), (chlorinated) dibenzo-p-dioxin(s), carbazole, estradiol, polyethylene glycols, chlorinated phenols, nonylphenols, and different herbicides and pesticides. The metabolic versatility of these organisms suggests that they have evolved mechanisms to adapt quicker and/or more efficiently to the degradation of novel compounds in the environment than members of other bacterial genera. Comparative analyses demonstrate that sphingomonads generally use similar degradative pathways as other groups of microorganisms but deviate from competing microorganisms by the existence of multiple \\tO\\nhydroxylating oxygenase\\tenzyme\\ns and the conservation of specific gene clusters. Furthermore, there is increasing evidence for the existence of plasmids that only can be disseminated among sphingomonads and which undergo after conjugative transfer pronounced rearrangements.\\tO\\n\\n',\n",
              " 'alUk0HHVUEOu_iv56ZVCjPLj5B_8-1720168.tsv': b'Bacterial metabolism of 3-chloroacrylic acid.\\tO\\n\\nAbstract\\tO\\n\\nTwo bacterial strains were isolated with 3-chloroacrylic acid (CAA) as sole source of carbon and energy. Strain CAA1, a Pseudomonas cepacia sp., was capable of growth with only the cis-isomer of CAA. Strain CAA2, a coryneform bacterium, utilized both isomers of CAA as sole source of carbon and energy. Strain CAA1 contained \\tO\\ncis-CAA hydratase\\tenzyme\\n and strain CAA2 contained two hydratases, one with \\tO\\ncis-CAA hydratase\\tenzyme\\n activity and one with \\tO\\ntrans-CAA hydratase\\tenzyme\\n activity. The product of the hydratase activities with CAA was malonate semialdehyde. In both strains malonate semialdehyde was subsequently decarboxylated by a cofactor-independent decarboxylase yielding acetaldehyde and CO2.\\tO\\n\\n',\n",
              " 'amqJtPoDnPqhqD2Mz64n9qFnjLnS-30765135.tsv': b'Influence of graphene oxide and biochar on anaerobic degradation of petroleum hydrocarbons.\\tO\\n\\nAbstract\\tO\\n\\nThe anaerobic degradation of petroleum is an important process in natural environments. So far, few studies have considered the response of the microbial community to nanomaterials during this process. This study explored the potential effects of graphene oxide and biochar on the anaerobic degradation of petroleum hydrocarbons in long-term experiments. Cyclic voltammetry and electrochemical impedance spectroscopy indicated that the addition of carbon-based materials promoted the electrochemical activity of anaerobic cultures that degrade petroleum hydrocarbons. The maximum degradation rates for benzene, toluene, ethylbenzene, and xylene (BTEXs) in the cultures incubated for 10 weeks with graphene oxide (0.02 mg/L) and biochar (20 mg/L) were 76.5% and 77.6%, respectively. The maximum degradation rates of n-alkanes in the cultures incubated for 10 weeks with graphene oxide (2 mg/L) and biochar (100 mg/L) were 70.0% and 77.8%, respectively. The 16S rDNA copy numbers in the treatments with 0.02 mg/L graphene oxide and 20 mg/L biochar were significantly higher than others during the process (P < 0.05). In the 2nd week, the maximum copy numbers of the \\tO\\nmasD\\tgene\\n and \\tO\\nbamA\\tgene\\n genes in the treatments with biochar were 349 copies/mL (20 mg/L) and 422 copies/mL (20 mg/L), respectively, and in the treatments with graphene oxide were 289 copies/mL (0 mg/L) and 366 copies/mL (0.02 mg/L). The contents of carbon-based materials had slight effects on the microbial community structure, whereas the culture time had obvious effects. Paracoccus denitrificans, Pseudomonas aeruginosa, and Hydrogenophaga caeni were the dominant microorganisms in the culture systems under all treatments.\\tO\\n\\n',\n",
              " 'anEWNcuFgnjOgnlsSzzA3t6wyxxm-11319111.tsv': b'Degradation of substituted phenylurea herbicides by Arthrobacter globiformis strain D47 and characterization of a plasmid-associated hydrolase gene, \\tO\\npuhA\\tgene\\n.\\tO\\n\\nAbstract\\tO\\n\\nArthrobacter globiformis D47 was shown to degrade a range of substituted phenylurea herbicides in soil. This strain contained two plasmids of approximately 47 kb (\\tO\\npHRIM620\\tgene\\n) and 34 kb (pHRIM621). Plasmid-curing experiments produced plasmid-free strains as well as strains containing either the 47- or the 34-kb plasmid. The strains were tested for their ability to degrade diuron, which demonstrated that the degradative genes were located on the 47-kb plasmid. Studies on the growth of these strains indicated that the ability to degrade diuron did not offer a selective advantage to A. globiformis D47 on minimal medium designed to contain the herbicide as a sole carbon source. The location of the genes on a plasmid and a lack of selection would explain why the degradative phenotype, as with many other pesticide-degrading bacteria, can be lost on subculture. A 22-kb EcoRI fragment of plasmid \\tO\\npHRIM620\\tgene\\n was expressed in Escherichia coli and enabled cells to degrade diuron. Transposon mutagenesis of this fragment identified one open reading frame that was essential for enzyme activity. A smaller subclone of this gene (2.5 kb) expressed in E. coli coded for the protein that degraded diuron. This gene and its predicted protein sequence showed only a low level of protein identity (25% over ca. 440 amino acids) to other database sequences and was named after the enzyme it encoded, \\tO\\nphenylurea hydrolase\\tenzyme\\n (\\tO\\npuhA\\tgene\\n gene).\\tO\\n\\n',\n",
              " 'anqXB6uId6MDusV6KvxX_W7x0HEC-22544249.tsv': b'Cloning of a novel \\tO\\narylamidase\\tenzyme\\n gene from Paracoccus sp. strain FLN-7 that hydrolyzes amide pesticides.\\tO\\n\\nAbstract\\tO\\n\\nThe bacterial isolate Paracoccus sp. strain FLN-7 hydrolyzes amide pesticides such as diflubenzuron, propanil, chlorpropham, and dimethoate through amide bond cleavage. A gene, \\tO\\nampA\\tgene\\n, encoding a novel \\tO\\narylamidase\\tenzyme\\n that catalyzes the amide bond cleavage in the amide pesticides was cloned from the strain. \\tO\\nampA\\tgene\\n contains a 1,395-bp open reading frame that encodes a 465-amino-acid protein. \\tO\\nAmpA\\tgene\\n was expressed in Escherichia coli BL21 and homogenously purified using Ni-nitrilotriacetic acid affinity chromatography. \\tO\\nAmpA\\tgene\\n is a homodimer with an isoelectric point of 5.4. \\tO\\nAmpA\\tgene\\n displays maximum enzymatic activity at 40\\xc2\\xb0C and a pH of between 7.5 and 8.0, and it is very stable at pHs ranging from 5.5 to 10.0 and at temperatures up to 50\\xc2\\xb0C. \\tO\\nAmpA\\tgene\\n efficiently hydrolyzes a variety of secondary amine compounds such as propanil, 4-acetaminophenol, propham, chlorpropham, dimethoate, and omethoate. The most suitable substrate is propanil, with K(m) and k(cat) values of 29.5 \\xce\\xbcM and 49.2 s(-1), respectively. The benzoylurea insecticides (diflubenzuron and hexaflumuron) are also hydrolyzed but at low efficiencies. No cofactor is needed for the hydrolysis activity. \\tO\\nAmpA\\tgene\\n shares low identities with reported arylamidases (less than 23%), forms a distinct lineage from closely related arylamidases in the phylogenetic tree, and has different biochemical characteristics and catalytic kinetics with related arylamidases. The results in the present study suggest that \\tO\\nAmpA\\tgene\\n is a good candidate for the study of the mechanism for amide pesticide hydrolysis, genetic engineering of amide herbicide-resistant crops, and bioremediation of amide pesticide-contaminated environments.\\tO\\n\\n',\n",
              " 'ao0KyFqjKjEwaK8YvffA4rnFfU9q-681275.tsv': b'Biodegradation of acyclic isoprenoids by Pseudomonas species.\\tO\\n\\nAbstract\\tO\\n\\nThe ability of various pseudomonads to utilize acyclic isoprenoids as a sole carbon source was investigated. Tests for utilization of acyclic isoprenols such as citronellol and geraniol were complicated by toxic effects of these alcohols, and most species tested were killed by exposure to citronellol or geraniol (0.1%, vol/vol) in liquid culture. In the case of Pseudomonas citronellolis, sensitivity to isoprenols is reduced by prior induction of the isoprenoid degradative pathway via either growth on succinate in the presence of citronellol or growth on citronellic acid. For this species, citronellic acid proved to be the best isoprenoid growth substrate tested. Geraniol utilization as a taxonomic indicator for different subgroups of pseudomonads is discussed. Only a few of the species tested were able to utilize acyclic isoprenoids. Two species which utilize C10 acyclic isoprenoids, P. aeruginosa and P. mendocina, were shown to contain the inducible enzyme \\tO\\ngeranyl-coenzyme A carboxylase\\tenzyme\\n, one of the unique enzymes in the isoprenol degradative pathway known to occur in P. citronellolis. Of the species which utilized geranitol, none showed definite growth on the homologous C15 and C20 isoprenols.\\tO\\n\\n',\n",
              " 'ao7_nk1Xq7z9.O6P5YMtRjkcAB7a-22247165.tsv': b'SulE, a sulfonylurea herbicide de-esterification esterase from Hansschlegelia zhihuaiae S113.\\tO\\n\\nAbstract\\tO\\n\\nDe-esterification is an important degradation or detoxification mechanism of sulfonylurea herbicide in microbes and plants. However, the biochemical and molecular mechanisms of sulfonylurea herbicide de-esterification are still unknown. In this study, a novel \\tO\\nesterase\\tenzyme\\n gene, \\tO\\nsulE\\tgene\\n, responsible for sulfonylurea herbicide de-esterification, was cloned from Hansschlegelia zhihuaiae S113. The gene contained an open reading frame of 1,194 bp, and a putative signal peptide at the N terminal was identified with a predicted cleavage site between Ala37 and Glu38, resulting in a 361-residue mature protein. \\tO\\nSulE\\tgene\\n minus the signal peptide was synthesized in Escherichia coli BL21 and purified to homogeneity. \\tO\\nSulE\\tgene\\n catalyzed the de-esterification of a variety of sulfonylurea herbicides that gave rise to the corresponding herbicidally inactive parent acid and exhibited the highest catalytic efficiency toward thifensulfuron-methyl. \\tO\\nSulE\\tgene\\n was a dimer without the requirement of a cofactor. The activity of the enzyme was completely inhibited by Ag(+), Cd(2+), Zn(2+), methamidophos, and sodium dodecyl sulfate. A \\tO\\nsulE\\tgene\\n-disrupted mutant strain, \\xce\\x94\\tO\\nsulE\\tgene\\n, was constructed by insertion mutation. \\xce\\x94\\tO\\nsulE\\tgene\\n lost the de-esterification ability and was more sensitive to the herbicides than the wild type of strain S113, suggesting that \\tO\\nsulE\\tgene\\n played a vital role in the sulfonylurea herbicide resistance of the strain. The transfer of \\tO\\nsulE\\tgene\\n into Saccharomyces cerevisiae BY4741 conferred on it the ability to de-esterify sulfonylurea herbicides and increased its resistance to the herbicides. This study has provided an excellent candidate for the mechanistic study of sulfonylurea herbicide metabolism and detoxification through de-esterification, construction of sulfonylurea herbicide-resistant transgenic crops, and bioremediation of sulfonylurea herbicide-contaminated environments.\\tO\\n\\n',\n",
              " 'ap6QaMTWuCa9XFmImeAixzmBeZZO-23022948.tsv': b'Monitoring bioremediation of atrazine in soil microcosms using molecular tools.\\tO\\n\\nAbstract\\tO\\n\\nMolecular tools in microbial community analysis give access to information on catabolic potential and diversity of microbes. Applied in bioremediation, they could provide a new dimension to improve pollution control. This concept has been demonstrated in the study using atrazine as model pollutant. Bioremediation of the herbicide, atrazine, was analyzed in microcosm studies by bioaugmentation, biostimulation and natural attenuation. Genes from the atrazine degrading pathway \\tO\\natzA\\tgene\\n/B/C/D/E/F, \\tO\\ntrzN\\tgene\\n, and \\tO\\ntrzD\\tgene\\n were monitored during the course of treatment and results demonstrated variation in \\tO\\natzC\\tgene\\n, \\tO\\ntrzD\\tgene\\n and \\tO\\ntrzN\\tgene\\n genes with time. Change in copy number of \\tO\\ntrzN\\tgene\\n gene under different treatment processes was demonstrated by real-time PCR. The amplified \\tO\\ntrzN\\tgene\\n gene was cloned and sequence data showed homology to genes reported in Arthrobacter and Nocardioides. Results demonstrate that specific target genes can be monitored, quantified and correlated to degradation analysis which would help in predicting the outcome of any bioremediation strategy.\\tO\\n\\n',\n",
              " 'arDLI8bpK.7etpUqvs5jM.I3rNqu-11831474.tsv': b'Biodegradation of atrazine and related s-triazine compounds: from enzymes to field studies.\\tO\\n\\nAbstract\\tO\\n\\ns-Triazine ring compounds are common industrial chemicals: pesticides, resin intermediates, dyes, and explosives. The fate of these compounds in the environment is directly correlated with the ability of microbes to metabolize them. Microbes metabolize melamine and the triazine herbicides such as atrazine via enzyme-catalyzed hydrolysis reactions. Hydrolytic removal of substituents on the s-triazine ring is catalyzed by enzymes from the \\tO\\namidohydrolase\\tenzyme\\n superfamily and yields cyanuric acid as an intermediate. Cyanuric acid is hydrolytically processed to yield 3 mol each of ammonia and carbon dioxide. In those cases studied, the genes underlying the hydrolytic reactions are localized to large catabolic plasmids. One such plasmid, \\tO\\npADP-1\\tgene\\n from Pseudomonas sp. ADP, has been completely sequenced and contains the genes for atrazine catabolism. Insertion sequence elements play a role in constructing different atrazine catabolic plasmids in different bacteria. \\tO\\nAtrazine chlorohydrolase\\tenzyme\\n has been purified to homogeneity from two sources. Recombinant Escherichia coli strains expressing \\tO\\natrazine chlorohydrolase\\tenzyme\\n have been constructed and chemically cross-linked to generate catalytic particles used for atrazine remediation in soil. The method was used for cleaning up a spill of 1,000 pounds of atrazine to attain a level of herbicide acceptable to regulatory agencies.\\tO\\n\\n',\n",
              " 'asarkR8qt4kkCy31TNJmI5eqS0Za-22176786.tsv': b'Long-term dynamics of the atrazine mineralization potential in surface and subsurface soil in an agricultural field as a response to atrazine applications.\\tO\\n\\nAbstract\\tO\\n\\nThe dynamics of the atrazine mineralization potential in agricultural soil was studied in two soil layers (topsoil and at 35-45 cm depth) in a 3 years field trial to examine the long term response of atrazine mineralizing soil populations to atrazine application and intermittent periods without atrazine and the effect of manure treatment on those processes. In topsoil samples, (14)C-atrazine mineralization lag times decreased after atrazine application and increased with increasing time after atrazine application, suggesting that atrazine application resulted into the proliferation of atrazine mineralizing microbial populations which decayed when atrazine application stopped. Decay rates appeared however much slower than growth rates. Atrazine application also resulted into the increase of the atrazine mineralization potential in deeper layers which was explained by the growth on leached atrazine as measured in soil leachates recovered from that depth. However, no decay was observed during intermittent periods without atrazine application in the deeper soil layer. \\tO\\natzA\\tgene\\n and \\tO\\ntrzN\\tgene\\n gene quantification confirmed partly the growth and decay of the atrazine degrading populations in the soil and suggested that especially \\tO\\ntrzN\\tgene\\n bearing populations are the dominant atrazine degrading populations in both topsoil and deeper soil. Manure treatment only improved the atrazine mineralization rate in deeper soil layers. Our results point to the importance of the atrazine application history on a field and suggests that the long term survival of atrazine degrading populations after atrazine application enables them to rapidly proliferate once atrazine is again applied.\\tO\\n\\n',\n",
              " 'asrNy8uPI86WRMSZOyIZ15gB_nI8-25582347.tsv': b'Genomic and metabolic analysis of fluoranthene degradation pathway in Celeribacter indicus P73T.\\tO\\n\\nAbstract\\tO\\n\\nCeleribacter indicus P73(T), isolated from deep-sea sediment from the Indian Ocean, is capable of degrading a wide range of polycyclic aromatic hydrocarbons (PAHs) and is the first fluoranthene-degrading bacterium within the family Rhodobacteraceae. Here, the complete genome sequence of strain P73(T) is presented and analyzed. Besides a 4.5-Mb circular chromosome, strain P73(T) carries five plasmids, and encodes 4827 predicted protein-coding sequences. One hundred and thirty-eight genes, including 14 dioxygenase genes, were predicted to be involved in the degradation of aromatic compounds, and most of these genes are clustered in four regions. P73_0346 is the first \\tO\\nfluoranthene 7,8-dioxygenase\\tenzyme\\n to be discovered and the first \\tO\\nfluoranthene dioxygenase\\tenzyme\\n within the toluene/biphenyl family. The degradative genes in regions B and D in P73(T) are absent in Celeribacter baekdonensis B30, which cannot degrade PAHs. Four intermediate metabolites [acenaphthylene-1(2H)-one, acenaphthenequinone, 1,2-dihydroxyacenaphthylene, and 1,8-naphthalic anhydride] of fluoranthene degradation by strain P73(T) were detected as the main intermediates, indicating that the degradation of fluoranthene in P73(T) was initiated by dioxygenation at the C-7,8 positions. Based on the genomic and metabolitic results, we propose a C-7,8 dioxygenation pathway in which fluoranthene is mineralized to TCA cycle intermediates.\\tO\\n\\n',\n",
              " 'at2euSJ6UnNz47kXoJpW0dqA1x3y-23186529.tsv': b'Biodegradation of butachlor by Rhodococcus sp. strain B1 and purification of its hydrolase (ChlH) responsible for N-dealkylation of chloroacetamide herbicides.\\tO\\n\\nAbstract\\tO\\n\\nRhodococcus sp. strain B1 could degrade 100 mg/L butachlor within 5 days. Butachlor was first hydrolyzed by strain B1 through N-dealkylation, which resulted in the production of butoxymethanol and 2-chloro-N-(2,6-dimethylphenyl)acetamide. Butoxymethanol could be further degraded and utilized as the carbon source for the growth of strain B1, whereas 2-chloro-N-(2,6-dimethylphenyl)acetamide could not be degraded further. The hydrolase designated \\tO\\nChlH\\tenzyme\\n, responsible for the N-dealkylation of the side chain of butachlor, was purified 185.1-fold to homogeneity with 16.1% recovery. The optimal pH and temperature of \\tO\\nChlH\\tenzyme\\n were observed to be 7.0-7.5 and 30 \\xc2\\xb0C, respectively. This enzyme was also able to catalyze the N-dealkylation of other chloroacetamide herbicides; the catalytic efficiency followed the order alachlor > acetochlor >butachlor > pretilachlor, which indicated that the alkyl chain length influenced the N-dealkylation of the chloroacetamide herbicides. This is the first report on the biodegradation of chloroacetamide herbicides at the enzyme level.\\tO\\n\\n',\n",
              " 'atSqjJMKCyOI03pI2eklLyIFEk7q-1938949.tsv': b'Potential DNA slippage structures acquired during evolutionary divergence of Acinetobacter calcoaceticus chromosomal benABC and Pseudomonas putida TOL pWW0 plasmid \\tO\\nxylXYZ\\tgene\\n, genes encoding \\tO\\nbenzoate dioxygenase\\tenzyme\\ns.\\tO\\n\\nAbstract\\tO\\n\\nThe \\tO\\nxylXYZ\\tgene\\n DNA region is carried on the TOL pWW0 plasmid in Pseudomonas putida and encodes a \\tO\\nbenzoate dioxygenase\\tenzyme\\n with broad substrate specificity. The DNA sequence of the region is presented and compared with benABC, the chromosomal region encoding the \\tO\\nbenzoate dioxygenase\\tenzyme\\n of Acinetobacter calcoaceticus. Corresponding genes from the two biological sources share common ancestry: comparison of aligned \\tO\\nXylX-BenA\\tgene\\n, \\tO\\nXylY-BenB\\tgene\\n, and \\tO\\nXylZ-BenC\\tgene\\n amino acid sequences revealed respective identities of 58.3, 61.3, and 53%. The aligned genes have diverged to assume G+C contents that differ by 14.0 to 14.9%. Usage of the unusual arginine codons AGA and AGG appears to have been selected in the P. putida \\tO\\nxylX\\tgene\\n gene as it diverged from the ancestor it shared with A. calcoaceticus benA. Homologous A. calcoaceticus and P. putida genes exhibit different patterns of DNA sequence repetition, and analysis of one such pattern suggests that mutations creating different DNA slippage structures made a significant contribution to the evolutionary divergence of \\tO\\nxylX\\tgene\\n.\\tO\\n\\n',\n",
              " 'av5KK4Dy7_iLJ7bZjPYS8aW2_utK-1987135.tsv': b'Cloning and characterization of plasmid-encoded genes for the degradation of 1,2-dichloro-, 1,4-dichloro-, and 1,2,4-trichlorobenzene of Pseudomonas sp. strain P51.\\tO\\n\\nAbstract\\tO\\n\\nPseudomonas sp. strain P51 is able to use 1,2-dichlorobenzene, 1,4-dichlorobenzene, and 1,2,4-trichlorobenzene as sole carbon and energy sources. Two gene clusters involved in the degradation of these compounds were identified on a catabolic plasmid, \\tO\\npP51\\tgene\\n, with a size of 110 kb by using hybridization. They were further characterized by cloning in Escherichia coli, Pseudomonas putida KT2442, and Alcaligenes eutrophus JMP222. Expression studies in these organisms showed that the upper-pathway genes (\\tO\\ntcbA\\tgene\\n and \\tO\\ntcbB\\tgene\\n) code for the conversion of 1,2-dichlorobenzene and 1,2,4-trichlorobenzene to 3,4-dichlorocatechol and 3,4,6-trichlorocatechol, respectively, by means of a dioxygenase system and a dehydrogenase. The lower-pathway genes have the order \\tO\\ntcbC\\tgene\\n-\\tO\\ntcbD\\tgene\\n-\\tO\\ntcbE\\tgene\\n and encode a \\tO\\ncatechol 1,2-dioxygenase II\\tenzyme\\n, a \\tO\\ncycloisomerase II\\tenzyme\\n, and a \\tO\\nhydrolase II\\tenzyme\\n, respectively. The combined action of these enzymes degrades 3,4-dichlorocatechol and 3,4,6-trichlorocatechol to a chloromaleylacetic acid. The release of one chlorine atom from 3,4-dichlorocatechol takes place during lactonization of 2,3-dichloromuconic acid.\\tO\\n\\n',\n",
              " 'awCM8T0i1WF_qAGxsa.VbPiG6ESG-24928877.tsv': b'Novel three-component Rieske \\tO\\nnon-heme iron oxygenase\\tenzyme\\n system catalyzing the N-dealkylation of chloroacetanilide herbicides in sphingomonads DC-6 and DC-2.\\tO\\n\\nAbstract\\tO\\n\\nSphingomonads DC-6 and DC-2 degrade the chloroacetanilide herbicides alachlor, acetochlor, and butachlor via N-dealkylation. In this study, we report a three-component Rieske \\tO\\nnon-heme iron oxygenase\\tenzyme\\n (RHO) system catalyzing the N-dealkylation of these herbicides. The oxygenase component gene \\tO\\ncndA\\tgene\\n is located in a transposable element that is highly conserved in the two strains. \\tO\\nCndA\\tgene\\n shares 24 to 42% amino acid sequence identities with the oxygenase components of some RHOs that catalyze N- or O-demethylation. Two putative [2Fe-2S] ferredoxin genes and one glutathione reductase (GR)-type reductase gene were retrieved from the genome of each strain. These genes were not located in the immediate vicinity of \\tO\\ncndA\\tgene\\n. The four ferredoxins share 64 to 72% amino acid sequence identities to the ferredoxin component of \\tO\\ndicamba O-demethylase\\tenzyme\\n (DMO), and the two reductases share 62 to 65% amino acid sequence identities to the reductase component of DMO. \\tO\\ncndA\\tgene\\n, the four ferredoxin genes, and the two reductases genes were expressed in Escherichia coli, and the recombinant proteins were purified using Ni-affinity chromatography. The individual components or the components in pairs displayed no activity; the enzyme mixture showed \\tO\\nN-dealkylase\\tenzyme\\n activities toward alachlor, acetochlor, and butachlor only when \\tO\\nCndA\\tgene\\n-\\tO\\nHis6\\tgene\\n was combined with one of the four ferredoxins and one of the two reductases, suggesting that the enzyme consists of three components, a \\tO\\nhomo-oligomer oxygenase\\tenzyme\\n, a [2Fe-2S] ferredoxin, and a GR-type reductase, and \\tO\\nCndA\\tgene\\n has a low specificity for the electron transport component (ETC). The \\tO\\nN-dealkylase\\tenzyme\\n utilizes NADH, but not NADPH, as the electron donor.\\tO\\n\\n',\n",
              " 'axkF9gyJczwVDWPUPFdllX89G2yi-22938719.tsv': b'Micropollutant biotransformation kinetics associate with WWTP process parameters and microbial community characteristics.\\tO\\n\\nAbstract\\tO\\n\\nThe objective of this work was to identify relevant wastewater treatment plant (WWTP) parameters and underlying microbial processes that influence the biotransformation of a diverse set of micropollutants. To do this, we determined biotransformation rate constants for ten organic micropollutants in batch reactors seeded with activated sludge from ten diverse WWTPs. The estimated biotransformation rate constants for each compound ranged between one and four orders of magnitude among the ten WWTPs. The biotransformation rate constants were tested for statistical associations with various WWTP process parameters, \\tO\\namoA\\tgene\\n transcript abundance, and \\tO\\nacetylene-inhibited monooxygenase\\tenzyme\\n activity. We determined that (i) ammonia removal associates with oxidative micropollutant biotransformation reaction rates; (ii) archaeal but not bacterial \\tO\\namoA\\tgene\\n transcripts associate with both ammonia removal and oxidative micropollutant biotransformation reaction rates; and (iii) the activity of \\tO\\nacetylene-inhibited monooxygenase\\tenzyme\\ns (including \\tO\\nammonia monooxygenase\\tenzyme\\n) associates with ammonia removal and the biotransformation rate of isoproturon, but does not associate with all oxidative micropollutant biotransformations. In combination, these results lead to the conclusion that ammonia removal and \\tO\\namoA\\tgene\\n transcript abundance can potentially be predictors of oxidative micropollutant biotransformation reactions, but that the biochemical mechanism is not necessarily linked to \\tO\\nammonia monooxygenase\\tenzyme\\n activity.\\tO\\n\\n',\n",
              " 'ay109tu2e7Y4hDEQbQsupvVzOxqi-28762129.tsv': b'Phytodetoxification of TNT by transplastomic tobacco (Nicotiana tabacum) expressing a bacterial \\tO\\nnitroreductase\\tenzyme\\n.\\tO\\n\\nAbstract\\tO\\n\\nExpression of the bacterial \\tO\\nnitroreductase\\tenzyme\\n gene, \\tO\\nnfsI\\tgene\\n, in tobacco plastids conferred the ability to detoxify TNT. The toxic pollutant 2,4,6-trinitrotoluene (TNT) is recalcitrant to degradation in the environment. Phytoremediation is a potentially low cost remediation technique that could be applied to soil contaminated with TNT; however, progress is hindered by the phytotoxicity of this compound. Previous studies have demonstrated that plants transformed with the bacterial \\tO\\nnitroreductase\\tenzyme\\n gene, \\tO\\nnfsI\\tgene\\n have increased ability to tolerate and detoxify TNT. It has been proposed that plants engineered to express \\tO\\nnfsI\\tgene\\n could be used to remediate TNT on military ranges, but this could require steps to mitigate transgene flow to wild populations. To address this, we have developed \\tO\\nnfsI\\tgene\\n transplastomic tobacco (Nicotiana tabacum L.) to reduce pollen-borne transgene flow. Here we have shown that when grown on solid or liquid media, the transplastomic tobacco expressing \\tO\\nnfsI\\tgene\\n were significantly more tolerant to TNT, produced increased biomass and removed more TNT from the media than untransformed plants. Additionally, transplastomic plants expressing \\tO\\nnfsI\\tgene\\n regenerated with high efficiency when grown on medium containing TNT, suggesting that \\tO\\nnfsI\\tgene\\n and TNT could together be used to provide a selectable screen for plastid transformation.\\tO\\n\\n',\n",
              " 'aybjIDVZs86m9ZxgZrF128jpHnVy-20401686.tsv': b'Screening for and isolation and identification of malathion-degrading bacteria: cloning and sequencing a gene that potentially encodes the malathion-degrading enzyme, carboxylestrase in soil bacteria.\\tO\\n\\nAbstract\\tO\\n\\nFive malathion-degrading bacterial strains were enriched and isolated from soil samples collected from different agricultural sites in Cairo, Egypt. Malathion was used as a sole source of carbon (50 mg/l) to enumerate malathion degraders, which were designated as IS1, IS2, IS3, IS4, and IS5. They were identified, based on their morphological and biochemical characteristics, as Pseudomonas sp., Pseudomonas putida, Micrococcus lylae, Pseudomonas aureofaciens, and Acetobacter liquefaciens, respectively. IS1 and IS2, which showed the highest degrading activity, were selected for further identification by partial sequence analysis of their 16S rRNA genes. The 16S rRNA gene of IS1 shared 99% similarity with that of Alphaprotoebacterium BAL284, while IS2 scored 100% similarity with that of Pseudomonas putida 32zhy. Malathion residues almost completely disappeared within 6 days of incubation in IS2 liquid cultures. LC/ESI-MS analysis confirmed the degradation of malathion to malathion monocarboxylic and dicarboxylic acids, which formed as a result of \\tO\\ncarboxylesterase\\tenzyme\\n activity. A \\tO\\ncarboxylesterase\\tenzyme\\n gene (CE) was amplified from the IS2 genome by using specifically designed PCR primers. The sequence analysis showed a significant similarity to a known CE gene in different Pseudomonas sp. We report here the isolation of a new malathion-degrading bacteria from soils in Egypt that may be very well adapted to the climatic and environmental conditions of the country. We also report the partial cloning of a new CE gene. Due to their high biodegradation activity, the bacteria isolated from this work merit further study as potential biological agents for the remediation of soil, water, or crops contaminated with the pesticide malathion.\\tO\\n\\n',\n",
              " 'azow9xfl_vua1heIv1oyGayw3.CW-1390687.tsv': b'Primary structure of an aliphatic nitrile-degrading enzyme, \\tO\\naliphatic nitrilase\\tenzyme\\n, from Rhodococcus rhodochrous K22 and expression of its gene and identification of its active site residue.\\tO\\n\\nAbstract\\tO\\n\\nPeptides obtained by cleavage of a Rhodococcus rhodochrous K22 nitrilase, which acts on aliphatic nitriles such as acrylonitrile, crotonitrile, and glutaronitrile, have been sequenced. The data allowed the design of oligonucleotide probes which were used to clone a nitrilase encoding gene. Plasmid pNK21, in which 2.05-kb sequence covering the region encoding the nitrilase was was placed under the control of the lac promoter, directed overproduction of enzymatically active nitrilase in response to addition of isopropyl beta-D-thiogalactopyranoside in Escherichia coli. Sodium dodecyl sulfate-polyacrylamide gel electrophoresis of the cell extract showed that the amount of nitrilase was about 40% of the total soluble proteins, leading to the establishment of a simple purification of the nitrilase. The nucleotide sequence of the nitrilase gene predicts a protein composed of 383 amino acids (M(r) = 42,275), including only one cysteine. The amino acid sequence homology between the Rhodococcus nitrilase and the Klebsiella ozaenae bromoxynil nitrilase [Stalker et al. (1988) J. Biol. Chem. 263, 6310-6314] was 38.3%, and a unique cysteinyl residue (Cys-170) in the former nitrilase was conserved at the corresponding position in the latter nitrilase. Cys-170 of the Rhodococcus nitrilase was replaced by Ala or Ser by site-directed mutagenesis. Both mutations resulted in the complete loss of nitrilase activity, clearly indicating that this cysteinyl residue is essential for the catalytic activity.\\tO\\n\\n',\n",
              " 'azrOx67dvJt8Z_RDwJlO6UaiI5py-1768085.tsv': b'Propachlor degradation by a soil bacterial community.\\tO\\n\\nAbstract\\tO\\n\\nSoil from a pesticide disposal site was used to enrich for microorganisms that degraded the acylanilide herbicide propachlor (2-chloro-N-isopropylacetanilide). After seven transfers of the enrichment, the culture contained about six strains. The highest yield of microbial biomass occurred if just two of these isolates, strains DAK3 and MAB2, were inoculated into a mineral salts medium containing propachlor. When only strain DAK3 was grown on propachlor, a metabolite (2-chloro-N-isopropylacetamide) was released into the medium. Strain MAB2 could grow on this metabolite. The results of morphological and physiological tests suggest that strains DAK3 and MAB2 most closely resemble species belonging to the genera Moraxella and Xanthobacter, respectively. Strain DAK3 can respire and grow on N-substituted acylanilides containing methyl, ethyl, or isopropyl substitutions, but is incapable of respiration or growth on acetanilide, aniline, or the acylanilide herbicides alachlor and metolachlor. Strain DAK3 appears to use the aromatic C atoms of propachlor for growth, as suggested by the growth yield on propachlor and the induction of \\tO\\ncatechol 2,3-oxygenase\\tenzyme\\n activity in acylanilide-grown cells.\\tO\\n\\n'}"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 4
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "K9Lx4mOFbNOA",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 187
        },
        "outputId": "3102917c-66db-4d9d-ccf6-e2f5416c6b9f"
      },
      "source": [
        "import nltk, re, string, unicodedata, inflect, glob, os\n",
        "import re\n",
        "from nltk.corpus.reader.plaintext import PlaintextCorpusReader\n",
        "import numpy as np \n",
        "import pandas as pd                                 #for data manipulation and analysis\n",
        "from nltk.corpus import stopwords                   #Stopwords corpus\n",
        "from nltk.stem import PorterStemmer \n",
        "from nltk.tokenize import RegexpTokenizer\n",
        "from string import punctuation\n",
        "from nltk import word_tokenize, sent_tokenize\n",
        "from nltk.corpus import stopwords\n",
        "from nltk.stem import LancasterStemmer, WordNetLemmatizer\n",
        "from nltk.tokenize import TreebankWordTokenizer\n",
        "nltk.download('stopwords')\n",
        "nltk.download('punkt')\n",
        "nltk.download('wordnet')\n",
        "nltk.download('averaged_perceptron_tagger')"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
            "[nltk_data]   Unzipping corpora/stopwords.zip.\n",
            "[nltk_data] Downloading package punkt to /root/nltk_data...\n",
            "[nltk_data]   Unzipping tokenizers/punkt.zip.\n",
            "[nltk_data] Downloading package wordnet to /root/nltk_data...\n",
            "[nltk_data]   Unzipping corpora/wordnet.zip.\n",
            "[nltk_data] Downloading package averaged_perceptron_tagger to\n",
            "[nltk_data]     /root/nltk_data...\n",
            "[nltk_data]   Unzipping taggers/averaged_perceptron_tagger.zip.\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 5
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rAg3-xZJbaRG",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "8ff9d0d9-6acd-403f-d9d2-7271f75b4fd8"
      },
      "source": [
        "for dirname, _, filenames in os.walk('/content/an_data'):\n",
        "    for filename in filenames:\n",
        "        print(os.path.join(dirname, filename))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "/content/an_data/aAvpUplV__m4CU1IcWvDMWWuHY0y-18606539.tsv\n",
            "/content/an_data/a_rI55cZbMHdHTM0cNz7wVUGh.YG-15240330.tsv\n",
            "/content/an_data/aO33b5hse6bt4xZVSrrcQgUo9jO4-28006730.tsv\n",
            "/content/an_data/a_fUcGCWV_jfoorNEfY9ZLrgTdzy-2310183.tsv\n",
            "/content/an_data/aXV8WPhnnXaKBDRW.KaFM9h3JK_8-24721596.tsv\n",
            "/content/an_data/asarkR8qt4kkCy31TNJmI5eqS0Za-22176786.tsv\n",
            "/content/an_data/a1pDgMCNLrLbKkZAlAdRBt2r.59S-28664326.tsv\n",
            "/content/an_data/aAMflZPODkJwGSZvpxMEZCCrxhLy-31394342.tsv\n",
            "/content/an_data/ajgdpPcEt7PN4OhUnb8AiAxsqqku-30326388.tsv\n",
            "/content/an_data/a.g3qPkC.UM7xF33mZVi5sVpKNDm-1123316.tsv\n",
            "/content/an_data/a3HbOFyvJXWX.dfWfO8dHsBvWdxi-1622259.tsv\n",
            "/content/an_data/aeRSJ.g51yv44AhUWaZmaS3zrcwS-1416975.tsv\n",
            "/content/an_data/anEWNcuFgnjOgnlsSzzA3t6wyxxm-11319111.tsv\n",
            "/content/an_data/azow9xfl_vua1heIv1oyGayw3.CW-1390687.tsv\n",
            "/content/an_data/agM5xp_DbJltbmulA8.crpkhzCjC-30647456.tsv\n",
            "/content/an_data/aYcUnBgYZ0CMSJ0ltle2cqxMkSES-23811502.tsv\n",
            "/content/an_data/aM4A2nEttHUsrp616fh2R6Qrk.ru-1268229.tsv\n",
            "/content/an_data/aMWcBtBI2Z2a0njuP81Tr3mX6I0m-28321487.tsv\n",
            "/content/an_data/ahwTYdMzdaIWKB3vd5dHo7Gtzk0m-28363960.tsv\n",
            "/content/an_data/a64iRnUPdE2jbaKQELV_UNrG2Zxi-1526465.tsv\n",
            "/content/an_data/aD3qB7nGh9V6zv33OQXsumxeJ7OC-1254564.tsv\n",
            "/content/an_data/aW12Y2HqOS2s.HdUDpjI4tCPW.aK-1444422.tsv\n",
            "/content/an_data/aFqlwZAcl3EaVCR._itfw0Bg7ecG-2394680.tsv\n",
            "/content/an_data/ahsvvoMu84Zjr7Rhu16yMIALmbNW-21615634.tsv\n",
            "/content/an_data/a1O3245an7NEHwTx7mcFWQNFyto4-22991035.tsv\n",
            "/content/an_data/aNUC2kQ9JHXhzRelFXkqKr8nCllq-1577754.tsv\n",
            "/content/an_data/at2euSJ6UnNz47kXoJpW0dqA1x3y-23186529.tsv\n",
            "/content/an_data/aHUtck9wjaHujHIYEMKTrb8lB3zu-8005.tsv\n",
            "/content/an_data/aWjQPKPgXqYQt4s0er.XVq9ksHf4-30324341.tsv\n",
            "/content/an_data/aS4kJlp8QNdWM003oQeKhHPXfZ34-16329946.tsv\n",
            "/content/an_data/aNtTIxCXZIoCbW2sw7pg6qma8hbK-15883866.tsv\n",
            "/content/an_data/a19opHP85j5SpAVTkODOJ4LNVwxa-1610174.tsv\n",
            "/content/an_data/afYImpCZ9lNrPIWIyzjfgfu2pcbu-26273258.tsv\n",
            "/content/an_data/a3Qc5NWvIJi.yuVfsskKvM2Tp4JG-24668245.tsv\n",
            "/content/an_data/a5jgtjZkpBoX6xpCfZU3376Kzlaq-234247.tsv\n",
            "/content/an_data/aCL5B7lodNpiF9VFK56lfoXwR6jK-26308673.tsv\n",
            "/content/an_data/a2l7vzT6_qWNbgnP8DGLDWCMEgyO-21468717.tsv\n",
            "/content/an_data/aTjaxECCPKlNaQUPN9wzGhuTosMG-20305027.tsv\n",
            "/content/an_data/aHlTwI8ihDgYtvTxoIMATAE97Y10-23076592.tsv\n",
            "/content/an_data/aAR7COUYPwWwooLSEEo7umBWUrEC-22003008.tsv\n",
            "/content/an_data/ai.4gEPfAl8ZZ1zaemJCkzjRZIcS-1587481.tsv\n",
            "/content/an_data/aBN1Qmxk3Xj19lwi8__Xx43FTPcO-29563.tsv\n",
            "/content/an_data/aiyTKDM2vSKVPC57nYa4PXKHMzGm-18451.tsv\n",
            "/content/an_data/aeLjOQVjTmU6CxOxaC6IOPkD15ai-29150672.tsv\n",
            "/content/an_data/asrNy8uPI86WRMSZOyIZ15gB_nI8-25582347.tsv\n",
            "/content/an_data/afHujGvs_M.1SLW0m4OXadVRJvT8-31549657.tsv\n",
            "/content/an_data/aSnQ6wHh4xWTKzoJ8bT4HhzeOVPy-23586778.tsv\n",
            "/content/an_data/ajipNizGkakHghlktMYLqC1CDxdO-10919784.tsv\n",
            "/content/an_data/aioL7f.VZae8CICcRDJxPjZZ9i.u-23066709.tsv\n",
            "/content/an_data/aBRIckae0Mpac9CxsT6pSNnkHqf0-26776948.tsv\n",
            "/content/an_data/aAjTJkYkdurOuzdNS2EqvP8DTTfS-1447132.tsv\n",
            "/content/an_data/aYzMIzEZRsCOVevEzvdU9zR6IuCa-19709256.tsv\n",
            "/content/an_data/adHR6sQH7bCPevVwArbdhgPTsT58-24676750.tsv\n",
            "/content/an_data/alUk0HHVUEOu_iv56ZVCjPLj5B_8-1720168.tsv\n",
            "/content/an_data/azrOx67dvJt8Z_RDwJlO6UaiI5py-1768085.tsv\n",
            "/content/an_data/awCM8T0i1WF_qAGxsa.VbPiG6ESG-24928877.tsv\n",
            "/content/an_data/alFClOzZRZSztDE_huAmaKx0apz0-19002456.tsv\n",
            "/content/an_data/aM2.ASbdUakhKQsFMULm0m.241eu-24898178.tsv\n",
            "/content/an_data/aVlBjfk8CtLMfZEIJbKUN7dxO2DW-1892384.tsv\n",
            "/content/an_data/aiJjGYw7ugb4aPZPKqQPn.7BC6Jq-1447127.tsv\n",
            "/content/an_data/ay109tu2e7Y4hDEQbQsupvVzOxqi-28762129.tsv\n",
            "/content/an_data/aJU0mGpHnbYRcwLiK45Pyiv8kPBS-24239819.tsv\n",
            "/content/an_data/aVCsGoaq69gyqXoQPrYR_Xh_8YHy-19497847.tsv\n",
            "/content/an_data/a74YbCmMNjsB.SiEQcIn2iQ3Q70m-8226980.tsv\n",
            "/content/an_data/ah7ZS_ptJZlahOrm7G4ReJWe5mFy-1429434.tsv\n",
            "/content/an_data/aVpYWI_FIBmNF9Q.wYEuetl8lCLi-1444373.tsv\n",
            "/content/an_data/aXadYOMx.9g1YFCDL6FE4VwsFQ9G-29886364.tsv\n",
            "/content/an_data/aVDY7Ev59m70WiI7f_a67hScGBiS-2034671.tsv\n",
            "/content/an_data/aHg0m7EJoE7Lqw1a3K8dzuvFIwIu-23560662.tsv\n",
            "/content/an_data/akTMPuJstdvfFzK0APrYk1dzo31q-21396889.tsv\n",
            "/content/an_data/aBP4ogv6rktF6czR15ipvZka18oS-1885512.tsv\n",
            "/content/an_data/akmW.Z1YzVjwZLOuxDEqL8FcuU84-1885526.tsv\n",
            "/content/an_data/aybjIDVZs86m9ZxgZrF128jpHnVy-20401686.tsv\n",
            "/content/an_data/a_gwKjTrMLtrwW4Ry3U8VFfZu_Lu-14706746.tsv\n",
            "/content/an_data/aYt2ZK7qs9azR0119c8jCx.UWB3C-20088886.tsv\n",
            "/content/an_data/ao0KyFqjKjEwaK8YvffA4rnFfU9q-681275.tsv\n",
            "/content/an_data/arDLI8bpK.7etpUqvs5jM.I3rNqu-11831474.tsv\n",
            "/content/an_data/a237AbH6ipbjuFjK_fKc7jngiNwS-25025097.tsv\n",
            "/content/an_data/aBlYzaPGSQp3xU8t_IxrAGMCyvrC-23454590.tsv\n",
            "/content/an_data/av5KK4Dy7_iLJ7bZjPYS8aW2_utK-1987135.tsv\n",
            "/content/an_data/a3cjro1JuxuGsMW3YFdMxf9Hv7KK-21531434.tsv\n",
            "/content/an_data/aiZwtrEfgPnqN6Sj4buoSESUOhmK-27005792.tsv\n",
            "/content/an_data/amqJtPoDnPqhqD2Mz64n9qFnjLnS-30765135.tsv\n",
            "/content/an_data/a0VbJUpV5YemaEUU65sXwnHgUeo0-22392141.tsv\n",
            "/content/an_data/atSqjJMKCyOI03pI2eklLyIFEk7q-1938949.tsv\n",
            "/content/an_data/aArRcX1.yw2C779LPKZEQgkPFd5O-1961727.tsv\n",
            "/content/an_data/a8.btHMLFHQHGdc7nUKzihjPP9y0-1592816.tsv\n",
            "/content/an_data/afJ8f.ek_mLp6PuUrK9e4uiRMQc8-2219955.tsv\n",
            "/content/an_data/a_40v3K2FGdNYfaS_6S6Jsri77.C-22635998.tsv\n",
            "/content/an_data/a97SQMyLZ_YUFwcnZKRlxeuttZyC-16391099.tsv\n",
            "/content/an_data/aEHh.KmFDmnx4LqVJgZmN2.IU97y-22854895.tsv\n",
            "/content/an_data/af5Vd1_V.IZVv2fy8SRV3TX_iUQ8-28401633.tsv\n",
            "/content/an_data/aPjCyemv4fUpmXpk7VoVwjyBDs_u-1371658.tsv\n",
            "/content/an_data/a9y0qNOVEp4ztjwkaTBe1Pz6eETa-2067577.tsv\n",
            "/content/an_data/aBgzC9zRQ5gI.RRbvzcyxs3v.erG-24123738.tsv\n",
            "/content/an_data/ao7_nk1Xq7z9.O6P5YMtRjkcAB7a-22247165.tsv\n",
            "/content/an_data/aGGjZft5cKAxMFYpf3D1CHm4eTP4-925012.tsv\n",
            "/content/an_data/aFQ5GrPy0N9DBLuocsDuRxtTrpWq-1327782.tsv\n",
            "/content/an_data/aCdU83YXibjOzSJ8u3BvmyF344eS-31445334.tsv\n",
            "/content/an_data/aU.B2YOcXQQ0c__tKQlZmVhxL9.W-24862007.tsv\n",
            "/content/an_data/aMecN20D4rUvYkTsLUdOC2E6wagm-27984820.tsv\n",
            "/content/an_data/a1G9_mwl5127bHyxwtrLqRP9Ep.u-20406281.tsv\n",
            "/content/an_data/ap6QaMTWuCa9XFmImeAixzmBeZZO-23022948.tsv\n",
            "/content/an_data/aLqTZF8guoL3.vLWZwhCJz03WWyG-22307296.tsv\n",
            "/content/an_data/aKEgV7EueJsNLXQ4_VvYw9dxQ7yu-1453953.tsv\n",
            "/content/an_data/a4.M_gLNftxxNQ_2aZgztZhuJPl8-31071671.tsv\n",
            "/content/an_data/aNYIc2VJb1lvg_2sFDoCarzYacLS-17117994.tsv\n",
            "/content/an_data/adFPfP5Za2Tqr7GICnCj4szpgbR0-21974851.tsv\n",
            "/content/an_data/akNQ3DlOY.5VE2HxxEG85I2F2dhi-19625116.tsv\n",
            "/content/an_data/a0G6qKR9kpJ.vYu5sRpUDK56l3Le-21418495.tsv\n",
            "/content/an_data/aVXBoAqrkI_nbpmM_mu.Br9fKHzC-19251892.tsv\n",
            "/content/an_data/aV17kDPU_zg1fV8gxsBjgBRUOw_W-1853203.tsv\n",
            "/content/an_data/a3OgoxXXgRa0homAO625jj3NfU2u-25487887.tsv\n",
            "/content/an_data/a5YzvBm7c31lRjx7z_2nfMDygwl0-1718942.tsv\n",
            "/content/an_data/anqXB6uId6MDusV6KvxX_W7x0HEC-22544249.tsv\n",
            "/content/an_data/axkF9gyJczwVDWPUPFdllX89G2yi-22938719.tsv\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Xf8S2tckbapf"
      },
      "source": [
        "df = pd.concat([pd.read_csv(f, sep = '\\t', header = None) for f in glob.glob('*.tsv')], ignore_index = True)#bind the tsv files together in a single file"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1bYdH9rdaC5v"
      },
      "source": [
        "df.columns = ['sentence','label']"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "2kEKloNXbjsB"
      },
      "source": [
        "with open('manual_sent.txt', 'w') as f:\n",
        "    for text in df['sentence'].tolist():\n",
        "        f.write(text + '\\n')"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9xLZZmE4cCiS"
      },
      "source": [
        "with open(\"/content/tost.txt\", 'rb') as corpus:\n",
        "  text_full = corpus.readlines()"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "UqKTY-nkzMKn"
      },
      "source": [
        "text_full"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "LPnwTkbOzMEY"
      },
      "source": [
        "listA = []\n",
        "with open('/content/tost.txt') as f:\n",
        "  for line in f:\n",
        "    words = [x.strip(string.punctuation) for x in line.split()]\n",
        "    listA.append(' '.join(w for w in words if w))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "d-Z-46LB2d7-"
      },
      "source": [
        "filtered_sentence = (\" \").join(listA)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "qUpGTheA2qhL"
      },
      "source": [
        "text_for_spacy = word_tokenize(filtered_sentence)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TSaBIWuX3iel",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "a0749510-22c0-4043-c836-ac30ba3be557"
      },
      "source": [
        "type(text_for_spacy)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "list"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 14
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wMLQsBF62sJt"
      },
      "source": [
        "for item in text_for_spacy:\n",
        "  item.replace(\"\\\\\\\\x\", '')"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "jCvOrXK1X-x9"
      },
      "source": [
        "import pickle\n",
        "import spacy"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "RF-4XWV8X-Ht",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 683
        },
        "outputId": "fa9211a6-dc14-408e-e5d1-12868a3a03b3"
      },
      "source": [
        "!pip install scispacy spacy"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting scispacy\n",
            "  Downloading https://files.pythonhosted.org/packages/93/0d/db88d1c2ad059c0b6854ca00519038b36ee1106e97899a3eb66bd25aeaaa/scispacy-0.2.5-py3-none-any.whl\n",
            "Requirement already satisfied: spacy in /usr/local/lib/python3.6/dist-packages (2.2.4)\n",
            "Collecting nmslib>=1.7.3.6\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/d5/fd/7d7428d29f12be5d1cc6d586d425b795cc9c596ae669593fd4f388602010/nmslib-2.0.6-cp36-cp36m-manylinux2010_x86_64.whl (12.9MB)\n",
            "\u001b[K     || 13.0MB 5.5MB/s \n",
            "\u001b[?25hRequirement already satisfied: requests<3.0.0conllu,>=2.0.0 in /usr/local/lib/python3.6/dist-packages (from scispacy) (2.23.0)\n",
            "Requirement already satisfied: scikit-learn>=0.20.3 in /usr/local/lib/python3.6/dist-packages (from scispacy) (0.22.2.post1)\n",
            "Collecting pysbd\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/89/f0/544ebd812f8fb534839cba7a121f2aaa520dc58760224f49bb5a0d063d06/pysbd-0.3.1-py3-none-any.whl (67kB)\n",
            "\u001b[K     || 71kB 8.8MB/s \n",
            "\u001b[?25hRequirement already satisfied: joblib in /usr/local/lib/python3.6/dist-packages (from scispacy) (0.16.0)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from scispacy) (1.18.5)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy) (0.7.1)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy) (7.4.0)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy) (3.0.2)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy) (0.4.1)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/dist-packages (from spacy) (1.1.3)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/dist-packages (from spacy) (1.0.0)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy) (2.0.3)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy) (1.0.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.6/dist-packages (from spacy) (49.6.0)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.6/dist-packages (from spacy) (4.41.1)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/dist-packages (from spacy) (1.0.2)\n",
            "Collecting pybind11>=2.2.3\n",
            "\u001b[?25l  Downloading https://files.pythonhosted.org/packages/89/e3/d576f6f02bc75bacbc3d42494e8f1d063c95617d86648dba243c2cb3963e/pybind11-2.5.0-py2.py3-none-any.whl (296kB)\n",
            "\u001b[K     || 296kB 50.2MB/s \n",
            "\u001b[?25hRequirement already satisfied: psutil in /usr/local/lib/python3.6/dist-packages (from nmslib>=1.7.3.6->scispacy) (5.4.8)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0conllu,>=2.0.0->scispacy) (2020.6.20)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0conllu,>=2.0.0->scispacy) (3.0.4)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0conllu,>=2.0.0->scispacy) (2.10)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0conllu,>=2.0.0->scispacy) (1.24.3)\n",
            "Requirement already satisfied: scipy>=0.17.0 in /usr/local/lib/python3.6/dist-packages (from scikit-learn>=0.20.3->scispacy) (1.4.1)\n",
            "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /usr/local/lib/python3.6/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy) (1.7.0)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy) (3.1.0)\n",
            "\u001b[31mERROR: scispacy 0.2.5 has requirement spacy<3.0.0,>=2.3.0, but you'll have spacy 2.2.4 which is incompatible.\u001b[0m\n",
            "Installing collected packages: pybind11, nmslib, pysbd, scispacy\n",
            "Successfully installed nmslib-2.0.6 pybind11-2.5.0 pysbd-0.3.1 scispacy-0.2.5\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Yr70puhTYAoL",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "7c283a96-ee9f-4764-bbf9-0844e62bb631"
      },
      "source": [
        "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_bc5cdr_md-0.2.3.tar.gz\n",
        "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_craft_md-0.2.3.tar.gz\n",
        "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_jnlpba_md-0.2.3.tar.gz\n",
        "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_bionlp13cg_md-0.2.3.tar.gz\n"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_bc5cdr_md-0.2.3.tar.gz\n",
            "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_bc5cdr_md-0.2.3.tar.gz (60.4MB)\n",
            "\u001b[K     || 60.5MB 77kB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy>=2.1.3 in /usr/local/lib/python3.6/dist-packages (from en-ner-bc5cdr-md==0.2.3) (2.2.4)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (0.7.1)\n",
            "Requirement already satisfied: numpy>=1.15.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.18.5)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (3.0.2)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (2.23.0)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.1.3)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.0.0)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (2.0.3)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (0.4.1)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (4.41.1)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (7.4.0)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (49.6.0)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (2.10)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.24.3)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (2020.6.20)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (3.0.4)\n",
            "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /usr/local/lib/python3.6/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (1.7.0)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-bc5cdr-md==0.2.3) (3.1.0)\n",
            "Building wheels for collected packages: en-ner-bc5cdr-md\n",
            "  Building wheel for en-ner-bc5cdr-md (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for en-ner-bc5cdr-md: filename=en_ner_bc5cdr_md-0.2.3-cp36-none-any.whl size=60877195 sha256=eca7f95d65c6560dfd165975457e317deb8efdc2296e38beca21a903edc26318\n",
            "  Stored in directory: /root/.cache/pip/wheels/1d/0d/81/00e4af94ff16aed4e28dc73f01d4e3bc7995fbc70b977d34c6\n",
            "Successfully built en-ner-bc5cdr-md\n",
            "Installing collected packages: en-ner-bc5cdr-md\n",
            "Successfully installed en-ner-bc5cdr-md-0.2.3\n",
            "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_craft_md-0.2.3.tar.gz\n",
            "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_craft_md-0.2.3.tar.gz (60.5MB)\n",
            "\u001b[K     || 60.5MB 69kB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy>=2.1.3 in /usr/local/lib/python3.6/dist-packages (from en-ner-craft-md==0.2.3) (2.2.4)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (3.0.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (49.6.0)\n",
            "Requirement already satisfied: numpy>=1.15.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.18.5)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (2.23.0)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (4.41.1)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (7.4.0)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (0.7.1)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (0.4.1)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.1.3)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (2.0.3)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.0.0)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-craft-md==0.2.3) (2.10)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-craft-md==0.2.3) (2020.6.20)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.24.3)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-craft-md==0.2.3) (3.0.4)\n",
            "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /usr/local/lib/python3.6/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-craft-md==0.2.3) (1.7.0)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-craft-md==0.2.3) (3.1.0)\n",
            "Building wheels for collected packages: en-ner-craft-md\n",
            "  Building wheel for en-ner-craft-md (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for en-ner-craft-md: filename=en_ner_craft_md-0.2.3-cp36-none-any.whl size=60895266 sha256=138b7f5a2bf9ac4a2a2ade4ad2697b03db5e26794908f1dd00625b757e443da7\n",
            "  Stored in directory: /root/.cache/pip/wheels/5e/fe/1a/616419ad4f220ce892e51b8a44039906f6013f4adf401c497f\n",
            "Successfully built en-ner-craft-md\n",
            "Installing collected packages: en-ner-craft-md\n",
            "Successfully installed en-ner-craft-md-0.2.3\n",
            "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_jnlpba_md-0.2.3.tar.gz\n",
            "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_jnlpba_md-0.2.3.tar.gz (60.5MB)\n",
            "\u001b[K     || 60.5MB 133kB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy>=2.1.3 in /usr/local/lib/python3.6/dist-packages (from en-ner-jnlpba-md==0.2.3) (2.2.4)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (3.0.2)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (4.41.1)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (2.0.3)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.0.0)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (0.4.1)\n",
            "Requirement already satisfied: numpy>=1.15.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.18.5)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (49.6.0)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.1.3)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (7.4.0)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (2.23.0)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (0.7.1)\n",
            "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /usr/local/lib/python3.6/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.7.0)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (3.0.4)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (2020.6.20)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (1.24.3)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (2.10)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-jnlpba-md==0.2.3) (3.1.0)\n",
            "Building wheels for collected packages: en-ner-jnlpba-md\n",
            "  Building wheel for en-ner-jnlpba-md (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for en-ner-jnlpba-md: filename=en_ner_jnlpba_md-0.2.3-cp36-none-any.whl size=60924971 sha256=9839f00a9bfc2ab33d7f6f2144865bf9675a4ff8d7f84da005b0e16ca3670873\n",
            "  Stored in directory: /root/.cache/pip/wheels/bb/d7/61/9eda84c1f9004bf97fb4000f5072cd5b3adf8ac3169a304644\n",
            "Successfully built en-ner-jnlpba-md\n",
            "Installing collected packages: en-ner-jnlpba-md\n",
            "Successfully installed en-ner-jnlpba-md-0.2.3\n",
            "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_bionlp13cg_md-0.2.3.tar.gz\n",
            "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.3/en_ner_bionlp13cg_md-0.2.3.tar.gz (60.5MB)\n",
            "\u001b[K     || 60.5MB 69kB/s \n",
            "\u001b[?25hRequirement already satisfied: spacy>=2.1.3 in /usr/local/lib/python3.6/dist-packages (from en-ner-bionlp13cg-md==0.2.3) (2.2.4)\n",
            "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (0.4.1)\n",
            "Requirement already satisfied: thinc==7.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (7.4.0)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (2.23.0)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (3.0.2)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (4.41.1)\n",
            "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (0.7.1)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (49.6.0)\n",
            "Requirement already satisfied: numpy>=1.15.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.18.5)\n",
            "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.1.3)\n",
            "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.0.0)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (2.0.3)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: srsly<1.1.0,>=1.0.2 in /usr/local/lib/python3.6/dist-packages (from spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.0.2)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.24.3)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (3.0.4)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (2020.6.20)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests<3.0.0,>=2.13.0->spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (2.10)\n",
            "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /usr/local/lib/python3.6/dist-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (1.7.0)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.1.3->en-ner-bionlp13cg-md==0.2.3) (3.1.0)\n",
            "Building wheels for collected packages: en-ner-bionlp13cg-md\n",
            "  Building wheel for en-ner-bionlp13cg-md (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for en-ner-bionlp13cg-md: filename=en_ner_bionlp13cg_md-0.2.3-cp36-none-any.whl size=60890010 sha256=a15c5a2c9106ba7c756dcfeab13c19c64b2724c7260a1be4e1c20e2be7094d97\n",
            "  Stored in directory: /root/.cache/pip/wheels/59/df/b4/36d96b68afb4ba3d15f5084d5bfde911a9fd24377f884e77b9\n",
            "Successfully built en-ner-bionlp13cg-md\n",
            "Installing collected packages: en-ner-bionlp13cg-md\n",
            "Successfully installed en-ner-bionlp13cg-md-0.2.3\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Tx7RS9tAYHCn"
      },
      "source": [
        "import scispacy\n",
        "from spacy import displacy\n",
        "from scispacy.abbreviation import AbbreviationDetector\n",
        "from scispacy.umls_linking import UmlsEntityLinker"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "cBQPzPRaYIIy"
      },
      "source": [
        "import en_ner_craft_md\n",
        "import en_ner_jnlpba_md\n",
        "import en_ner_bc5cdr_md\n",
        "import en_ner_bionlp13cg_md"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "hWSPMlBjYKmR"
      },
      "source": [
        "nlp = en_ner_craft_md.load()"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "mVpYHIBpYLDj"
      },
      "source": [
        "doc = nlp(str(text_for_spacy))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "4CXkHj5fYMVv",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "42638844-9a30-4566-ff0e-76b9f9814261"
      },
      "source": [
        "empty_list = []\n",
        "for entity in doc.ents:\n",
        "  empty_list.append(entity.text)\n",
        "for entity in doc.ents:\n",
        "  print(entity.label_, ' | ', entity.text) "
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "CL  |  cell\n",
            "CHEBI  |  cyclohexanol\n",
            "CHEBI  |  cyclohexanone\n",
            "CL  |  cells\n",
            "CHEBI  |  catechol\n",
            "CL  |  cells\n",
            "CHEBI  |  catechol\n",
            "CHEBI  |  resorcinol\n",
            "CHEBI  |  m-cresol\n",
            "CHEBI  |  orcinol\n",
            "SO  |  enzyme\n",
            "SO  |  Genetic\n",
            "CHEBI  |  proteins\n",
            "SO  |  products\n",
            "SO  |  genes\n",
            "CHEBI  |  protein\n",
            "CHEBI  |  xylene\n",
            "CHEBI  |  peroxynitrite\n",
            "TAXON  |  bovine\n",
            "CHEBI  |  superoxide\n",
            "CHEBI  |  superoxide\n",
            "CHEBI  |  superoxide\n",
            "CHEBI  |  peroxynitrite\n",
            "CHEBI  |  nitrophenol\n",
            "CHEBI  |  aldehyde\n",
            "CHEBI  |  A\n",
            "CL  |  cell\n",
            "CHEBI  |  beta-oxidation\n",
            "TAXON  |  bacterial\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  salicylate\n",
            "TAXON  |  bacteria\n",
            "SO  |  genes\n",
            "SO  |  plasmid\n",
            "CHEBI  |  salicylaldehyde\n",
            "SO  |  gene\n",
            "SO  |  fragment\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  dihydrodiol\n",
            "SO  |  gene\n",
            "SO  |  operons\n",
            "SO  |  enzymes\n",
            "CHEBI  |  putida\n",
            "CHEBI  |  alkanes\n",
            "SO  |  sequence\n",
            "SO  |  bp\n",
            "SO  |  region\n",
            "SO  |  operon\n",
            "GGP  |  AlkL\n",
            "CHEBI  |  polypeptides\n",
            "SO  |  predicted\n",
            "CHEBI  |  amino\n",
            "CHEBI  |  acid\n",
            "SO  |  sequence\n",
            "CHEBI  |  flavin\n",
            "CHEBI  |  proteins\n",
            "CHEBI  |  choline\n",
            "CHEBI  |  glucose\n",
            "CHEBI  |  alkanes\n",
            "CHEBI  |  putida\n",
            "CHEBI  |  alcohol\n",
            "SO  |  mutants\n",
            "SO  |  terminus\n",
            "SO  |  predicted\n",
            "CHEBI  |  protein\n",
            "TAXON  |  organism\n",
            "SO  |  forms\n",
            "SO  |  enzymes\n",
            "CHEBI  |  aromatic\n",
            "SO  |  enzymes\n",
            "SO  |  mutant\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  C\n",
            "CHEBI  |  acid\n",
            "CHEBI  |  Furfural\n",
            "CHEBI  |  furfuryl\n",
            "CHEBI  |  alcohol\n",
            "CHEBI  |  acid\n",
            "CHEBI  |  acetic\n",
            "CHEBI  |  acid\n",
            "CHEBI  |  methane\n",
            "CHEBI  |  carbon\n",
            "CHEBI  |  dioxide\n",
            "SO  |  DNA\n",
            "SO  |  region\n",
            "SO  |  plasmid\n",
            "CHEBI  |  benzoate\n",
            "SO  |  genes\n",
            "CHEBI  |  amino\n",
            "CHEBI  |  acid\n",
            "SO  |  sequences\n",
            "TAXON  |  organisms\n",
            "SO  |  genes\n",
            "CHEBI  |  1,2-dichlorobenzene\n",
            "CHEBI  |  1,2,4-trichlorobenzene\n",
            "SO  |  genes\n",
            "CHEBI  |  mercury\n",
            "SO  |  enzymes\n",
            "CHEBI  |  mercuric\n",
            "CHEBI  |  ion\n",
            "TAXON  |  Human\n",
            "CHEBI  |  glutathione\n",
            "CHEBI  |  molecule\n",
            "CHEBI  |  FAD-dependent\n",
            "CHEBI  |  disulphide\n",
            "GGP  |  MerA\n",
            "SO  |  Tn501\n",
            "CHEBI  |  residues\n",
            "SO  |  conserved\n",
            "SO  |  enzymes\n",
            "CHEBI  |  indole\n",
            "SO  |  Enzymatic\n",
            "SO  |  product\n",
            "CHEBI  |  compound\n",
            "CHEBI  |  cis-muconate\n",
            "CHEBI  |  A\n",
            "CHEBI  |  eutrophus\n",
            "CHEBI  |  eutrophus\n",
            "CHEBI  |  eutrophus\n",
            "CHEBI  |  chloro\n",
            "SO  |  genes\n",
            "CHEBI  |  proteins\n",
            "CHEBI  |  alkanes\n",
            "CHEBI  |  acyl-CoA\n",
            "CHEBI  |  propylene\n",
            "SO  |  isomerization\n",
            "CHEBI  |  CO2\n",
            "CHEBI  |  acetone\n",
            "CHEBI  |  aryl\n",
            "CHEBI  |  sulfides\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  enantiomeric\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  enantiomeric\n",
            "CHEBI  |  benzene\n",
            "CHEBI  |  oxygen\n",
            "CHEBI  |  atom\n",
            "CHEBI  |  methyl\n",
            "CHEBI  |  sulfoxide\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  O2\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  plant\n",
            "GO  |  cytochrome\n",
            "GGP  |  P-450\n",
            "CHEBI  |  methane\n",
            "TAXON  |  bacterial\n",
            "GO  |  monooxygenases\n",
            "SO  |  genes\n",
            "CHEBI  |  aldehyde\n",
            "SO  |  plasmid\n",
            "SO  |  enzymes\n",
            "CHEBI  |  alcohol\n",
            "CL  |  cells\n",
            "TAXON  |  organisms\n",
            "CHEBI  |  putida\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  naphthalene\n",
            "SO  |  region\n",
            "SO  |  1.9-kb\n",
            "SO  |  gene\n",
            "SO  |  gene\n",
            "SO  |  R\n",
            "SO  |  Rhodococcus\n",
            "SO  |  sequence\n",
            "CHEBI  |  amino\n",
            "CHEBI  |  acids\n",
            "SO  |  deduced\n",
            "CHEBI  |  amino\n",
            "CHEBI  |  acid\n",
            "SO  |  sequence\n",
            "SO  |  Rhodococcus\n",
            "CHEBI  |  acyl\n",
            "CHEBI  |  hydroxylamine\n",
            "CHEBI  |  hydroxamate\n",
            "TAXON  |  bacterial\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  toluene\n",
            "SO  |  enzyme\n",
            "CL  |  cells\n",
            "CHEBI  |  catechol\n",
            "CL  |  cells\n",
            "SO  |  operon\n",
            "CHEBI  |  PNP\n",
            "SO  |  genetic\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  phenanthrene\n",
            "SO  |  sequence\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  Alcaligenes\n",
            "CHEBI  |  eutrophus\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  plasmid\n",
            "SO  |  enzyme\n",
            "CHEBI  |  glyoxylate\n",
            "CHEBI  |  alpha-ketoglutarate\n",
            "SO  |  form\n",
            "CHEBI  |  succinate\n",
            "CHEBI  |  carbon\n",
            "CHEBI  |  dioxide\n",
            "SO  |  sequence\n",
            "SO  |  operon\n",
            "SO  |  genes\n",
            "CHEBI  |  protein\n",
            "CHEBI  |  protein\n",
            "CHEBI  |  cis-dihydrodiol\n",
            "CHEBI  |  polycyclic\n",
            "CHEBI  |  aromatic\n",
            "CHEBI  |  hydrocarbons\n",
            "CHEBI  |  pyrene\n",
            "CHEBI  |  BAA\n",
            "CHEBI  |  4-hydroxybenzoate\n",
            "CHEBI  |  protocatechuate\n",
            "SO  |  KZ-4\n",
            "SO  |  gene\n",
            "SO  |  products\n",
            "CHEBI  |  ferredoxin\n",
            "CHEBI  |  NADH\n",
            "SO  |  genes\n",
            "SO  |  gene\n",
            "CHEBI  |  ferredoxin\n",
            "CHEBI  |  ferredoxin\n",
            "SO  |  genes\n",
            "SO  |  coding\n",
            "GO  |  microsomal\n",
            "CHEBI  |  ethylbenzene\n",
            "SO  |  mutagenesis\n",
            "SO  |  genes\n",
            "CHEBI  |  atrazine\n",
            "CHEBI  |  carbon\n",
            "CHEBI  |  dioxide\n",
            "CHEBI  |  ammonia\n",
            "CHEBI  |  chloride\n",
            "CHEBI  |  antibiotics\n",
            "CHEBI  |  azoxy\n",
            "CHEBI  |  p-cymene\n",
            "CHEBI  |  alcohol\n",
            "CHEBI  |  aldehyde\n",
            "GO  |  membrane\n",
            "CHEBI  |  protein\n",
            "SO  |  gene\n",
            "SO  |  products\n",
            "CHEBI  |  aromatic\n",
            "CHEBI  |  hydrocarbon\n",
            "SO  |  operons\n",
            "CHEBI  |  p-cymene\n",
            "CHEBI  |  coenzyme\n",
            "CHEBI  |  ferredoxin\n",
            "CHEBI  |  electrons\n",
            "GGP  |  AtzC\n",
            "CHEBI  |  protein\n",
            "CHEBI  |  cytosine\n",
            "CHEBI  |  adenine\n",
            "CHEBI  |  phosphotriester\n",
            "SO  |  gene\n",
            "CHEBI  |  cyanuric\n",
            "CHEBI  |  acid\n",
            "SO  |  plasmids\n",
            "SO  |  Rhodococcus\n",
            "CHEBI  |  RDX\n",
            "CHEBI  |  2,4,6-trinitrotoluene\n",
            "CHEBI  |  RDX\n",
            "SO  |  gene\n",
            "SO  |  plasmid\n",
            "CHEBI  |  Phenol\n",
            "SO  |  R\n",
            "SO  |  position\n",
            "SO  |  genes\n",
            "SO  |  sequence\n",
            "CHEBI  |  proteins\n",
            "SO  |  sequence\n",
            "CHEBI  |  NADPH\n",
            "SO  |  site\n",
            "CHEBI  |  cyclohexanone\n",
            "CHEBI  |  Acinetobacter\n",
            "CHEBI  |  2-methylnaphthalene\n",
            "CHEBI  |  amino\n",
            "CHEBI  |  acids\n",
            "SO  |  sequences\n",
            "SO  |  enzyme\n",
            "CHEBI  |  phenylurea\n",
            "SO  |  gene\n",
            "TAXON  |  organisms\n",
            "CHEBI  |  dissimilatory\n",
            "CHEBI  |  Fe\n",
            "GO  |  Extracellular\n",
            "SO  |  enzymes\n",
            "CHEBI  |  lignin\n",
            "CHEBI  |  manganese\n",
            "CHEBI  |  aniline\n",
            "CHEBI  |  catechol\n",
            "SO  |  enzymes\n",
            "CHEBI  |  aniline\n",
            "SO  |  genes\n",
            "CHEBI  |  intradiol\n",
            "SO  |  enzyme\n",
            "CHEBI  |  maleylacetate\n",
            "SO  |  ORF5\n",
            "SO  |  ORF8\n",
            "CHEBI  |  protein\n",
            "SO  |  transposable\n",
            "CHEBI  |  Amino\n",
            "CHEBI  |  acid\n",
            "SO  |  sequence\n",
            "SO  |  enzyme\n",
            "CHEBI  |  protein\n",
            "CHEBI  |  Fe\n",
            "CHEBI  |  catechol\n",
            "SO  |  enzyme\n",
            "CHEBI  |  Malonate\n",
            "CHEBI  |  semialdehyde\n",
            "CHEBI  |  MSAD\n",
            "CHEBI  |  protein\n",
            "CHEBI  |  orf130\n",
            "SO  |  gene\n",
            "SO  |  genomic\n",
            "SO  |  gene\n",
            "CHEBI  |  malonate\n",
            "CHEBI  |  semialdehyde\n",
            "CHEBI  |  acetaldehyde\n",
            "CHEBI  |  atrazine\n",
            "SO  |  enzymes\n",
            "CHEBI  |  manganese-dependent\n",
            "CHEBI  |  lignin\n",
            "CHEBI  |  drug\n",
            "CHEBI  |  isoniazid\n",
            "SO  |  site\n",
            "SO  |  horseradish\n",
            "SO  |  sequence\n",
            "TAXON  |  bacterial\n",
            "GO  |  chromosome\n",
            "SO  |  clone\n",
            "SO  |  genome\n",
            "SO  |  plasmid\n",
            "CHEBI  |  hydrazide\n",
            "CHEBI  |  methane\n",
            "CHEBI  |  sMMO\n",
            "CHEBI  |  Methylosinus\n",
            "CHEBI  |  trichosporium\n",
            "CHEBI  |  OB3b\n",
            "CHEBI  |  propane\n",
            "SO  |  enzyme\n",
            "CHEBI  |  toluene\n",
            "CHEBI  |  NDMA\n",
            "SO  |  DNA\n",
            "SO  |  DNA\n",
            "SO  |  gene\n",
            "SO  |  DNA\n",
            "SO  |  cadA\n",
            "SO  |  gene\n",
            "SO  |  DNA\n",
            "SO  |  gene\n",
            "SO  |  DNA\n",
            "CHEBI  |  salicylate\n",
            "CHEBI  |  O2\n",
            "CHEBI  |  salicylaldehyde\n",
            "CHEBI  |  salicylate\n",
            "SO  |  enzymes\n",
            "CHEBI  |  carbaryl\n",
            "CHEBI  |  1-naphthol\n",
            "CL  |  cell\n",
            "CHEBI  |  carbaryl\n",
            "CHEBI  |  1-naphthol\n",
            "SO  |  genes\n",
            "CHEBI  |  atrazine\n",
            "CHEBI  |  atrazine\n",
            "SO  |  gene\n",
            "SO  |  genes\n",
            "CHEBI  |  alachlor\n",
            "TAXON  |  bacteria\n",
            "CHEBI  |  xenobiotics\n",
            "CHEBI  |  alachlor\n",
            "CHEBI  |  acenaphthene\n",
            "CHEBI  |  acenaphthylene\n",
            "CHEBI  |  fluorene\n",
            "GGP  |  PAO1\n",
            "CHEBI  |  naphthalene\n",
            "SO  |  genes\n",
            "SO  |  plasmid\n",
            "CHEBI  |  naphthalene\n",
            "CHEBI  |  fluorene\n",
            "SO  |  products\n",
            "CHEBI  |  aromatic\n",
            "CHEBI  |  Isoniazid\n",
            "CHEBI  |  nonenzymatically\n",
            "CHEBI  |  NAD\n",
            "CHEBI  |  NADP\n",
            "CHEBI  |  isonicotinoyl\n",
            "CHEBI  |  pyridine\n",
            "CHEBI  |  nucleotides\n",
            "SO  |  enzymes\n",
            "SO  |  peptide\n",
            "TAXON  |  organism\n",
            "CHEBI  |  pyrene\n",
            "CHEBI  |  dihydrodiol\n",
            "CL  |  cells\n",
            "SO  |  tfdA\n",
            "SO  |  genes\n",
            "SO  |  tfdA\n",
            "SO  |  genes\n",
            "SO  |  product\n",
            "SO  |  region\n",
            "CHEBI  |  water\n",
            "CHEBI  |  wastewater\n",
            "CHEBI  |  plant\n",
            "CHEBI  |  acid\n",
            "CHEBI  |  acid\n",
            "CHEBI  |  salicylate\n",
            "SO  |  enzymes\n",
            "CHEBI  |  salicylaldehyde\n",
            "CL  |  cell\n",
            "CHEBI  |  acenaphthylene\n",
            "CHEBI  |  salicylate\n",
            "CHEBI  |  catechol\n",
            "CHEBI  |  ammonia\n",
            "CHEBI  |  oxidizing\n",
            "TAXON  |  bacteria\n",
            "TAXON  |  bacteria\n",
            "TAXON  |  fungi\n",
            "GO  |  extracellular\n",
            "SO  |  enzymes\n",
            "CHEBI  |  lignin\n",
            "CHEBI  |  manganese-dependent\n",
            "CHEBI  |  herbicide\n",
            "CHEBI  |  atrazine\n",
            "SO  |  products\n",
            "CHEBI  |  xenobiotics\n",
            "SO  |  genes\n",
            "CHEBI  |  atrazine\n",
            "CHEBI  |  s-triazine\n",
            "CHEBI  |  atrazine\n",
            "CHEBI  |  atrazine\n",
            "CL  |  cells\n",
            "SO  |  enzymatic\n",
            "SO  |  genes\n",
            "CHEBI  |  herbicide\n",
            "SO  |  R\n",
            "CHEBI  |  2,4-dichlorophenoxy\n",
            "CHEBI  |  propionate\n",
            "SO  |  Primers\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "SO  |  gene\n",
            "SO  |  clusters\n",
            "CHEBI  |  acidovorans\n",
            "SO  |  gene\n",
            "SO  |  R\n",
            "CHEBI  |  herbicide\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "CHEBI  |  herbicide\n",
            "SO  |  R\n",
            "CHEBI  |  carbendazim\n",
            "CHEBI  |  vegetables\n",
            "CHEBI  |  herbs\n",
            "TAXON  |  bacteria\n",
            "CHEBI  |  herbicides\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  enzyme\n",
            "TAXON  |  bacterial\n",
            "SO  |  DNA\n",
            "SO  |  fragment\n",
            "SO  |  polymorphism\n",
            "CHEBI  |  rRNA\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  sequences\n",
            "SO  |  primer\n",
            "CHEBI  |  herbicide\n",
            "TAXON  |  bacterial\n",
            "TAXON  |  microcosms\n",
            "CHEBI  |  2,4-D\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  sequences\n",
            "TAXON  |  microcosms\n",
            "SO  |  cluster\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  sequence\n",
            "CHEBI  |  herbicide\n",
            "SO  |  gene\n",
            "SO  |  cluster\n",
            "SO  |  enzymes\n",
            "GGP  |  NpsA1\n",
            "CHEBI  |  His\n",
            "CHEBI  |  proteins\n",
            "SO  |  gene\n",
            "SO  |  cluster\n",
            "SO  |  gene\n",
            "SO  |  cluster\n",
            "SO  |  R\n",
            "CHEBI  |  herbicide\n",
            "TAXON  |  microcosms\n",
            "SO  |  genes\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  transcript\n",
            "SO  |  tfdA\n",
            "SO  |  Gene\n",
            "SO  |  tfdA\n",
            "SO  |  genes\n",
            "SO  |  DNA\n",
            "SO  |  conserved\n",
            "CHEBI  |  mRNA\n",
            "SO  |  tfdA\n",
            "SO  |  sequences\n",
            "SO  |  genes\n",
            "SO  |  transcripts\n",
            "SO  |  transcripts\n",
            "SO  |  genes\n",
            "TAXON  |  bacteria\n",
            "CHEBI  |  atrazine\n",
            "SO  |  gene\n",
            "CHEBI  |  propanil\n",
            "CHEBI  |  propanil\n",
            "CHEBI  |  3,4-DCA\n",
            "SO  |  enzymes\n",
            "SO  |  enzymes\n",
            "SO  |  modified\n",
            "SO  |  gene\n",
            "CHEBI  |  atrazine\n",
            "CHEBI  |  atrazine\n",
            "SO  |  gene\n",
            "CHEBI  |  herbicide\n",
            "CHEBI  |  linuron\n",
            "CHEBI  |  linuron\n",
            "TAXON  |  microcosms\n",
            "TAXON  |  microcosms\n",
            "GO  |  matrix\n",
            "CHEBI  |  linuron\n",
            "SO  |  phylotype\n",
            "SO  |  gene\n",
            "CHEBI  |  pesticides\n",
            "GO  |  membranes\n",
            "GO  |  membrane\n",
            "CHEBI  |  water\n",
            "SO  |  genes\n",
            "SO  |  sequences\n",
            "SO  |  DNA\n",
            "SO  |  genetic\n",
            "SO  |  transcript\n",
            "CHEBI  |  ammonia\n",
            "SO  |  removal\n",
            "TAXON  |  bacterial\n",
            "SO  |  transcripts\n",
            "CHEBI  |  ammonia\n",
            "SO  |  removal\n",
            "CHEBI  |  ammonia\n",
            "CHEBI  |  ammonia\n",
            "SO  |  removal\n",
            "CHEBI  |  isoproturon\n",
            "CHEBI  |  biotransformations\n",
            "SO  |  Genes\n",
            "CHEBI  |  atrazine\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "SO  |  introduction\n",
            "CHEBI  |  man-made\n",
            "CHEBI  |  s-triazine\n",
            "CHEBI  |  herbicides\n",
            "CHEBI  |  atrazine\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "SO  |  tfdA\n",
            "SO  |  genes\n",
            "CHEBI  |  MCPA\n",
            "CHEBI  |  microcosm\n",
            "CHEBI  |  rRNA\n",
            "SO  |  genes\n",
            "TAXON  |  fungal\n",
            "SO  |  fragment\n",
            "SO  |  genes\n",
            "SO  |  tfdA\n",
            "SO  |  cadA\n",
            "SO  |  tfdA\n",
            "SO  |  cadA\n",
            "SO  |  genes\n",
            "CHEBI  |  MCPA\n",
            "SO  |  genus\n",
            "CHEBI  |  linuron\n",
            "SO  |  gene\n",
            "CHEBI  |  linuron\n",
            "CHEBI  |  linuron\n",
            "CHEBI  |  herbicide\n",
            "CHEBI  |  metal-dependent\n",
            "CHEBI  |  protein\n",
            "TAXON  |  bacterial\n",
            "GGP  |  Rieske\n",
            "CHEBI  |  non-heme\n",
            "CHEBI  |  iron\n",
            "SO  |  genes\n",
            "SO  |  transposable\n",
            "SO  |  element\n",
            "SO  |  flanked\n",
            "SO  |  repeats\n",
            "SO  |  insertion\n",
            "SO  |  element\n",
            "SO  |  homology\n",
            "SO  |  cluster\n",
            "TAXON  |  bacterial\n",
            "SO  |  genes\n",
            "SO  |  carA\n",
            "SO  |  gene\n",
            "SO  |  genes\n",
            "CHEBI  |  pollutants\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "CHEBI  |  ATZ\n",
            "SO  |  genomes\n",
            "SO  |  gene\n",
            "SO  |  cluster\n",
            "CHEBI  |  atrazine\n",
            "SO  |  genes\n",
            "SO  |  FRAAL1471\n",
            "CHEBI  |  adenosine\n",
            "CHEBI  |  N-ethyl\n",
            "CHEBI  |  atrazine\n",
            "SO  |  gene\n",
            "SO  |  predicted\n",
            "CHEBI  |  atrazine\n",
            "SO  |  Genes\n",
            "CHEBI  |  atrazine\n",
            "SO  |  tfdA\n",
            "SO  |  genes\n",
            "CHEBI  |  protein\n",
            "SO  |  tfdA\n",
            "SO  |  gene\n",
            "TAXON  |  bacteria\n",
            "SO  |  gene\n",
            "SO  |  transposable\n",
            "SO  |  element\n",
            "SO  |  conserved\n",
            "SO  |  enzyme\n",
            "CHEBI  |  alachlor\n",
            "CHEBI  |  ferredoxins\n",
            "SO  |  enzyme\n",
            "CHEBI  |  ferredoxin\n",
            "CHEBI  |  electron\n",
            "GO  |  ETC\n",
            "SO  |  tceA\n",
            "SO  |  vcrA\n",
            "SO  |  gene\n",
            "SO  |  DNA\n",
            "SO  |  vcrA\n",
            "CHEBI  |  mRNA\n",
            "CHEBI  |  lactate\n",
            "SO  |  tceA\n",
            "SO  |  gene\n",
            "SO  |  transcripts\n",
            "SO  |  vcrA\n",
            "SO  |  gene\n",
            "CHEBI  |  chloride\n",
            "SO  |  genes\n",
            "SO  |  tceA\n",
            "SO  |  vcrA\n",
            "SO  |  gene\n",
            "TAXON  |  organisms\n",
            "SO  |  products\n",
            "SO  |  vcrA\n",
            "SO  |  gene\n",
            "CHEBI  |  wastewater\n",
            "CHEBI  |  fluoranthene\n",
            "CHEBI  |  fluoranthene\n",
            "CHEBI  |  amino\n",
            "CHEBI  |  acid\n",
            "SO  |  sequence\n",
            "SO  |  plasmid\n",
            "SO  |  genes\n",
            "SO  |  genes\n",
            "SO  |  plasmid\n",
            "SO  |  plasmid\n",
            "SO  |  gene\n",
            "SO  |  clusters\n",
            "CHEBI  |  proteins\n",
            "CHEBI  |  phenylacetate\n",
            "CHEBI  |  flavin\n",
            "CHEBI  |  protein\n",
            "SO  |  enzyme\n",
            "CL  |  cells\n",
            "CHEBI  |  isophthalate\n",
            "CHEBI  |  ATP-dependent\n",
            "CHEBI  |  isophthalate\n",
            "CHEBI  |  flavin\n",
            "GGP  |  UbiX\n",
            "SO  |  decarboxylation\n",
            "CHEBI  |  superoxide\n",
            "SO  |  enzymes\n",
            "SO  |  Rhodococcus\n",
            "SO  |  genomes\n",
            "CHEBI  |  atrazine\n",
            "SO  |  gene\n",
            "SO  |  fragments\n",
            "SO  |  genes\n",
            "SO  |  primers\n",
            "SO  |  vcrA\n",
            "TAXON  |  bacteria\n",
            "TAXON  |  bacteria\n",
            "CHEBI  |  wastewaters\n",
            "SO  |  genes\n",
            "GGP  |  SadC\n",
            "GGP  |  SadC\n",
            "CHEBI  |  sulfonamide\n",
            "CHEBI  |  molecules\n",
            "GGP  |  SadC\n",
            "TAXON  |  bacterial\n",
            "SO  |  gene\n",
            "SO  |  plastids\n",
            "SO  |  plants\n",
            "TAXON  |  bacterial\n",
            "SO  |  gene\n",
            "SO  |  plants\n",
            "SO  |  plastid\n",
            "TAXON  |  fungi\n",
            "CHEBI  |  lignocellulose\n",
            "GO  |  extracellular\n",
            "SO  |  enzymes\n",
            "SO  |  enzymatic\n",
            "TAXON  |  fungi\n",
            "CHEBI  |  manganese\n",
            "CHEBI  |  manganese\n",
            "CHEBI  |  manganese\n",
            "SO  |  gene\n",
            "SO  |  rice\n",
            "GGP  |  mnp2\n",
            "SO  |  mnp3\n",
            "CHEBI  |  manganese\n",
            "GGP  |  mnp2\n",
            "GGP  |  mnp3\n",
            "CHEBI  |  arsenic\n",
            "SO  |  gene\n",
            "TAXON  |  fungus\n",
            "TAXON  |  aurantiaca\n",
            "SO  |  enzymes\n",
            "CHEBI  |  protocatechuate\n",
            "CHEBI  |  protocatechuate\n",
            "CHEBI  |  protocatechuate\n",
            "CHEBI  |  salicylate\n",
            "CHEBI  |  fluoranthene\n",
            "SO  |  enzyme\n",
            "SO  |  enzymes\n",
            "SO  |  expected\n",
            "SO  |  enzymes\n",
            "CHEBI  |  Manganese\n",
            "CHEBI  |  Aroclors\n",
            "GO  |  extracellular\n",
            "SO  |  enzymes\n",
            "CHEBI  |  inhibitor\n",
            "CHEBI  |  catechol\n",
            "CHEBI  |  microorganisms\n",
            "CHEBI  |  aromatic\n",
            "CHEBI  |  ortho\n",
            "CHEBI  |  catechol\n",
            "CHEBI  |  catechol\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9zgo1aS8YNfW"
      },
      "source": [
        "#len(set(empty_list))"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "AjM-h6-53v-7",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "9120f253-f0fd-45f3-a91f-c9a851885b74"
      },
      "source": [
        "empty_list"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['chemical',\n",
              " 'site',\n",
              " 'sequence',\n",
              " 'sequence',\n",
              " 'known',\n",
              " 'compounds',\n",
              " 'molecular',\n",
              " 'acetamide',\n",
              " 'carbon',\n",
              " 'acetamide',\n",
              " 'xc2\\\\\\\\xb0c',\n",
              " 'enzyme',\n",
              " 'chloroacetamide',\n",
              " 'herbicides',\n",
              " 'alachlor',\n",
              " 'pretilachlor',\n",
              " 'chloroacetamide',\n",
              " 'herbicides',\n",
              " 'chloroacetamide',\n",
              " 'herbicides',\n",
              " 'enzyme',\n",
              " 'antibiotics',\n",
              " 'antibiotic',\n",
              " 'regions',\n",
              " 'antibiotic',\n",
              " 'mutation',\n",
              " 'antibiotic',\n",
              " 'genes',\n",
              " 'evolved',\n",
              " 'bacteria',\n",
              " 'genes',\n",
              " 'animal',\n",
              " 'human',\n",
              " 'waste',\n",
              " 'pollutants',\n",
              " 'genetic',\n",
              " 'genes',\n",
              " 'anthropogenic',\n",
              " 'antibiotic',\n",
              " 'antibiotic',\n",
              " 'tfdA',\n",
              " 'gene',\n",
              " 'phenoxy',\n",
              " 'acid',\n",
              " 'herbicide',\n",
              " 'sediments',\n",
              " 'sms',\n",
              " 'water',\n",
              " 'herbicide',\n",
              " 'herbicide',\n",
              " 'herbicide',\n",
              " 'tfdA',\n",
              " 'gene',\n",
              " 'tfdA',\n",
              " 'genes',\n",
              " 'tfdA',\n",
              " 'gene',\n",
              " 'herbicide',\n",
              " 'water',\n",
              " 'water',\n",
              " 'acids',\n",
              " 'microorganisms',\n",
              " 'acid',\n",
              " 'microcosm',\n",
              " 'genes',\n",
              " 'fungal',\n",
              " 'fragment',\n",
              " 'genes',\n",
              " 'tfdA',\n",
              " 'cadA',\n",
              " 'tfdA',\n",
              " 'cadA',\n",
              " 'genes',\n",
              " 'tfdA',\n",
              " 'genes',\n",
              " 'fungal',\n",
              " 'herbicide',\n",
              " 'water',\n",
              " 'water',\n",
              " 'wastewater',\n",
              " 'irrigation',\n",
              " 'irrigation',\n",
              " 'water',\n",
              " 'water',\n",
              " 'salinity',\n",
              " 'nutrients',\n",
              " 'metals',\n",
              " 'genus',\n",
              " 'linuron',\n",
              " 'gene',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'herbicide',\n",
              " 'phylotype',\n",
              " 'linuron',\n",
              " 'herbicide',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'phylotype',\n",
              " 'linuron',\n",
              " 'microcosm',\n",
              " 'organism',\n",
              " 'phylotype',\n",
              " 'replacement',\n",
              " 'phylotypes',\n",
              " 'linuron',\n",
              " 'genus',\n",
              " 'linuron',\n",
              " 'water',\n",
              " 'remove',\n",
              " 'pollutant',\n",
              " 'introducing',\n",
              " 'cells',\n",
              " 'ionic',\n",
              " 'cell',\n",
              " 'cells',\n",
              " 'cells',\n",
              " 'known',\n",
              " 'products',\n",
              " 'water',\n",
              " 'phenylurea',\n",
              " 'herbicide',\n",
              " 'linuron',\n",
              " 'enzyme',\n",
              " 'linuron',\n",
              " 'enzyme',\n",
              " 'linuron',\n",
              " 'enzyme',\n",
              " 'enzymes',\n",
              " 'bacteria',\n",
              " 'enzyme',\n",
              " 'phenylurea',\n",
              " 'herbicide',\n",
              " 'metal-dependent',\n",
              " 'protein',\n",
              " 'gene',\n",
              " 'genome',\n",
              " 'sequence',\n",
              " 'linuron',\n",
              " 'mutants',\n",
              " 'linuron',\n",
              " 'gene',\n",
              " 'gene',\n",
              " 'modules',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'genes',\n",
              " 'proteins',\n",
              " 'genetic',\n",
              " 'modules',\n",
              " 'water',\n",
              " 'chemical',\n",
              " 'bacterial',\n",
              " 'bacteria',\n",
              " 'bacterial',\n",
              " 'bacterial',\n",
              " 'fragment',\n",
              " 'polymorphism',\n",
              " 'molecular',\n",
              " 'bacterial',\n",
              " 'human',\n",
              " 'bacterial',\n",
              " 'human',\n",
              " 'bacterial',\n",
              " 'bacterial',\n",
              " 'bacterial',\n",
              " 'pesticide',\n",
              " 'removes',\n",
              " 'pesticides',\n",
              " 'products',\n",
              " 'pesticide',\n",
              " 'pesticide',\n",
              " 'pesticide',\n",
              " 'pesticide',\n",
              " 'persulfate',\n",
              " 'chemical',\n",
              " 'metal',\n",
              " 'chromium',\n",
              " 'toxic',\n",
              " 'metal',\n",
              " 'human',\n",
              " 'persulfate',\n",
              " 'chemical',\n",
              " 'chromium',\n",
              " 'physical-chemical',\n",
              " 'unactivated',\n",
              " 'persulfate',\n",
              " 'persulfate',\n",
              " 'chromium',\n",
              " 'chromium',\n",
              " 'persulfate',\n",
              " 'chemical',\n",
              " 'oxidant',\n",
              " 'chromium',\n",
              " 'oxidant',\n",
              " 'chromium',\n",
              " 'alkaline',\n",
              " 'alkaline',\n",
              " 'chromium',\n",
              " 'persulfate',\n",
              " 'chemical',\n",
              " 'unactivated',\n",
              " 'persulfate',\n",
              " 'chromium',\n",
              " 'chromium',\n",
              " 'site',\n",
              " 'chromium',\n",
              " 'metal',\n",
              " 'aquifers',\n",
              " 'toluene',\n",
              " 'toluene',\n",
              " 'water',\n",
              " 'carbon',\n",
              " 'toluene',\n",
              " 'removal',\n",
              " 'carbon',\n",
              " 'removal',\n",
              " 'bacterial',\n",
              " 'ecosystems',\n",
              " 'aquifers',\n",
              " 'bacterial',\n",
              " 'acetamide',\n",
              " 'gene',\n",
              " 'herbicide',\n",
              " 'propanil',\n",
              " 'phenylurea',\n",
              " 'herbicides',\n",
              " 'phenylurea',\n",
              " 'herbicides',\n",
              " 'molecular',\n",
              " 'genes',\n",
              " 'phenylurea',\n",
              " 'herbicides',\n",
              " 'bacterial',\n",
              " 'non-heme',\n",
              " 'iron',\n",
              " 'genes',\n",
              " 'transposable',\n",
              " 'element',\n",
              " 'flanked',\n",
              " 'repeats',\n",
              " 'insertion',\n",
              " 'element',\n",
              " 'transposable',\n",
              " 'element',\n",
              " 'conserved',\n",
              " 'phenylurea',\n",
              " 'sphingomonads',\n",
              " 'gene',\n",
              " 'electron',\n",
              " 'ferredoxin',\n",
              " 'electron',\n",
              " 'putida',\n",
              " 'enzyme',\n",
              " 'ferredoxin',\n",
              " 'coli',\n",
              " 'electron',\n",
              " 'ferredoxin',\n",
              " 'homology',\n",
              " 'cluster',\n",
              " 'bacterial',\n",
              " 'ammonia',\n",
              " 'nitrate',\n",
              " 'nitrogen',\n",
              " 'simazine',\n",
              " 'bacteria',\n",
              " 'microcosms',\n",
              " 'herbicide',\n",
              " 'fragment',\n",
              " 'polymorphism',\n",
              " 'inorganic',\n",
              " 'nitrogen',\n",
              " 'ammonia',\n",
              " 'nitrate',\n",
              " 'simazine',\n",
              " 'nitrogen',\n",
              " 'herbicide',\n",
              " 'nitrogen',\n",
              " 's-triazine',\n",
              " 'microorganisms',\n",
              " 'genes',\n",
              " 'dichlorodiphenyltrichloroethane',\n",
              " 'hexachlorocyclohexane',\n",
              " 'atrazine',\n",
              " 'marine',\n",
              " 'sediments',\n",
              " 'genes',\n",
              " 'genes',\n",
              " 'genes',\n",
              " 'locations',\n",
              " 'genes',\n",
              " 'carA',\n",
              " 'gene',\n",
              " 'genes',\n",
              " 'pollutants',\n",
              " 'genes',\n",
              " 'genes',\n",
              " 'pollutants',\n",
              " 'sediments',\n",
              " 'sediments',\n",
              " 'assimilable',\n",
              " 'carbon',\n",
              " 'introduce',\n",
              " 'modified',\n",
              " 'water',\n",
              " 'bacterial',\n",
              " 'pesticide',\n",
              " 'cell',\n",
              " 'aquatic',\n",
              " 'sewage',\n",
              " 'water',\n",
              " 'water',\n",
              " 'introduction',\n",
              " 'water',\n",
              " 'compound-specific',\n",
              " 'removal',\n",
              " 'carbon',\n",
              " 'membrane',\n",
              " 'removal',\n",
              " 'removal',\n",
              " 'removal',\n",
              " 'wastewater',\n",
              " 'micropollutants',\n",
              " 'water',\n",
              " 'bacterial',\n",
              " 'catalysts',\n",
              " 'pollutant',\n",
              " 'ecosystems',\n",
              " 'xenobiotics',\n",
              " 'carbon',\n",
              " 'bacterial',\n",
              " 'wdl6',\n",
              " 'linuron',\n",
              " 'carbon',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'cells',\n",
              " 'linuron',\n",
              " 'removal',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'carbon',\n",
              " 'linuron',\n",
              " 'linuron',\n",
              " 'removal',\n",
              " 'cell',\n",
              " 'aquifers',\n",
              " 'water',\n",
              " 'stormwater',\n",
              " 'groundwater',\n",
              " 'pesticides',\n",
              " 'polycyclic',\n",
              " 'aromatic',\n",
              " 'hydrocarbons',\n",
              " 'chemicals',\n",
              " 'pesticides',\n",
              " 'pesticide',\n",
              " 'products',\n",
              " 'chemicals',\n",
              " 'fragrances',\n",
              " 'stormwater',\n",
              " 'pesticides',\n",
              " 'metolachlor',\n",
              " 'chlorpyrifos',\n",
              " 'chlorpyrifos',\n",
              " 'metolachlor',\n",
              " 'simazine',\n",
              " 'triallate',\n",
              " 'fluorene',\n",
              " 'tris',\n",
              " 'phosphate',\n",
              " 'chemical',\n",
              " 'sites',\n",
              " 'stormwater',\n",
              " 'water',\n",
              " 'water',\n",
              " 'residues',\n",
              " 'human',\n",
              " 'pharmaceuticals',\n",
              " 'human',\n",
              " 'pharmaceuticals',\n",
              " 'pharmaceuticals',\n",
              " 'substances',\n",
              " 'chemical',\n",
              " 'pharmaceuticals',\n",
              " 'substances',\n",
              " 'pharmaceuticals',\n",
              " 'dimethyl',\n",
              " 'fumarate',\n",
              " 'rodenticide',\n",
              " 'inhibitor',\n",
              " 'pharmaceuticals',\n",
              " 'pharmaceuticals',\n",
              " 'human',\n",
              " 'substances',\n",
              " 'human',\n",
              " 'pharmaceuticals',\n",
              " 'human',\n",
              " 'substances',\n",
              " 'human',\n",
              " 'pharmaceuticals',\n",
              " 'substances',\n",
              " 'chlorophenols',\n",
              " 'pollutants',\n",
              " 'pesticides',\n",
              " 'products',\n",
              " 'toxic',\n",
              " 'beings',\n",
              " 'carcinogenic',\n",
              " 'mutagenic',\n",
              " 'bacterial',\n",
              " 'removing',\n",
              " 'bacteria',\n",
              " 'carbon',\n",
              " 'bacteria',\n",
              " 'genes',\n",
              " 'enzymes',\n",
              " 'genetic',\n",
              " 'compounds',\n",
              " 'compounds',\n",
              " 'human',\n",
              " 'medications',\n",
              " 'sediments',\n",
              " 'wastewater',\n",
              " 'pharmaceuticals',\n",
              " 'drug',\n",
              " 'caffeine',\n",
              " 'methamphetamine',\n",
              " 'acetaminophen',\n",
              " 'chemical',\n",
              " 'removed',\n",
              " 'methamphetamine',\n",
              " 'methamphetamine',\n",
              " 'methamphetamine',\n",
              " 'sediments',\n",
              " 'remove',\n",
              " 'pharmaceuticals',\n",
              " 'drug',\n",
              " 'pharmaceuticals',\n",
              " 'water',\n",
              " 'pharmaceuticals',\n",
              " 'pharmaceuticals',\n",
              " 'water',\n",
              " 'substances',\n",
              " 'pesticides',\n",
              " 'compounds',\n",
              " 'hormones',\n",
              " 'caffeine',\n",
              " 'pentachlorophenol',\n",
              " 'carbamazepine',\n",
              " 'regions',\n",
              " 'antibiotic',\n",
              " 'genes',\n",
              " 'genetic',\n",
              " 'water',\n",
              " 'antibiotic',\n",
              " 'sequence',\n",
              " 'sequences',\n",
              " 'freshwater',\n",
              " 'fishpond',\n",
              " 'rifampin',\n",
              " 'genes',\n",
              " 'sequences',\n",
              " 'plasmids',\n",
              " 'genes',\n",
              " 'plasmid',\n",
              " 'water',\n",
              " 'carbon',\n",
              " 'chemicals',\n",
              " 'synthetic',\n",
              " 'firmicutes',\n",
              " 'xenobiotics',\n",
              " 'humic',\n",
              " 'acid',\n",
              " 'Cytochrome',\n",
              " 'genes',\n",
              " 'genes',\n",
              " 'atrazine',\n",
              " 'hydroxyatrazine',\n",
              " 'atrazine',\n",
              " 'nitrogen',\n",
              " 'carbon',\n",
              " 'genomes',\n",
              " 'gene',\n",
              " 'cluster',\n",
              " 'atrazine',\n",
              " 'genes',\n",
              " 'fraal1471',\n",
              " 'rna',\n",
              " 'gene',\n",
              " 'genes',\n",
              " 'fraal1471',\n",
              " 'cluster',\n",
              " 'atrazine',\n",
              " 'adenosine',\n",
              " 'atrazine',\n",
              " 'gene',\n",
              " 'predicted',\n",
              " 'atrazine',\n",
              " 'adenosine',\n",
              " 'atrazine',\n",
              " 'pesticides',\n",
              " 'pollution',\n",
              " 'atrazine',\n",
              " 'pollutant',\n",
              " 'atrazine',\n",
              " 'atrazine',\n",
              " 'atrazine',\n",
              " 'pollutant',\n",
              " 'removal',\n",
              " 'atrazine',\n",
              " 'atrazine',\n",
              " 'atrazine',\n",
              " 'remove',\n",
              " 'atrazine',\n",
              " 'bacterial',\n",
              " 'bacterial',\n",
              " 'genetic',\n",
              " 'atrazine',\n",
              " 'atrazine',\n",
              " 'cyanuric',\n",
              " 'acid',\n",
              " 'genes',\n",
              " 'atrazine',\n",
              " 'genes',\n",
              " 'bacterial',\n",
              " 'atrazine',\n",
              " 'waste',\n",
              " 'substances',\n",
              " 'known',\n",
              " 'wastewater',\n",
              " 'wastewater',\n",
              " 'human',\n",
              " 'wastewater',\n",
              " 'influents',\n",
              " 'products',\n",
              " 'nutrient',\n",
              " 'removal',\n",
              " 'membrane',\n",
              " 'wastewater',\n",
              " 'mbr',\n",
              " 'wastewater',\n",
              " 'chemicals',\n",
              " 'chemicals',\n",
              " 'enzymes',\n",
              " 'enzymes',\n",
              " 'assembly',\n",
              " 'variations',\n",
              " 'wastewaters',\n",
              " 'pollutants',\n",
              " 'microorganisms',\n",
              " 'wastewater',\n",
              " 'micropollutants',\n",
              " 'substance',\n",
              " 'pollutants',\n",
              " 'microorganisms',\n",
              " 'wastewater',\n",
              " 'pharmaceuticals',\n",
              " 'products',\n",
              " 'antibiotics',\n",
              " 'molecular',\n",
              " 'enzymes',\n",
              " 'products',\n",
              " 'gaps',\n",
              " 'bacterial',\n",
              " 'agricultural',\n",
              " 'sites',\n",
              " 'sequence',\n",
              " 'genes',\n",
              " 'primers',\n",
              " 'conserved',\n",
              " 'domain',\n",
              " 'tfdA',\n",
              " 'known',\n",
              " 'tfdA',\n",
              " 'genes',\n",
              " 'protein',\n",
              " 'tfdA',\n",
              " 'gene',\n",
              " 'bacteria',\n",
              " 'carbon',\n",
              " 'herbicides',\n",
              " 'acid',\n",
              " 'compounds',\n",
              " 'water',\n",
              " 'water',\n",
              " 'wastewater',\n",
              " 'plants',\n",
              " 'removed',\n",
              " 'water',\n",
              " 'feature',\n",
              " 'wastewater',\n",
              " 'carbon',\n",
              " 'wastewater',\n",
              " 'water',\n",
              " 'feature',\n",
              " \"part', 'of', 'a', 'broader', 'multipronged', 'mitigation\",\n",
              " 'herbicides',\n",
              " 'alachlor',\n",
              " 'non-heme',\n",
              " 'iron',\n",
              " 'herbicides',\n",
              " 'gene',\n",
              " 'transposable',\n",
              " 'element',\n",
              " 'conserved',\n",
              " 'amino',\n",
              " 'acid',\n",
              " 'sequence',\n",
              " 'ferredoxin',\n",
              " 'genes',\n",
              " 'glutathione',\n",
              " 'gene',\n",
              " 'genome',\n",
              " 'genes',\n",
              " 'ferredoxins',\n",
              " 'amino',\n",
              " 'acid',\n",
              " 'sequence',\n",
              " 'ferredoxin',\n",
              " 'dicamba',\n",
              " 'amino',\n",
              " 'acid',\n",
              " 'sequence',\n",
              " 'ferredoxin',\n",
              " 'genes',\n",
              " 'genes',\n",
              " 'proteins',\n",
              " 'enzyme',\n",
              " 'alachlor',\n",
              " 'ferredoxins',\n",
              " 'enzyme',\n",
              " 'ferredoxin',\n",
              " 'electron',\n",
              " 'electron',\n",
              " 'water',\n",
              " 'regions',\n",
              " 'chemicals',\n",
              " 'synthetic',\n",
              " 'compounds',\n",
              " 'pesticide',\n",
              " 'products',\n",
              " 'products',\n",
              " 'chemicals',\n",
              " 'corrosion',\n",
              " 'inhibitors',\n",
              " 'organophosphate',\n",
              " 'plasticizers',\n",
              " 'volatile',\n",
              " 'compounds',\n",
              " 'polycyclic',\n",
              " 'aromatic',\n",
              " 'hydrocarbons',\n",
              " 'reclaimed',\n",
              " 'water',\n",
              " 'chemical',\n",
              " 'chemicals',\n",
              " 'substances',\n",
              " 'chemicals',\n",
              " 'ecosystems',\n",
              " 'human',\n",
              " 'chemical',\n",
              " 'pollutants',\n",
              " 'pesticides',\n",
              " 'alkylphenols',\n",
              " 'pharmaceuticals',\n",
              " 'hormones',\n",
              " 'compounds',\n",
              " 'rivers',\n",
              " 'rivers',\n",
              " 'river',\n",
              " 'compound',\n",
              " 'chemical',\n",
              " 'river',\n",
              " 'fish',\n",
              " 'compounds',\n",
              " 'rivers',\n",
              " 'pesticides',\n",
              " 'chlorpyriphos',\n",
              " 'prochloraz',\n",
              " 'chemicals',\n",
              " 'nonylphenol',\n",
              " 'surfactants',\n",
              " 'genes',\n",
              " 'chlorinated',\n",
              " 'compounds',\n",
              " 'lactate',\n",
              " 'gene',\n",
              " 'vcrA',\n",
              " 'gene',\n",
              " 'dna',\n",
              " 'vcrA',\n",
              " 'mrna',\n",
              " 'lactate',\n",
              " 'tceA',\n",
              " 'gene',\n",
              " 'transcripts',\n",
              " 'vcrA',\n",
              " 'gene',\n",
              " 'chloride',\n",
              " 'genes',\n",
              " 'tceA',\n",
              " 'vcrA',\n",
              " 'gene',\n",
              " 'organisms',\n",
              " 'products',\n",
              " 'vcrA',\n",
              " 'gene',\n",
              " 'wastewater',\n",
              " 'inclusion',\n",
              " 'wastewater',\n",
              " 'pesticides',\n",
              " 'microcosm',\n",
              " 'pesticide',\n",
              " 'matrix',\n",
              " 'inclusion',\n",
              " 'microcosms',\n",
              " 'pesticides',\n",
              " 'fungicide',\n",
              " 'herbicides',\n",
              " 'isoproturon',\n",
              " 'linuron',\n",
              " 'metamitron',\n",
              " 'polychlorinated',\n",
              " 'methane',\n",
              " 'methanogenic',\n",
              " 'co-substrates',\n",
              " 'formate',\n",
              " 'pcb',\n",
              " 'bacteria',\n",
              " 'domain',\n",
              " 'bacteria',\n",
              " 'rcs-pcb-pf',\n",
              " 'bacteria',\n",
              " 'firmicutes',\n",
              " 'minnesota',\n",
              " 'wastewater-derived',\n",
              " 'wastewater',\n",
              " 'cotinine',\n",
              " 'erythromycin',\n",
              " 'pesticides',\n",
              " 'atrazine',\n",
              " 'metolachlor',\n",
              " 'compounds',\n",
              " 'carbaryl',\n",
              " 'iprodione',\n",
              " 'caffeine',\n",
              " 'human',\n",
              " 'compounds',\n",
              " 'caffeine',\n",
              " 'inorganic',\n",
              " 'water',\n",
              " 'aquatic',\n",
              " 'field-scale',\n",
              " 'water',\n",
              " 'products',\n",
              " 'coliforms',\n",
              " 'coli',\n",
              " 'prokaryotic',\n",
              " 'prokaryotic',\n",
              " 'cells',\n",
              " 'geochemical',\n",
              " 'water',\n",
              " 'water',\n",
              " 'microbes',\n",
              " 'water',\n",
              " 'water',\n",
              " 'sediments',\n",
              " 'bacterial',\n",
              " 'water',\n",
              " 'bacterial',\n",
              " 'water',\n",
              " 'taxa',\n",
              " 'water',\n",
              " 'bacterial',\n",
              " 'water',\n",
              " 'water',\n",
              " 'habitat',\n",
              " 'water',\n",
              " 'pesticides',\n",
              " 'products',\n",
              " 'aquatic',\n",
              " 'known',\n",
              " 'pesticides',\n",
              " 'pesticides',\n",
              " 'pesticides',\n",
              " 'environmentally',\n",
              " 'assimilable',\n",
              " 'carbon',\n",
              " 'microbes',\n",
              " 'assimilable',\n",
              " 'carbon',\n",
              " 'pesticide',\n",
              " 'assimilable',\n",
              " 'carbon',\n",
              " 'microbes',\n",
              " 'assimilable',\n",
              " 'carbon',\n",
              " 'microbes',\n",
              " 'assimilable',\n",
              " 'carbon',\n",
              " 'microbes',\n",
              " 'potable',\n",
              " 'water',\n",
              " 'water',\n",
              " 'greywater',\n",
              " 'potable',\n",
              " 'water',\n",
              " 'introduced',\n",
              " 'greywater',\n",
              " 'water',\n",
              " 'compounds',\n",
              " 'compounds',\n",
              " 'compounds',\n",
              " 'greywater',\n",
              " 'compounds',\n",
              " 'human',\n",
              " 'potable',\n",
              " 'water',\n",
              " 'compounds',\n",
              " 'greywater',\n",
              " 'potable',\n",
              " 'greywater',\n",
              " 'water',\n",
              " 'plants',\n",
              " 'potable',\n",
              " 'water',\n",
              " 'compounds',\n",
              " 'bacterial',\n",
              " 'compounds',\n",
              " 'pharmaceuticals',\n",
              " 'chemical',\n",
              " 'additives',\n",
              " 'pesticides',\n",
              " 'products',\n",
              " 'pollutants',\n",
              " 'compounds',\n",
              " 'acetylene',\n",
              " 'methane',\n",
              " 'inhibitor',\n",
              " 'copper',\n",
              " 'enzyme',\n",
              " 'compound',\n",
              " 'pollutants',\n",
              " 'aromatic',\n",
              " 'aromatic',\n",
              " 'compounds',\n",
              " 'water',\n",
              " 'wastewater',\n",
              " 'site',\n",
              " 'wastewater',\n",
              " 'wastewater',\n",
              " 'micropollutants',\n",
              " 'pharmaceuticals',\n",
              " 'products',\n",
              " 'pesticides',\n",
              " 'chemicals',\n",
              " 'metabolites',\n",
              " 'water',\n",
              " 'compounds',\n",
              " 'pharmaceuticals',\n",
              " 'compounds',\n",
              " 'introduced',\n",
              " 'wastewater',\n",
              " 'wastewater',\n",
              " 'wastewater',\n",
              " 'molecular',\n",
              " 'removed',\n",
              " 'cationic',\n",
              " 'compounds',\n",
              " 'sites',\n",
              " 'micro-pollutants',\n",
              " 'biogenic',\n",
              " 'manganese',\n",
              " 'oxide',\n",
              " 'nanoparticles',\n",
              " 'wastewater',\n",
              " 'pollutant',\n",
              " 'removal',\n",
              " 'removal',\n",
              " 'putida',\n",
              " 'micro-pollutants',\n",
              " 'removed',\n",
              " 'hormones',\n",
              " 'carbendazim',\n",
              " 'removed',\n",
              " 'pesticide',\n",
              " 'pesticide',\n",
              " 'pesticide',\n",
              " 'practices',\n",
              " 'pesticide',\n",
              " 'site',\n",
              " 'pesticide',\n",
              " 'pesticides',\n",
              " 'pesticide',\n",
              " 'pesticide',\n",
              " 'agronomy',\n",
              " 'pesticides',\n",
              " 'water',\n",
              " 'water',\n",
              " 'gene',\n",
              " 'amplicons',\n",
              " 'water',\n",
              " 'water',\n",
              " 'water',\n",
              " 'tap',\n",
              " 'water',\n",
              " 'water',\n",
              " 'water',\n",
              " 'probe',\n",
              " 'water',\n",
              " 'match',\n",
              " 'water',\n",
              " 'water',\n",
              " 'water',\n",
              " 'p73',\n",
              " 'polycyclic',\n",
              " 'aromatic',\n",
              " 'hydrocarbons',\n",
              " 'bacterium',\n",
              " 'genome',\n",
              " 'sequence',\n",
              " 'p73',\n",
              " 'circular',\n",
              " 'chromosome',\n",
              " 'p73',\n",
              " 'plasmids',\n",
              " 'predicted',\n",
              " 'protein-coding',\n",
              " 'sequences',\n",
              " 'genes',\n",
              " 'genes',\n",
              " 'predicted',\n",
              " 'aromatic',\n",
              " 'compounds',\n",
              " 'genes',\n",
              " 'clustered',\n",
              " 'regions',\n",
              " 'fluoranthene',\n",
              " 'fluoranthene',\n",
              " 'genes',\n",
              " 'regions',\n",
              " 'p73',\n",
              " 'metabolites',\n",
              " 'fluoranthene',\n",
              " 'p73',\n",
              " 'fluoranthene',\n",
              " 'p73',\n",
              " 'positions',\n",
              " 'genomic',\n",
              " 'fluoranthene',\n",
              " 'genes',\n",
              " 'genomes',\n",
              " 'gene',\n",
              " 'genomes',\n",
              " 'molecular',\n",
              " 'pesticides',\n",
              " 'pesticide',\n",
              " 'residues',\n",
              " 'water',\n",
              " 'aquatic',\n",
              " 'ecosystem',\n",
              " 'human',\n",
              " 'water',\n",
              " 'pesticides',\n",
              " 'pesticide',\n",
              " 'expected',\n",
              " 'water',\n",
              " 'compounds',\n",
              " 'molecules',\n",
              " 'products',\n",
              " 'sites',\n",
              " 'substances',\n",
              " 'compounds',\n",
              " 'pharmaceuticals',\n",
              " 'products',\n",
              " 'pesticides',\n",
              " 'products',\n",
              " 'compounds',\n",
              " 'compounds',\n",
              " 'compounds',\n",
              " 'compounds',\n",
              " 'perfluorinated',\n",
              " 'compounds',\n",
              " 'triazine',\n",
              " 'products',\n",
              " 'caffeine',\n",
              " 'triazine',\n",
              " 'products',\n",
              " 'water',\n",
              " 'human',\n",
              " 'animal',\n",
              " 'antibiotic',\n",
              " 'genes',\n",
              " 'antibiotic',\n",
              " 'bacteria',\n",
              " 'genes',\n",
              " 'gaps',\n",
              " 'antibiotic',\n",
              " ...]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 148
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "KdZJeI51YOiK",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "4fb0e39d-c5a6-453a-dfcb-8a7c24bb1f43"
      },
      "source": [
        "len(empty_list)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "2701"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 149
        }
      ]
    }
  ]
}